---
title: "**DrosEU Phenotyping Working Group Updates **"
author: "by Esra Durmaz Mitchell & Envel Kerdaffrec"
date: "Last updated: `r format(Sys.Date())`"
output: 
  html_document:
    keep_md: true
    toc: true
    toc_depth: 5
    toc_float: true
    theme: yeti  
---


# Phenotyping effort


**Collaborative phenotyping effort of European _Drosophila melanogaster_ populations**  

Genomic analyses (Kapun et al. 2020) showed that there is longitudinal population structure, continent-wide sweeps, candidate genes for local climate adaptation by using 48 pooled population samples from 32 locations.

In 2017, DrosEU consortium has decided to make a collaborative phenotyping effort of European _Drosophila melanogaster_ populations at the Groningen meeting. 

The call was made with approximate numbers of populations, lines, etc. and under predefined criteria (finish within a year, reps, blocks, etc) and participating labs have indicated which traits they are willing to phenotype. 

## Sampling Locations

```{r libraries, echo=FALSE, message=FALSE}

library(tidyverse)
library(ggpubr)
library(multcompView)
library(hrbrthemes)
library(rworldmap)
library(maps)
library(rnaturalearth)
library(maptools)
library(rgeos)
library(qwraps2)
library(afex)
library(Hmisc)
library(lme4)
library(nlme)
library(multcomp)
library(emmeans)
library(coxme)
library(knitr)
library(sf)
library(rnaturalearthdata)
library(cowplot)
library(googleway)
library(ggrepel)
library(ggspatial)
library(DT)
library(data.table)
library(meta)
library(metafor)
library(survival)
library(survminer)
```

Nine sampling locations were chosen based on genomic data and these locations covered a wide range of latitude (~20°) and longitude (~40°) across the continent (see table below). From each location, 15 to 20 isofemale lines were established in corresponding labs at the sampling location and the isofemale lines were centrally maintained by Élio Sucena  at Instituto Gulbenkian de Ciência (IGC), Lisbon, Portugal. A total of 173 isofemale lines were used in this study. 


```{r droseu_coord, echo=FALSE, message=FALSE}
droseu_coord <- read.csv("InfoTables/DrosEU_Coordinates.csv")

droseu_coord <- data.frame(droseu_coord %>%
  add_column(Collector = c("Jorge Vieira", "Josefa Gonzalez", "Mads Schou", "Amanda Glaser-Schmitt","Andrea Betancourt","Maaria Kankare","Iryna Kozeretska","Banu Onder","Elena Pasyukova")))
```

```{r echo=FALSE, message=FALSE}
knitr::kable(droseu_coord, align = "llcccl", caption = 'Table X. Sampling locations, coordinates and collectors, sorted by increasing latitude')
```

```{css, echo = FALSE}
caption {
      color: black;
      font-weight: bold;
      font-size: 1.0em;
    }
```

```{r samples_map, echo=FALSE, message=TRUE}
world <- ne_countries(scale = "medium", returnclass = "sf")
samples_map <- ggplot(data = world) + geom_sf() + xlab("Latitude") + ylab("Longitude") + ggplot2::coord_sf(xlim = c(-12, 38), ylim = c(30, 65), expand = FALSE) + annotation_scale(location = "bl", width_hint = 0.5) + ggtitle("Sampling locations")
```

```{r map_theme, echo=FALSE, message=FALSE}
map_theme <- theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black",), axis.title.x = element_text(size = 16), axis.text.x = element_text(size = 14, colour = "black"),axis.text.y = element_text(size = 14, colour = "black"),axis.title.y = element_text(size = 16), plot.title = element_text(color="black", size=14, face="bold"))
```

```{r panmix_theme, echo=FALSE, message=FALSE}
panmix_theme <- theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(),panel.background = element_blank(), axis.line = element_line(colour = "black",), axis.title.x = element_text(size = 16), axis.text.x = element_text(size = 16, colour = "black"), axis.text.y = element_text(size = 16, colour = "black"),axis.title.y = element_text(size = 16))
```

```{r samples_map_2, echo=FALSE, message=TRUE}
samples_map + map_theme + 
  annotate(geom = "point", x = -8.410, y = 41.150, colour = "orange", size = 3) + 
  annotate(geom = "point", x = 0.620, y = 41.618,colour = "orange", size = 3) + 
  annotate(geom = "point", x = 10.213, y = 55.945, colour = "orange", size = 3) + 
  annotate(geom = "point", x = 32.260, y = 40.231, colour = "orange", size = 3) + 
  annotate(geom = "point", x = 11.610, y = 48.180, colour = "orange", size = 3) + 
  annotate(geom = "point", x = 15.560, y = 48.375, colour = "orange", size = 3) + 
  annotate(geom = "point", x = 33.244, y = 57.979, colour = "orange", size = 3) + 
  annotate(geom = "point", x = 30.206, y = 48.753, colour = "orange", size = 3) + 
  annotate(geom = "point", x = 23.520, y = 61.100, colour = "orange", size = 3) +
  annotate(geom = "text", x = -8.410, y = 41.150, label ="PT",fontface = "bold", color = "black", size =5, angle=15) +
  annotate(geom = "text", x = 0.620, y = 41.618, label ="ES",fontface = "bold", color = "black", size =5, angle=15) + 
  annotate(geom = "text", x = 10.213, y = 55.945, label ="DK",fontface = "bold", color = "black", size =5, angle=15) + 
  annotate(geom = "text", x = 32.260, y = 40.231, label ="TR",fontface = "bold", color = "black", size =5, angle=15) + 
  annotate(geom = "text", x = 11.610, y = 48.180, label ="DE",fontface = "bold", color = "black", size =5, angle=15) + 
  annotate(geom = "text", x = 15.560, y = 48.375, label ="AT",fontface = "bold", color = "black", size =5, angle=15) + 
  annotate(geom = "text", x = 33.244, y = 57.979, label ="RU",fontface = "bold", color = "black", size =5, angle=15) + 
  annotate(geom = "text", x = 30.206, y = 48.753, label ="UA",fontface = "bold", color = "black", size =5, angle=15) + 
  annotate(geom = "text", x = 23.520, y = 61.100, label ="FI",fontface = "bold", color = "black", size =5, angle=15) 

```

```{r droseu_labs, echo=FALSE, message=FALSE}
droseu_labs <- data.frame(read.csv("InfoTables/DrosEU_Labs.csv"))
```

## Contributers (phenotyping)

Contributed labs are `r unique(droseu_labs$PI)` and detailed list below :


```{r echo=FALSE}
knitr::kable(droseu_labs, allign = "llll", caption = "Table X. Contributed labs and traits, sorted by PI") 
```

```{r echo=FALSE, message=FALSE}
cities <- read.csv("InfoTables/DrosEU_cities.csv")
```


```{r research_groups_map, echo=FALSE, message=TRUE}

globe_labs <- ggplot(data = world) + geom_sf() + xlab("Latitude") + ylab("Longitude") +
  coord_sf(xlim = c(-80.15, 40.12), ylim = c(-45, 60), expand = TRUE) +
  ggtitle("Contributed labs", subtitle = paste0("(", length(unique(droseu_labs$Country)), " countries", ", ", length(unique(droseu_labs$PI)), " research groups", ")")) +
  annotate(geom = "point", x = cities$Long, y = cities$Lat, colour = "red", size = 2) 
  
globe_labs + map_theme 

```

## Contributers (analyses)

Esra Durmaz (all), Envel Kerdaffrec (all), Katja Hoedjes (Via, DT, DW), Banu Onder (SR, WA), Marija Tanaskovic (WA), Chris Wegener (CET), Eran Tauber (LA), Rudolf Rohr (UniFr), coordination-team


# Methods

## Wolbachia _work in progress_

Sucena, Pasyukova, Kapun, Viera and Kozeretska Labs have genotyped the isofemale lines for Wolbachia using different methods and some discrepancies were observed, mainly due to the sensitivity differences in protocols -- we are working on it. 

Tables below were made using Sucena Lab's results. 

```{r wolbachia-data, echo=FALSE, message=FALSE}
droseu_wol <- data.frame(read.csv("InfoTables/DrosEU_wolbachia_2021.csv"))
```

```{r, echo=FALSE}
DT::datatable(droseu_wol, options = list(pageLength = 10))
```

```{r Wol_Freqs, echo=FALSE}
Wol_Freqs <- data.frame(unique(droseu_wol$Population))

Wol_Pops <- c((AK_freq <- nrow(droseu_wol %>% filter(Population=="AK") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "AK")) * 100), 
  (GI_freq <- nrow(droseu_wol %>% filter(Population=="GI") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "GI")) * 100), 
  (KA_freq <- nrow(droseu_wol %>% filter(Population=="KA") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "KA")) * 100), 
  (MA_freq <- nrow(droseu_wol %>% filter(Population=="MA") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "MA")) * 100), 
  (MU_freq <- nrow(droseu_wol %>% filter(Population=="MU") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "MU")) * 100), 
  (RE_freq <- nrow(droseu_wol %>% filter(Population=="RE") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "RE")) * 100),
  (UM_freq <- nrow(droseu_wol %>% filter(Population=="UM") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "UM")) * 100),
  (VA_freq <- nrow(droseu_wol %>% filter(Population=="VA") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "VA")) * 100),
  (YE_freq <- nrow(droseu_wol %>% filter(Population=="YE") %>% filter(Wolbachia=="1")) / 
  nrow(droseu_wol %>% filter(Population == "YE")) * 100))

pop_freqs <- cbind(unique(droseu_wol$Country), Wol_Freqs, Wol_Pops)

cols <- c("Country", "Population", "Wolbachia_Freq")
wolbachia_pop_freqs <- setnames(pop_freqs, cols)

```

```{r wolbachia_pop_freqs}
wolbachia_pop_freqs # Wolbachia frequency by country and population abbreviation 
```

## Diets

```{r droseu_diets, echo=FALSE, message=FALSE}
droseu_diets <- data.frame(read.csv("InfoTables/DrosEU_Diets_Feb22.csv"))
```

```{r, echo=FALSE}
DT::datatable(droseu_diets, options = list(pageLength = 10))
```


## Inversion Frequencies _work in progress_
_need to do a few more PCRs, will add them here soon_

## Traits 

_Contributors and methods for experiments_

### Viability
##### Gibert, Grath, Hoedjes, Schmidt, Stamenkovic-Radak and Zwaan Labs


3-5 day old adults (at least 25 pairs) are allowed to lay eggs en masse. Yeast is provided to stimulate egg laying (for at least 2 hours). Eggs are collected, and 40 are placed in each vial[^1]. Viability is calculated per vial, as the percentage of individuals that emerged from the 40 eggs.


[^1]: @ Schmidt lab, females were allowed to lay eggs in vials and total number of eggs were recorded. Viability is calculated as the percentage of individuals that emerged from total number of eggs, per vial.

### Developmental time

#### Egg-to-pupa developmental time
##### Schmidt Lab


#### Egg-to-adult developmental time
##### Gibert, Grath, Hoedjes, Schmidt, Stamenkovic-Radak and Zwaan Labs


Developmental time is scored as both the egg-to-pupa and egg-to-adult development time. Both were scored twice a day, when the chamber lights are turned on and two hours before they are turned off. In order to measure the egg-to-pupa developmental time, the spot where a pupa is found is marked with a permanent marker to keep track of which pupae have emerged in each day. The egg-to-adult developmental time is estimated by counting all emerged adults from the vial, and by sexing them.


### Dry weight
##### Colinet, Hoedjes and Onder Labs

At day 7 after emergence, flies are killed by snap freezing them in liquid nitrogen, by putting them at -20ºC or by putting them into an ethyl acetate solution and stored at -20ºC. Then they are sexed and placed into 96 wells plates, and placed in an oven set at 60-70 °C, for at least 3 days. At this point flies can be stored at room temperature using a protective cover. If this is the case, the day before measurements are made dry flies are again placed for 24h in the oven (60-70 °C) to ensure material is well dehydrated. Flies are then placed on a small piece of aluminium foil for direct weight measurement on microbalance (accuracy 1µg).


### Thorax length
##### Kozeretska, Posnien, Ritchie and Schmidt Labs

Five to seven days old flies are placed onto a double-sided sticky tape attached to a microscope slide and a picture of the thorax taken using a digital camera connected to a dissecting microscope. The same magnification and resolution is always used to increase reproducibility, and a scale bar inserted on each photo to allow transforming pixels into µm units. Thorax length is defined as the distance from the anterior margin of the thorax to the posterior tip of the scutellum and it is measured using the “Straight Line” in ImageJ/Fiji.


### Wing area
##### Onder, Posnien, Ritchie and Stamenkovic-Radak Labs

Both the left and right wings of five to seven days old flies (10 flies per sex per replicate) are removed and placed into a drop of Entellan®Merck, Hoyer’s Medium, sticked to a double-side sticky tape, or taped directly to the slide. Pictures of the wing preparations are taken using a digital camera connected to a dissecting microscope. The same magnification and resolution is always used to increase reproducibility. A scale bar is placed on each photo to allow transforming pixels into µm units. Manual measurements of wing length and wing area are performed using the “Straight Line” and “Polygon Selection” tools, respectively of ImageJ/Fiji (10.1371/journal.pone.0000007).


### Fecundity
##### Billeter and Fricke Labs

For each isofemale line, 10 males and 10 females are placed together in single-sex groups and allowed to mature for five days. Then, they are placed together (5-7 pairs), and mating interactions observed to ensure successful mating (at least 10 min copulation duration) to ensure that we have five successfully mated females. After a successful mating, males are discarded and females allowed to oviposit alone for 48 hours, moved to another vial, and allowed to oviposit for four days, and again moved to another vial and allowed to oviposit for two days to check that of egg-laying stopped. Vials are incubated until all offspring is born. Individuals are then frozen and the offspring counted.


### Lifespan
##### Flatt, Parsch and Pasyukova Labs

Line level lifespan: Ten flies per sex/line are placed in each vial. The age at death will be scored when changing the food, at least three times a week. Five replicates are used.

Population level lifespan: Flies are kept in 1L demography cages (5 flies per line/sex for each population). The age at death will be scored when changing the food, at least three times a week. Ten replicates were performed.


### Cold-shock mortality
##### Gonzalez, Kozeretska and Vieira Labs

Batches of 15-20 seven days old flies are placed for 18 hours in an empty vial immersed in an ice-water slurry box placed at a 4°C room for 18 hours. Then the vials are removed to a bench in a 25°C room and mortality scored 24 hours later.


### Chill-coma recovery time
##### Vieira and Mensh Labs

Sexed flies are placed in an empty vial immersed in an ice-water slurry box placed at a 4°C room in the morning. Six hours later, flies are removed from the tube to individual wells of 24 well plates while being kept on ice. A timer is started once the plate is moved from the ice to a bench in a 25°C room. Each fly is checked by eye for recovery for a maximum of 60 minutes. Flies that are able to stand on their legs are considered recovered and the CCRT (in seconds) recorded.


### Heat-shock mortality
##### Parsch and Vieira Labs

Batches of 15-20 seven days old flies are placed in empty vials inside a 37ºC incubator and mortality checked for 7 hours every 30 minutes.


### Diapause
##### Bergland, Flatt and Schlotterer Labs

In order to induce diapause, two hour old 'phenotypic virgins' (pharate or melanized with meconium visible) female flies are exposed to 12°C and 10:14 light/dark hours for 3 weeks, using an incubator that allows temperature tracking. Vials are changed once per week. After three weeks under diapause conditions, flies are frozen at -80C until dissection. Both ovaries will be examined and classified according to the following simplified 'classification': 1) < stage 10: diapause; 2) stage 10-13: intermediate; 3) stage 14: non-diapause.


- Standard diapause conditions: 12°C at 10:14 hrs L:D (under these conditions we might expect around 50% dipause frequency)
- duration of exposure: 3 weeks
- assays will be done in incubators (Percival or Memmert); temperature conditions will be tracked using data loggers / iButtons
- Try to have haphazard standardization; similar conditions coupled with consistent tracking of temperature (multiple iButtons, for example)
- We will use egg lays in bottles to be able to collect more flies for the assay (approx. 1-2 bottles per line)
- Larval density will be controlled (approximately) prior to assays: we will do this by letting a fixed number of (approx. 10-20) females (housed together with 5-10 males) oviposit over night (fixed time window) - how many females and the duration of egg laying might need to be adjusted a bit depending on the fertility of the lines (to be determined when lines are being amplified prior to assays)
- The Bergland and Flatt labs will standardize their fly food according to the Gulbenkian recipe (in terms of amounts / proportions); Manolis from the Schlötterer lab will not standardize the food and use their local diet; importantly, flies for the assays will not be exposed to any (additional) live yeast (i.e., the diet will contain liophilized yeast but no live yeast sprinkled on top).
- From the bottles we will collect upon eclosion 'phenotypic virgins' (=pharate adults or melanized with meconium visible), approx. within 2 hours after eclosion
- Flies will be kept in vials for assay (15-20 flies in the same vial to account for mortality); we aim to measure a minimum of 10 females per line (from approx. 10-15 lines x 10 populations, yielding about 1000-1500 flies for dissections in total)
- The food in the assays will be changed once per week
- To keep the workload and effort manageable the assays will be performed in 2-3 blocks (optimally only 2) over time: e.g., from each population we will measure a subset of all lines (e.g., 5-7 lines, i.e. 1/2 or 1/3 of all lines) in each of the 2-3 blocks. 
- After 3 weeks under diapause conditions, flies will either be dissected or kept frozen until dissection (dry, at -80C; can be stored for a few months) 
- We will adopt a simplified diapause 'classification':
- Stage 10 will be defined as the checkpoint
- < Stage 10: diapause
- Stage 10-13: intermediate
- Stage 14: non-diapause
- We will examine both ovaries.


### Circadian eclosion timing
##### Wegener Lab

The eclosion rhythmicity has been measured in outcrossed Munich, Akaa, Gimenells, Uman, Mauternbach, Karensminde, Valday, Yesiloz and Recarei strains at:

1) LD14:10 @ 29°C, flies raised at 29°C

2) DD 29°C after LD14:10 entrainment, flies raised at 29°C

3) LD14:10 @ 18°C, flies raised at 18°C

4) DD 18°C after LD14:10 entrainment, flies raised at 18°C

### Locomotor activity
##### Tauber Lab

The locomotor activity of the flies (males only) was measured using the DAM2 Drosophila monitors (Trikinetics Inc., Waltham, MA). Flies were 1-3 days old. Single flies were placed in glass tubes (10 cm × 0.5 cm) that were filled with 2 cm sugar/agar medium. The monitors were placed in light chambers driven by LED,  in an incubator at 24°C, ~30% humidity. The flies were entrained to a light-dark cycle (LD 12:12) for 5d and then allowed to free-run for 10 d in constant darkness (DD). The activity data were processed into 30 min bins, and four different variables were analysed. These included the circadian period, and the phase, which were analyzed using the FFT NLLS algorithm available at the BioDare2 server (https://biodare2.ed.ac.uk/). The other two variables, level of activity and the nocturnal/diurnal ratio were analyzed by a custom-made R script.
 

### Starvation resistance
##### Gonzalez, Onder and Pasyukova Labs

Batches of 10 sexed flies are transferred to glass vials filled with 5 mL of 2% agar for starvation 3-7 days after eclosion. The age at death will be scored every 8 hours.

### Pigmentation
##### Abbott, Gibert and Schmidt Labs

Abbott and Gibert Labs : For each line, 10 females, 13-15 days old, either alive or stored in 95% ethanol, are air dried and placed on their left side, and pictures taken using a dissecting microscope. Images are then analysed in ImageJ 1.46r, using the Area Fraction measurement in the Analyze menu. Area Fraction measures the percentage of pixels in a selected area that have been highlighted in red using the Threshold tool. This gives an estimate of the percentage of dark pigmentation on the three terminal tergites of the abdomen (4, 5 and 6). 

Schmidt Lab : The last three tergites were scored according to David et al. 1990.  

### Parasitoid resistance 
##### Hrcek Lab

The resistance of the lines to parasitoids will be measured using _Leptopilina boulard_. 

## Data reformatting 

MasterSheets 


All data files are reformatted as so called _Master Sheets_, in which the data structure is the same for all the sub-data sets and all details/edits logged in a README sheet. 

```{r README_example, echo=FALSE}
headers <- c("Supervisor/PI", "Diet", "Batch", "Population", "Line", "Sex", "ReplicateVial", "TraitValue") 

descriptions <- c("Lab identifier", "Either standardized _S_ or non-standardized _NS_ ", "From 1 to n" , "Population identifier in abbreviated form (i.e. _AK_)", "Line identifier (i.e. _MU1_)", "Either female (F) or male (M)", "Replicate identifier (from 1 to n)", "Trait value in x units")

README_example <- data.table(Headers =  headers, Descriptions = descriptions)
```

```{r echo=FALSE, message=FALSE}
knitr::kable(README_example, align = "ll", caption = 'Table X. Filling in the data spreadsheet and header description')
```


```{r MS_example, echo=FALSE}
MS_example <- data.table(read.csv("Data/MasterSheets_Feb22_git/VIA_MasterSheet_Feb22.csv"))
```

```{r, echo=FALSE}
DT::datatable(MS_example, options = list(pageLength = 10)) 
```


## Data check

For each trait and lab, we ran counts for sex, population, line and replicate vial (if applicable). If we could spot anything suspicious or unusual (typos, shifts, etc.), I have contacted you already. Please let me know if there is anything that you have spotted and we missed. Thanks! 


# Results I : Linear Models  {#anchor}

## Examples & Directories & Legends 

### Tables 

Here is an example code for tables (egg-to-adult viability). The very same code is used for other traits (either with or without _Batch_) by filtering the data at supervisor, batch, population and/or line levels (and also for sex, if applicable). 

```{r descriptive_stats, eval=TRUE, echo=TRUE, message=FALSE, warning=FALSE}

## First defined standard error and coefficient of variation

std_err <- function(x) sd(x)/sqrt(length(x))
coef_var <- function(x) sd(x)/mean(x)
estimate_mode <- function(v) {
   uniqv <- unique(v)
   uniqv[which.max(tabulate(match(v, uniqv)))]
}
```

```{r, eval=FALSE, echo=TRUE, message=FALSE, warning=FALSE}

# Then compute descriptive statistics (mean, median, minimum, maximum, standard error, coefficient of variation and mode) at the defined levels (line with batch information example below) : 

table_Via_Line_wbatch <- write.csv(d_Via %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(ProportionEggtoAdultSurvival), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Viability/table_Via_Line_wbatch.csv", row.names = T)

table_Via_Line_wbatch <- read.csv("Viability/table_Via_Line_wbatch.csv") #file name is here 
```

Tables for descriptive statistics at population and line levels can be found in the trait directory, under the file name **__table_TraitAbbreviation_Level_BatchInfo.csv__**  (_i.e. Viability/table_Via_Line_wobatch.csv_)

### Figures

Data range is calculated with `#r min()` and `#r max()` functions for each trait.

y-axis on the figures is scaled by the minimum (`#r min_trait`) and maximum (`#r max_trait`) values in the full data set for a given trait. 

The very same ggplot theme has been used all across the document, called _droseu_theme_

```{r droseu_theme, eval=FALSE, echo=TRUE}
droseu_theme <- theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black",),  axis.title.x = element_text(size = 16), axis.text.x = element_text(size = 16),axis.text.y = element_text(size = 16),axis.title.y = element_text(size = 16))

p_TraitAbbreviation_Lab + ylim(c(min_trait, max_triat)) + droseu_theme 
```

```{r, echo=FALSE}
## Need to run this in the background
droseu_theme <- theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(), panel.background = element_blank(), axis.line = element_line(colour = "black",),  axis.title.x = element_text(size = 16), axis.text.x = element_text(size = 16),axis.text.y = element_text(size = 16),axis.title.y = element_text(size = 16))
```


Here is an example code for figures (egg-to-adult viabiliy, Gibert Lab). The very same code is used to generate plots for other contributing labs and traits (either for _Batch_ or _Population_) by filtering the data at supervisor level (for females and males separately, if applicable). 

```{r plot_example, eval=FALSE, echo=TRUE, message=FALSE, warning=FALSE}

p_Via_Gibert <- ggplot(data = (subset(d_Via,Supervisor.PI=='Gibert')), 
                       #subsets for supervisor
                       aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Batch)) +#define x- and y-axis
                       geom_boxplot(outlier.shape = NA, notch=FALSE) +  #boxplot
                       labs(title="p_Via_Gibert", x="Population", y = "ProportionEggtoAdultSurvival") + 
                      ylim(c(min_Via, max_Via)) + 
                      droseu_theme #y-axis limits, axis labels and droseu_theme

pdf(file="Viability/p_Via_Gibert.pdf",width=8, height=5)

p_Via_Gibert

invisible(dev.off())
```



### Linear Models

Here is an example code for linear models (egg-to-adult development time, males, Gibert Lab). The very same code is used for other contributing labs and traits by filtering the data at supervisor level (for females and males separately, if applicable), adding or removing factors if they cause convergence problems or singularity (i.e. if only one replicate vial has been phenotyped per line and/or population and/or batch, the lowest level [ _replicate_vial_  in this example] has been dropped from given model). 

```{r lmer_example, eval=FALSE, echo=TRUE, message=FALSE, warning=FALSE}
DT_A_M_lmer_Gibert <- lmer(DT_EggAdult ~ Population + 
                             #Population (fixed)
                             (1|Line:Population) + 
                             #Line (random) is nested in Population (fixed)
                             (1|Batch) +
                             #Batch is a random effect 
                             (1|ReplicateVial : Line),
                            #Replicate vial (random) is nested in Line (random)
                           data = d_DT_A_M[d_DT_A_M$Supervisor.PI == "Gibert",]) 
                            #Filter by Lab in development time, males data 
capture.output(summary(DT_A_M_lmer_Gibert),file = "DevelopmentTime/DT_A_M_lmer_Gibert_sum.txt") #save output summary 
capture.output(anova(DT_A_M_lmer_Gibert),file = "DevelopmentTime/DT_A_M_lmer_Gibert.txt") #save anova output 

```

```{r tukey_example, eval=FALSE, echo=TRUE, message=FALSE, warning=FALSE}
#need to put in tukey example here 
```

```{r anova_example, eval=FALSE, echo=TRUE, message=FALSE, warning=FALSE}
anova(DT_A_M_lmer_Gibert) #print anova output
```

```{r summary_example, eval=FALSE, echo=TRUE, message=FALSE, warning=FALSE}
summary(DT_A_M_lmer_Gibert) #print model output summary 
```


Here is an example code for linear models for geography (altitude, dry weight, males, Onder Lab). The very same code is used for other contributing labs and traits by filtering the data at supervisor level (for females and males separately, if applicable), adding or removing factors if they cause convergence problems or singularity (i.e. if only one replicate vial has been phenotyped per line and/or population and/or batch, the lowest level [ _replicate_vial_  in this example] has been dropped from given model) and also for latitude and longitude. 

```{r genography_example, eval=FALSE, echo=TRUE, message=FALSE, warning=FALSE}
DW_M_lmer_Alt_Onder <- lmer(DW_micrograms ~ Altitude + (1|Population) + (1|Population:Line), data = d_DW_M[d_DW_M$Supervisor.PI == "Onder",])
capture.output(summary(DW_M_lmer_Alt_Onder),file = "DryWeight/DW_M_lmer_Alt_Onder_sum.txt")
capture.output(anova(DW_M_lmer_Alt_Onder),file = "DryWeight/DW_M_lmer_Alt_Onder.txt")
```


### Survival Models

Here is an example code for survival analyses (starvation resistance, females, Gonzalez Lab). The very same code is used for other contributing labs and traits by filtering the data at supervisor level and sex. Please note that the level "replicate_vial" has been dropped from all survival analyses, as it clogs the models due to the volume of data.

```{r coxme_example, eval=FALSE, echo=TRUE, message=FALSE, warning=FALSE}
SR_F_coxme_Gonzalez <- coxme(Surv(AgeAtDeath_hours, Censor) ~ Population + (1|Batch) + (1|Population/Line) , data = filter(d_SR_surv, Supervisor.PI == "Gonzalez", Sex == "F"))
capture.output(summary(SR_F_coxme_Gonzalez), file = "Starvation/SR_F_coxme_Gonzalez_sum.txt")
capture.output(anova(SR_F_coxme_Gonzalez), file = "Starvation/SR_F_coxme_Gonzalez.txt")

```

All analyses outputs can be found in the trait directory (Analyses/Trait), under the file names  **__TraitAbbreviation_Function_Output.txt__** . The analyses at the lab level can be found in the same trait directory, under _by_lab_ folders (_i.e. LinearModelsPop/DevelopmentTime/by_lab_txt_output/DT_A_M_Gibert_lmer_pop_anova.txt_)

**Reading the data, tables and model outputs in R**
```{r load_Rdas, include=TRUE}
droseu <- readRDS("Data/droseu_master_list_2022-04-05.rds")
lmers_anova <- readRDS("LinearModelsPop/all_lmers_list_pop_anova.rds")
lmers_tukey <- readRDS("LinearModelsPop/all_lmers_list_pop_tukey.rds")
lmers_sum <- readRDS("LinearModelsPop/all_lmers_list_pop_summary.rds")
table_line_wbatch <- readRDS("DescriptiveStatistics/all_table_Line_wbatch.rds")
table_line_wobatch <- readRDS("DescriptiveStatistics/all_table_Line_wobatch.rds")
table_pop_wbatch <- readRDS("DescriptiveStatistics/all_table_Pop_wbatch.rds")
table_pop_wobatch <- readRDS("DescriptiveStatistics/all_table_Pop_wobatch.rds")
```


**_Please note that "Plots and Linear Models by Lab" are presented in alphabetical order._** 

## Viability

```{r include=FALSE}
dir.create(file.path("Viability"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

### Contributors  

Gibert Lab : Cristina Vieira, Laurence Mouton, Natacha Kremer, Sonia Martinez, Patricia Gibert

Grath Lab : Ingo Müller, Sonja Grath

Hoedjes Lab : Hristina Kostic, Katja Hoedjes

Schmidt Lab : Ozan Kiratli, Yonatan Babore, Liam Forsythe, Paul Schmidt

Stamenkovic-Radak Lab : Marija Savic Veselinovic, Marija Tanaskovic, Aleksandra Patenkovic, Mihailo Jelic, Katarina Eric, Pavle Eric, Slobodan Davidovic, Marina Stamenkovic-Radak

Zwaan Lab : Joost van den Heuvel, Bas Zwaan


```{r}
str(droseu$via)
```

### Descriptive Statistics

Descriptive statistics at the line level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$Via, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$Via, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$Via, options = list(pageLength = 10))  %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$Via, options = list(pageLength = 10)) %>% formatRound(4:11, 3)
```

### Plots and Linear Models by Lab  

#### Overview 

```{r min_max_Via, echo=TRUE}
min_Via <- min(droseu$via$ProportionEggtoAdultSurvival)
max_Via <- max(droseu$via$ProportionEggtoAdultSurvival)
```

y-axis is scaled by the minimum (`r min_Via`) and maximum (`r max_Via`) values in the full data set.

```{r p_Via_all, echo=FALSE}
p_Via_all <- ggplot(data = droseu$via, aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Supervisor.PI)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_Via_all", x="Population", y = "DW_micrograms") + 
  ylim(c(min_Via, max_Via))+ droseu_theme
```

```{r echo=FALSE}
p_Via_all
```

```{r echo=FALSE}
pdf(file="Viability/p_Via_all.pdf",width=8, height=5)
p_Via_all
invisible(dev.off())
```

#### Gibert Lab

```{r}
lmers_anova$Via_Gibert_lmer_pop
```

```{r}
lmers_sum$Via_Gibert_lmer_pop
```

```{r p_Via_Gibert, echo=FALSE}
p_Via_Gibert <- ggplot(data = filter(droseu$via, Supervisor.PI == "Gibert"), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_Gibert", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via)) + droseu_theme
```

```{r echo=FALSE}
p_Via_Gibert
```

```{r echo=FALSE}
pdf(file="Viability/p_Via_Gibert.pdf",width=8, height=5)
p_Via_Gibert
invisible(dev.off())
```

```{r p_Via_pop_Gibert, echo=FALSE, message=FALSE, warning=FALSE}
p_Via_pop_Gibert <- ggplot(data = filter(droseu$via, Supervisor.PI == "Gibert"), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Population))+geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_pop_Gibert", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via)) + geom_text(data = lmers_tukey$Via_Gibert_lmer_pop$letters, aes(x = Population, label = cld), y = Inf, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_Via_pop_Gibert
```

```{r, echo=FALSE, message=FALSE, warning=FALSE}
pdf(file="Viability/p_Via_pop_Gibert.pdf",width=8, height=5)
p_Via_pop_Gibert
invisible(dev.off())
```

#### Grath Lab

```{r}
lmers_anova$Via_Grath_lmer_pop
```

```{r}
lmers_sum$Via_Grath_lmer_pop
```

```{r p_Via_Grath, include=FALSE}
p_Via_Grath <- ggplot(data = filter(droseu$via, Supervisor.PI == "Grath"), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_Grath", x="Population", y = "ProportionEggtoAdultSurvival")+ ylim(c(min_Via, max_Via))+ droseu_theme
```

```{r echo=FALSE}
p_Via_Grath
```

```{r include=FALSE}
pdf(file="Viability/p_Via_Grath.pdf",width=8, height=5)
p_Via_Grath 
invisible(dev.off())
```

```{r, include=FALSE}
p_Via_pop_Grath <- ggplot(data = filter(droseu$via, Supervisor.PI == "Grath"), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_pop_Grath", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via))+ geom_text(data = lmers_tukey$Via_Grath_lmer_pop$letters, aes(x = Population, label = cld), y = Inf, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_Via_pop_Grath
```

```{r include=FALSE}
pdf(file="Viability/p_Via_pop_Grath.pdf",width=8, height=5)
p_Via_pop_Grath
invisible(dev.off())
```

#### Hoedjes Lab

```{r}
lmers_anova$Via_Hoedjes_lmer_pop
```

```{r}
lmers_sum$Via_Hoedjes_lmer_pop
```

```{r p_Via_Hoedjes, include=FALSE}
p_Via_Hoedjes <- ggplot(data = filter(droseu$via, Supervisor.PI == "Hoedjes"), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_Hoedjes", x="Population", y = "ProportionEggtoAdultSurvival")+ ylim(c(min_Via, max_Via))+ droseu_theme
```

```{r echo=FALSE}
p_Via_Hoedjes
```

```{r echo=FALSE}
pdf(file="Viability/p_Via_Hoedjes.pdf",width=8, height=5)
p_Via_Hoedjes
invisible(dev.off())
```

```{r p_Via_pop_Hoedjes, include=FALSE}
p_Via_pop_Hoedjes <- ggplot(data = (subset(d_Via,Supervisor.PI=='Hoedjes')), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_pop_Hoedjes", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via))+  geom_text(data = lmers_tukey$Via_Hoedjes_lmer_pop$letters, aes(x = Population, label = cld), y = Inf, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_Via_pop_Hoedjes
```

```{r echo=FALSE}
pdf(file="Viability/p_Via_pop_Hoedjes.pdf",width=8, height=5)
p_Via_pop_Hoedjes
invisible(dev.off())
```

#### Schmidt Lab

```{r}
lmers_anova$Via_Schmidt_lm_pop
```

```{r}
lmers_sum$Via_Schmidt_lm_pop
```

```{r p_Via_Schmidt, include=FALSE}
p_Via_Schmidt <- ggplot(data = filter(droseu$via, Supervisor.PI=='Schmidt'), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_Schmidt", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via))+ droseu_theme
```

```{r echo=FALSE}
p_Via_Schmidt
```

```{r echo=FALSE}
pdf(file="Viability/p_Via_Schmidt.pdf",width=8, height=5)
p_Via_Schmidt
invisible(dev.off())
```

```{r include=FALSE}
p_Via_pop_Schmidt <- ggplot(data = filter(droseu$via, Supervisor.PI=='Schmidt'), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Population)) +geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_pop_Schmidt", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via))+   geom_text(data = lmers_tukey$Via_Schmidt_lm_pop$letters, aes(x = Population, label = cld), y = max_Via, vjust = 1) + droseu_theme
```

```{r}
p_Via_pop_Schmidt
```

```{r p_Via_pop_Schmidt, include=FALSE}
pdf(file="Viability/p_Via_pop_Schmidt.pdf",width=8, height=5)
p_Via_pop_Schmidt
invisible(dev.off())
```

#### Stamenkovic-Radak Lab

```{r}
lmers_anova$Via_StamenkovicRadak_lmer_pop
```

```{r}
lmers_sum$Via_StamenkovicRadak_lmer_pop
```

```{r p_Via_StamenkovicRadak, include=FALSE}
p_Via_StamenkovicRadak <- ggplot(data = filter(droseu$via,Supervisor.PI=='StamenkovicRadak'), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_StamenkovicRadak", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via))+ droseu_theme
```

```{r echo=FALSE}
p_Via_StamenkovicRadak
```

```{r , include=FALSE}
pdf(file="Viability/p_Via_StamenkovicRadak.pdf",width=8, height=5)
p_Via_StamenkovicRadak
invisible(dev.off())
```

```{r p_Via_pop_StamenkovicRadak, include=FALSE}
p_Via_pop_StamenkovicRadak <- ggplot(data = filter(droseu$via,Supervisor.PI=='StamenkovicRadak'), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_pop_StamenkovicRadak", x="Population", y = "ProportionEggtoAdultSurvival")  + ylim(c(min_Via, max_Via))+ geom_text(data = lmers_tukey$Via_StamenkovicRadak_lmer_pop$letters, aes(x = Population, label = cld), y = Inf, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_Via_pop_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="Viability/p_Via_pop_StamenkovicRadak.pdf",width=8, height=5)
p_Via_pop_StamenkovicRadak
invisible(dev.off())
```

#### Zwaan Lab

```{r}
lmers_anova$Via_Zwaan_lmer_pop
```

```{r}
lmers_sum$Via_Zwaan_lmer_pop
```

```{r p_Via_Zwaan, include=FALSE}
p_Via_Zwaan <- ggplot(data = filter(droseu$via,Supervisor.PI=='Zwaan'), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_Zwaan", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via))+ droseu_theme
```

```{r echo=FALSE}
p_Via_Zwaan
```

```{r include=FALSE}
pdf(file="Viability/p_Via_Zwaan.pdf",width=8, height=5)
p_Via_Zwaan
invisible(dev.off())
```

```{r p_Via_pop_Zwaan, include=FALSE}
p_Via_pop_Zwaan <- ggplot(data = filter(droseu$via,Supervisor.PI=='Zwaan'), aes(x=Population_Lat, y=ProportionEggtoAdultSurvival, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Via_pop_Zwaan", x="Population", y = "ProportionEggtoAdultSurvival") + ylim(c(min_Via, max_Via))+  geom_text(data = lmers_tukey$Via_Zwaan_lmer_pop$letters, aes(x = Population, label = cld), y = Inf, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_Via_pop_Zwaan
```

```{r include=FALSE}
pdf(file="Viability/p_Via_pop_Zwaan.pdf",width=8, height=5)
p_Via_pop_Zwaan
invisible(dev.off())
```

### Linear Models / Residuals 

#### Gibert Lab
```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Gibert_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Gibert_lmer_pop_qq_plot_residuals.png")
```

#### Grath Lab
```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Grath_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Grath_lmer_pop_qq_plot_residuals.png")
```

#### Hoedjes Lab
```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Hoedjes_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Hoedjes_lmer_pop_qq_plot_residuals.png")
```

#### Schmidt Lab
```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Schmidt_lm_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Schmidt_lm_pop_qq_plot_residuals.png")
```

#### Stamenkovic-Radak Lab
```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_StamenkovicRadak_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_StamenkovicRadak_lmer_pop_qq_plot_residuals.png")
```

#### Zwaan Lab
```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Zwaan_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Viability/by_lab_lmer_residuals/Via_Zwaan_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations

```{r  intersect_Via, echo=FALSE}
intersect_Via_line  <- Reduce(intersect, list(droseu$via$Line[droseu$via$Supervisor.PI == "Gibert"],
                                         droseu$via$Line[droseu$via$Supervisor.PI == "Grath"],
                                         droseu$via$Line[droseu$via$Supervisor.PI == "Hoedjes"],
                                         droseu$via$Line[droseu$via$Supervisor.PI == "Schmidt"],
                                         droseu$via$Line[droseu$via$Supervisor.PI == "StamenkovicRadak"],
                                         droseu$via$Line[droseu$via$Supervisor.PI == "Zwaan"]))

write(intersect_Via_line, file ="Viability/Via_int_line.txt", sep = "\t")

intersect_Via_pop  <- Reduce(intersect, list(droseu$via$Population[droseu$via$Supervisor.PI == "Gibert"],
                                         droseu$via$Population[droseu$via$Supervisor.PI == "Grath"],
                                         droseu$via$via$Population[droseu$via$Supervisor.PI == "Hoedjes"],
                                         droseu$via$Population[droseu$via$Supervisor.PI == "Schmidt"],
                                         droseu$via$Population[droseu$via$Supervisor.PI == "StamenkovicRadak"],
                                         droseu$via$Population[droseu$via$Supervisor.PI == "Zwaan"]))

write(intersect_Via_pop, file ="Viability/Via_int_pop.txt", sep = "\t")

```

There are `r length(intersect_Via_pop) ` populations and `r length(intersect_Via_line) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 


```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/Viability/Via_NA_lab_correlations.png")
```

## Developmental time

```{r include=FALSE}
dir.create(file.path("DevelopmentTime"), showWarnings = FALSE)
```

### Egg-to-pupa developmental time

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

#### Contributors  

Schmidt Lab : Paul Schmidt

#### Data Reformatting

```{r}
str(droseu$dtp) 
```

#### Descriptive Statistics

```{r}
# Note that the trait has been phenotyped only in Schmidt lab and in one batch.
```

Descriptive statistics at the line level :  
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$DT_P, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level: 
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$DT_P, options = list(pageLength = 10))  %>% formatRound(4:11, 3)
```


#### Plots and Linear Models by Lab

```{r min_max_DT_P, echo=TRUE}
min_DT_P <- min(droseu$dtp$DT_EggPupa)
max_DT_P <- max(droseu$dtp$DT_EggPupa)
```

y-axis is scaled by the minimum (`r min_DT_P`) and maximum (`r max_DT_P`) values in the full data set. 

##### Schmidt Lab
```{r}
lmers_anova$DT_P_Schmidt_lmer_pop
```

```{r}
lmers_sum$DT_P_Schmidt_lmer_pop
```

```{r p_DT_P_Schmidt, echo=FALSE, message= FALSE, warning=FALSE}
p_DT_P_Schmidt <- ggplot(data = filter(droseu$dtp, Supervisor.PI=='Schmidt'), aes(x=Population_Lat, y=DT_EggPupa, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_P_Schmidt", x="Population", y = "DT_EggPupa") + ylim(c(min_DT_P, max_DT_P))+   geom_text(data = lmers_tukey$DT_P_Schmidt_lmer_pop$letters, aes(x = Population, label = cld), y = Inf, vjust = 1) + droseu_theme
```

```{r}
p_DT_P_Schmidt
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_P_Schmidt.pdf",width=8, height=5)
p_DT_P_Schmidt
invisible(dev.off())
```

### Egg-to-adult developmental time

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

#### Contributors  

Gibert Lab : Cristina Vieira, Laurence Mouton, Natacha Kremer, Sonia Martinez, Patricia Gibert

Grath Lab : Ingo Müller, Sonja Grath

Hoedjes Lab : Hristina Kostic, Katja Hoedjes

Schmidt Lab : Ozan Kiratli, Yonatan Babore, Liam Forsythe, Paul Schmidt

Stamenkovic-Radak Lab : Marija Savic Veselinovic, Marija Tanaskovic, Aleksandra Patenkovic, Mihailo Jelic, Katarina Eric, Pavle Eric, Slobodan Davidovic, Marina Stamenkovic-Radak

Zwaan Lab : Joost van den Heuvel, Bas Zwaan

#### Data Reformatting


```{r d_DT_A}
str(droseu$dta) 
```

#### Descriptive Statistics

Descriptive statistics at the line level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$DT_A, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$DT_A, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$DT_A, options = list(pageLength = 10))  %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$DT_A, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

#### Plots and Linear Models by Lab

##### Overview 

```{r min_max_DT_A, echo=TRUE}
min_DT_A <- min(droseu$dta$DT_EggAdult)
max_DT_A <- max(droseu$dta$DT_EggAdult)
```
  
y-axis is scaled by the minimum (`r min_DT_A`) and maximum (`r max_DT_A`) values in the full data set. 

```{r p_DT_A_F_all, message=FALSE, warning=FALSE, echo=FALSE}
p_DT_A_F_all <- ggplot(data = filter(droseu$dta, Sex == "F"), aes(x=Population_Lat, y=DT_EggAdult, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_all", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_all
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_all.pdf",width=8, height=5)
p_DT_A_F_all
invisible(dev.off())
```

```{r p_DT_A_M_all, message=FALSE, warning=FALSE, echo=FALSE}
p_DT_A_M_all <- ggplot(data =  filter(droseu$dta, Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_all", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_all
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_all.pdf",width=8, height=5)
p_DT_A_M_all
invisible(dev.off())
```

##### Gibert Lab

```{r}
lmers_anova$DT_A_F_Gibert_lmer_pop
```

```{r}
lmers_sum$DT_A_F_Gibert_lmer_pop
```

```{r}
lmers_anova$DT_A_M_Gibert_lmer_pop
```

```{r}
lmers_sum$DT_A_M_Gibert_lmer_pop
```

```{r p_DT_A_F_Gibert, include=FALSE}
p_DT_A_F_Gibert <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Gibert" & Sex == "F"), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_Gibert", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_Gibert
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_Gibert.pdf",width=8, height=5)
p_DT_A_F_Gibert
invisible(dev.off())
```

```{r p_DT_A_F_pop_Gibert, echo=FALSE}
p_DT_A_F_pop_Gibert <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Gibert" & Sex == "F"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_pop_Gibert", x="Population", y = "DT_EggAdult")+ ylim(c(min_DT_A, max_DT_A))+   geom_text(data = lmers_tukey$DT_A_F_Gibert_lmer_pop$letters, aes(x = Population, label = cld), y=Inf, vjust = 1) + droseu_theme
```

```{r, echo=FALSE}
p_DT_A_F_pop_Gibert
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_pop_Gibert.pdf",width=8, height=5)
p_DT_A_F_pop_Gibert
invisible(dev.off())
```

```{r p_DT_A_M_Gibert, echo=FALSE}
p_DT_A_M_Gibert <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Gibert" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_Gibert", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r, echo=FALSE}
p_DT_A_M_Gibert
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_Gibert.pdf",width=8, height=5)
p_DT_A_M_Gibert
invisible(dev.off())
```

```{r p_DT_A_M_pop_Gibert, include=FALSE}

p_DT_A_M_pop_Gibert <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Gibert" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_pop_Gibert", x="Population", y = "DT_EggAdult")+ ylim(c(min_DT_A, max_DT_A))+  geom_text(data = lmers_tukey$DT_A_M_Gibert_lmer_pop$letters, aes(x = Population, label = cld), y=Inf, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_DT_A_M_pop_Gibert
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_pop_Gibert.pdf",width=8, height=5)
p_DT_A_M_pop_Gibert
invisible(dev.off())
```

##### Grath Lab
```{r}
lmers_anova$DT_A_F_Grath_lmer_pop
```

```{r}
lmers_sum$DT_A_F_Grath_lmer_pop
```

```{r}
lmers_anova$DT_A_M_Grath_lmer_pop
```

```{r}
lmers_sum$DT_A_M_Grath_lmer_pop
```

```{r p_DT_A_F_Grath, include=FALSE}
p_DT_A_F_Grath <- ggplot(data = (subset(d_DT_A_F,Supervisor.PI=='Grath')), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_Grath", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
invisible(dev.off())
```

```{r echo=FALSE}
p_DT_A_F_Grath
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_Grath.pdf",width=8, height=5)
p_DT_A_F_Grath
invisible(dev.off())
```

```{r p_DT_A_F_pop_Grath, include=FALSE}
p_DT_A_F_pop_Grath <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Grath" & Sex == "F"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_pop_Grath", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ geom_text(data = lmers_tukey$DT_A_F_Grath_lmer_pop$letters, aes(x = Population, label = cld), y=Inf, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_pop_Grath
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_pop_Grath.pdf",width=8, height=5)
p_DT_A_F_pop_Grath
invisible(dev.off())
```

```{r p_DT_A_M_Grath, include=FALSE}

p_DT_A_M_Grath <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Grath" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_Grath", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme

```

```{r echo=FALSE}
p_DT_A_M_Grath
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_Grath.pdf",width=8, height=5)
p_DT_A_M_Grath
invisible(dev.off())
```

```{r p_DT_A_M_pop_Grath,  include=FALSE}
p_DT_A_M_pop_Grath <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Grath" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_pop_Grath", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ geom_text(data = lmers_tukey$DT_A_M_Grath_lmer_pop$letters, aes(x = Population, label = cld), y=Inf, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_pop_Grath
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_pop_Grath.pdf",width=8, height=5)
p_DT_A_M_pop_Grath
invisible(dev.off())
```

##### Hoedjes Lab

```{r}
lmers_anova$DT_A_F_Hoedjes_lmer_pop
```

```{r}
lmers_sum$DT_A_F_Hoedjes_lmer_pop
```

```{r}
lmers_anova$DT_A_M_Hoedjes_lmer_pop
```

```{r}
lmers_sum$DT_A_M_Hoedjes_lmer_pop
```

```{r p_DT_A_F_Hoedjes, include=FALSE}
p_DT_A_F_Hoedjes <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Hoedjes" & Sex == "F"), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_Hoedjes", x="Population", y = "DT_EggAdult")+ ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_Hoedjes
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_Hoedjes.pdf",width=8, height=5)
p_DT_A_F_Hoedjes
invisible(dev.off())
```

```{r p_DT_A_F_pop_Hoedjes, include=FALSE}
p_DT_A_F_pop_Hoedjes <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Hoedjes" & Sex == "F"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_DT_A_F_pop_Hoedjes", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ geom_text(data = lmers_tukey$DT_A_F_Hoedjes_lmer_pop$letters, aes(x = Population, label = cld), y = Inf, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_pop_Hoedjes
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_pop_Hoedjes.pdf",width=8, height=5)
p_DT_A_F_pop_Hoedjes
invisible(dev.off())
```

```{r p_DT_A_M_Hoedjes, include=FALSE}
p_DT_A_M_Hoedjes <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Hoedjes" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_Hoedjes", x="Population", y = "DT_EggAdult")+ ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_Hoedjes
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_Hoedjes.pdf",width=8, height=5)
p_DT_A_M_Hoedjes
invisible(dev.off())
```


```{r p_DT_A_M_pop_Hoedjes, include=FALSE}
p_DT_A_M_pop_Hoedjes <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Hoedjes" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_DT_A_M_pop_Hoedjes", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ geom_text(data = lmers_tukey$DT_A_M_Hoedjes_lmer_pop$letters, aes(x = Population, label = cld), y = Inf, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_pop_Hoedjes
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_pop_Hoedjes.pdf",width=8, height=5)
p_DT_A_M_pop_Hoedjes
invisible(dev.off())
```

##### Schmidt Lab

```{r}
lmers_anova$DT_A_F_Schmidt_lmer_pop
```

```{r}
lmers_sum$DT_A_F_Schmidt_lmer_pop
```

```{r}
lmers_anova$DT_A_M_Schmidt_lmer_pop
```

```{r}
lmers_sum$DT_A_M_Schmidt_lmer_pop
```

```{r p_DT_A_F_Schmidt, include=FALSE}
p_DT_A_F_Schmidt <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Schmidt" & Sex == "F"), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_Schmidt", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_Schmidt
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_Schmidt.pdf",width=8, height=5)
p_DT_A_F_Schmidt
invisible(dev.off())
```

```{r p_DT_A_F_pop_Schmidt, include=FALSE}
p_DT_A_F_pop_Schmidt <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Schmidt" & Sex == "F"),  aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_pop_Schmidt", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+geom_text(data = lmers_tukey$DT_A_F_Schmidt_lmer_pop$letters, aes(x = Population, label = cld), y=Inf, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_DT_A_F_pop_Schmidt
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_pop_Schmidt.pdf",width=8, height=5)
p_DT_A_F_pop_Schmidt
invisible(dev.off())
```

```{r p_DT_A_M_Schmidt, include=FALSE}
p_DT_A_M_Schmidt <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Schmidt" & Sex == "M"),  aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_Schmidt", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_Schmidt
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_Schmidt.pdf",width=8, height=5)
p_DT_A_M_Schmidt
invisible(dev.off())
```


```{r p_DT_A_M_pop_Schmidt, include=FALSE}
p_DT_A_M_pop_Schmidt <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Schmidt" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_pop_Schmidt", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+geom_text(data = lmers_tukey$DT_A_M_Schmidt_lmer_pop$letters, aes(x = Population, label = cld), y=max_DT_A, vjust = 1) +  droseu_theme

```

```{r echo=FALSE}
p_DT_A_M_pop_Schmidt
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_pop_Schmidt.pdf",width=8, height=5)
p_DT_A_M_pop_Schmidt
invisible(dev.off())
```

##### Stamenkovic-Radak Lab

```{r}
lmers_anova$DT_A_F_StamenkovicRadak_lmer_pop
```

```{r}
lmers_sum$DT_A_F_StamenkovicRadak_lmer_pop
```

```{r}
lmers_anova$DT_A_M_StamenkovicRadak_lmer_pop
```

```{r}
lmers_sum$DT_A_M_StamenkovicRadak_lmer_pop
```

```{r p_DT_A_F_StamenkovicRadak,  include=FALSE}
p_DT_A_F_StamenkovicRadak <- ggplot(data = filter(droseu$dta,Supervisor.PI=="StamenkovicRadak" & Sex == "F"),  aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_StamenkovicRadak", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_StamenkovicRadak
```

```{r  include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_StamenkovicRadak.pdf",width=8, height=5)
p_DT_A_F_StamenkovicRadak
invisible(dev.off())
```


```{r p_DT_A_F_pop_StamenkovicRadak,  include=FALSE}
p_DT_A_F_pop_StamenkovicRadak <- ggplot(data = filter(droseu$dta,Supervisor.PI=="StamenkovicRadak" & Sex == "F"),  aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_pop_StamenkovicRadak", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ geom_text(data = lmers_tukey$DT_A_F_StamenkovicRadak_lmer_pop$letters, aes(x = Population, label = cld), y=max_DT_A, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_pop_StamenkovicRadak
```

```{r  include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_pop_StamenkovicRadak.pdf",width=8, height=5)
p_DT_A_F_pop_StamenkovicRadak
invisible(dev.off())
```

```{r p_DT_A_M_StamenkovicRadak,  include=FALSE}
p_DT_A_M_StamenkovicRadak <- ggplot(data = filter(droseu$dta,Supervisor.PI=="StamenkovicRadak" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_StamenkovicRadak", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_StamenkovicRadak
```

```{r  include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_StamenkovicRadak.pdf",width=8, height=5)
p_DT_A_M_StamenkovicRadak
invisible(dev.off())
```


```{r p_DT_A_M_pop_StamenkovicRadak,  include=FALSE}
p_DT_A_M_pop_StamenkovicRadak <- ggplot(data = filter(droseu$dta,Supervisor.PI=="StamenkovicRadak" & Sex == "M"),  aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_pop_StamenkovicRadak", x="Population", y = "DT_EggAdult")+ ylim(c(min_DT_A, max_DT_A))+ geom_text(data = lmers_tukey$DT_A_M_StamenkovicRadak_lmer_pop$letters, aes(x = Population, label = cld), y=max_DT_A, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_pop_StamenkovicRadak
```

```{r  include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_pop_StamenkovicRadak.pdf",width=8, height=5)
p_DT_A_M_pop_StamenkovicRadak
invisible(dev.off())
```


##### Zwaan Lab

```{r}
lmers_anova$DT_A_F_Zwaan_lmer_pop
```

```{r}
lmers_sum$DT_A_F_Zwaan_lmer_pop
```

```{r}
lmers_anova$DT_A_M_Zwaan_lmer_pop
```

```{r}
lmers_sum$DT_A_M_Zwaan_lmer_pop
```

```{r p_DT_A_F_Zwaan,  include=FALSE}
p_DT_A_F_Zwaan <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Zwaan" & Sex == "F"),  aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_Zwaan", x="Population", y = "DT_EggAdult")+ ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_Zwaan
```

```{r  include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_Zwaan.pdf",width=8, height=5)
p_DT_A_F_Zwaan
invisible(dev.off())
```

```{r p_DT_A_F_pop_Zwaan,  include=FALSE}
p_DT_A_F_pop_Zwaan <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Zwaan" & Sex == "F"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_F_pop_Zwaan", x="Population", y = "DT_EggAdult")+ ylim(c(min_DT_A, max_DT_A))+geom_text(data = lmers_tukey$DT_A_F_Zwaan_lmer_pop$letters, aes(x = Population, label = cld), y=max_DT_A, vjust = 1) +  droseu_theme
```

```{r echo=FALSE}
p_DT_A_F_pop_Zwaan
```

```{r  include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_F_pop_Zwaan.pdf",width=8, height=5)
p_DT_A_F_pop_Zwaan
invisible(dev.off())
```

```{r p_DT_A_M_Zwaan,  include=FALSE}
p_DT_A_M_Zwaan <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Zwaan" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_Zwaan", x="Population", y = "DT_EggAdult")+ ylim(c(min_DT_A, max_DT_A))+ droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_Zwaan
```

```{r  include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_Zwaan.pdf",width=8, height=5)
p_DT_A_M_Zwaan
invisible(dev.off())
```

```{r p_DT_A_M_pop_Zwaan,  include=FALSE}
p_DT_A_M_pop_Zwaan <- ggplot(data = filter(droseu$dta,Supervisor.PI=="Zwaan" & Sex == "M"), aes(x=Population_Lat, y=DT_EggAdult, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DT_A_M_pop_Zwaan", x="Population", y = "DT_EggAdult") + ylim(c(min_DT_A, max_DT_A))+geom_text(data = lmers_tukey$DT_A_M_Zwaan_lmer_pop$letters, aes(x = Population, label = cld), y=max_DT_A, vjust = 1) +  droseu_theme
```

```{r echo=FALSE}
p_DT_A_M_pop_Zwaan
```

```{r include=FALSE}
pdf(file="DevelopmentTime/p_DT_A_M_pop_Zwaan.pdf",width=8, height=5)
p_DT_A_M_pop_Zwaan
dev.off
```

#### Linear Models / Residuals

##### Gibert Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Gibert_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Gibert_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Gibert_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Gibert_lmer_pop_qq_plot_residuals.png")
```

##### Grath Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Grath_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Grath_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Grath_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Grath_lmer_pop_qq_plot_residuals.png")
```

#### Hoedjes Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Hoedjes_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Hoedjes_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Hoedjes_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Hoedjes_lmer_pop_qq_plot_residuals.png")
```

#### Schmidt Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Schmidt_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Schmidt_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Schmidt_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Schmidt_lmer_pop_qq_plot_residuals.png")
```

#### Stamenkovic-Radak Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_StamenkovicRadak_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_StamenkovicRadak_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_StamenkovicRadak_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_StamenkovicRadak_lmer_pop_qq_plot_residuals.png")
```

#### Zwaan Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Zwaan_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Zwaan_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Zwaan_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/DevelopmentTime/by_lab_lmer_residuals/DT_A_F_Zwaan_lmer_pop_qq_plot_residuals.png")
```

#### Lab correlations

```{r include=FALSE}
intersect_Line_DT_A  <- Reduce(intersect, list(droseu$dta$Line[droseu$dta$Supervisor.PI == "Gibert"],
                                          droseu$dta$Line[droseu$dta$Supervisor.PI == "Grath"],
                                          droseu$dta$Line[droseu$dta$Supervisor.PI == "Hoedjes"],
                                          droseu$dta$Line[droseu$dta$Supervisor.PI == "Schmidt"],
                                          droseu$dta$Line[droseu$dta$Supervisor.PI == "StamenkovicRadak"],
                                          droseu$dta$Line[droseu$dta$Supervisor.PI == "Zwaan"]))
write(intersect_Line_DT_A, file ="DevelopmentTime/intersect_Line_DT_A.txt", sep = "\t")

intersect_Pop_DT_A  <- Reduce(intersect, list(droseu$dta$Population[droseu$dta$Supervisor.PI == "Gibert"],
                                          droseu$dta$Population[droseu$dta$Supervisor.PI == "Grath"],
                                          droseu$dta$Population[droseu$dta$Supervisor.PI == "Hoedjes"],
                                          droseu$dta$Population[droseu$dta$Supervisor.PI == "Schmidt"],
                                          droseu$dta$Population[droseu$dta$Supervisor.PI == "StamenkovicRadak"],
                                          droseu$dta$Population[droseu$dta$Supervisor.PI == "Zwaan"]))
write(intersect_Pop_DT_A, file ="DevelopmentTime/intersect_Pop_DT_A.txt", sep = "\t")

```

There are `r length(intersect_Pop_DT_A) ` populations and `r length(intersect_Line_DT_A) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 

_Females_
```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/DevelopmentTime/DT_A_F_lab_correlations.png")
```

_Males_
```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/DevelopmentTime/DT_A_M_lab_correlations.png")
```


## Dry weight

```{r include=FALSE}
dir.create(file.path("DryWeight"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

### Contributors  

Colinet Lab : Sapho-Lou Marti , Hervé Colinet

Hoedjes Lab : Hristina Kostic, Katja Hoedjes

Onder Lab : Seda Coskun, Senel Selin Senkal, Dogus Can, Banu Sebnem Onder

### Data Reformatting

```{r}
str(droseu$dw)
```

### Descriptive Statistics

Descriptive statistics at the line level, with batch information : 
```{r table_DW_Line_wobatch, echo=FALSE}
DT::datatable(table_line_wbatch$DW, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$DW, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$DW, options = list(pageLength = 10))  %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$DW, options = list(pageLength = 10))  %>% formatRound(5:12, 3)
```

### Plots and Linear Models by Lab

#### Overview 

```{r min_max_DW}
min_DW <- min(droseu$dw$DW_micrograms)
max_DW <- max(droseu$dw$DW_micrograms)
```

y-axis is scaled by the minimum (`r min_DW`) and maximum (`r max_DW`) values in the full data set. 

```{r p_DW_F_all, include=FALSE}
p_DW_F_all <- ggplot(data = d_DW_F, aes(x=Population_Lat, y=DW_micrograms, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_F_all", x="Population", y = "DW_micrograms")+ ylim(c(min_DW, max_DW))+ droseu_theme
```

```{r echo=FALSE}
p_DW_F_all
```

```{r include=FALSE}
pdf(file="DryWeight/p_DW_F_all.pdf",width=8, height=5)
p_DW_F_all
invisible(dev.off())
```


```{r p_DW_M_all, include=FALSE}
p_DW_M_all <- ggplot(data = d_DW_M, aes(x=Population_Lat, y=DW_micrograms, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_M_all", x="Population", y = "DW_micrograms")+ ylim(c(min_DW, max_DW))+ droseu_theme
```


```{r echo=FALSE}
p_DW_M_all
```

```{r include=FALSE}
pdf(file="DryWeight/p_DW_M_all.pdf",width=8, height=5)
p_DW_M_all
invisible(dev.off())
```

#### Colinet Lab

```{r}
lmers_anova$DW_F_Colinet_lmer_pop
```

```{r}
lmers_sum$DW_F_Colinet_lmer_pop
```

```{r}
lmers_anova$DW_M_Colinet_lmer_pop
```

```{r}
lmers_sum$DW_M_Colinet_lmer_pop
```

```{r p_DW_F_Colinet, include=FALSE}
p_DW_F_Colinet <- ggplot(data = filter(droseu$dw,Supervisor.PI=="Colinet" & Sex == "F"), aes(x=Population_Lat, y=DW_micrograms, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_F_Colinet", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+ droseu_theme
```

```{r echo=FALSE}
p_DW_F_Colinet
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_F_Colinet.pdf",width=8, height=5)
p_DW_F_Colinet
invisible(dev.off())
```


```{r p_DW_F_pop_Colinet, include=FALSE}
p_DW_F_pop_Colinet <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Colinet" & Sex == "F"), aes(x=Population_Lat, y=DW_micrograms, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_F_pop_Colinet", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+geom_text(data = lmers_tukey$DW_F_Colinet_lmer_pop$letters, aes(x = Population, label = cld), y = max_DW, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_DW_F_pop_Colinet
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_F_pop_Colinet.pdf",width=8, height=5)
p_DW_F_pop_Colinet
invisible(dev.off())
```

```{r p_DW_M_Colinet, include=FALSE}
p_DW_M_Colinet <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Colinet" & Sex == "M"),aes(x=Population_Lat, y=DW_micrograms, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_M_Colinet", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+ droseu_theme
```

```{r echo=FALSE}
p_DW_M_Colinet
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_M_Colinet.pdf",width=8, height=5)
p_DW_M_Colinet
invisible(dev.off())
```

```{r p_DW_M_pop_Colinet, include=FALSE}
p_DW_M_pop_Colinet <- ggplot(data = filter(droseu$dw,Supervisor.PI=="Colinet" & Sex == "M"), aes(x=Population_Lat, y=DW_micrograms, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_M_pop_Colinet", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+geom_text(data = lmers_tukey$DW_M_Colinet_lmer_pop$letters, aes(x = Population, label = cld), y = max_DW, vjust = 1) +  droseu_theme
```

```{r echo=FALSE}
p_DW_M_pop_Colinet
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_M_pop_Colinet.pdf",width=8, height=5)
p_DW_M_pop_Colinet
invisible(dev.off())
```

#### Hoedjes Lab 

```{r}
lmers_anova$DW_F_Hoedjes_lmer_pop
```

```{r}
lmers_sum$DW_F_Hoedjes_lmer_pop
```

```{r}
lmers_anova$DW_M_Hoedjes_lmer_pop
```

```{r}
lmers_sum$DW_M_Hoedjes_lmer_pop
```

```{r p_DW_F_Hoedjes, include=FALSE}
p_DW_F_Hoedjes <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Hoedjes" & Sex == "F"), aes(x=Population_Lat, y=DW_micrograms, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_F_Hoedjes", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+ droseu_theme
```

```{r echo=FALSE}
p_DW_F_Hoedjes
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_F_Hoedjes.pdf",width=8, height=5)
p_DW_F_Hoedjes
invisible(dev.off())
```


```{r p_DW_F_pop_Hoedjes, include=FALSE}
p_DW_F_pop_Hoedjes <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Hoedjes" & Sex == "F"), aes(x=Population_Lat, y=DW_micrograms, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_F_pop_Hoedjes", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+geom_text(data = lmers_tukey$DW_F_Hoedjes_lmer_pop$letters, aes(x = Population, label = cld), y = max_DW, vjust = 1) +  droseu_theme
```

```{r echo=FALSE}
p_DW_F_pop_Hoedjes
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_F_pop_Hoedjes.pdf",width=8, height=5)
p_DW_F_pop_Hoedjes
invisible(dev.off())
```

```{r p_DW_M_Hoedjes, include=FALSE}
p_DW_M_Hoedjes <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Hoedjes" & Sex == "M"),aes(x=Population_Lat, y=DW_micrograms, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_M_Hoedjes", x="Population", y = "DW_micrograms")+ ylim(c(min_DW, max_DW))+ droseu_theme
```

```{r echo=FALSE}
p_DW_M_Hoedjes
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_M_Hoedjes.pdf",width=8, height=5)
p_DW_M_Hoedjes
invisible(dev.off())
```

```{r p_DW_M_pop_Hoedjes, include=FALSE}
p_DW_M_pop_Hoedjes <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Hoedjes" & Sex == "M"), aes(x=Population_Lat, y=DW_micrograms, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_M_pop_Hoedjes", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+geom_text(data = model_means_DW_F_Hoedjes_cld, aes(x = Population, label = cld), y = max_DW, vjust = 1) +  droseu_theme
```

```{r echo=FALSE}
p_DW_M_pop_Hoedjes
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_M_pop_Hoedjes.pdf",width=8, height=5)
p_DW_M_pop_Hoedjes
invisible(dev.off())
```

#### Onder Lab

```{r}
lmers_anova$DW_F_Colinet_lmer_pop
```

```{r}
lmers_sum$DW_F_Colinet_lmer_pop
```

```{r}
lmers_anova$DW_M_Colinet_lmer_pop
```

```{r}
lmers_sum$DW_M_Colinet_lmer_pop
```


```{r p_DW_F_Onder, include=FALSE}
p_DW_F_Onder <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Onder" & Sex == "F"), aes(x=Population_Lat, y=DW_micrograms, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_F_Onder", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+ droseu_theme
```

```{r echo=FALSE}
p_DW_F_Onder
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_F_Onder.pdf",width=8, height=5)
p_DW_F_Onder
invisible(dev.off())
```


```{r p_DW_F_pop_Onder, include=FALSE}
p_DW_F_pop_Onder <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Onder" & Sex == "F"), aes(x=Population_Lat, y=DW_micrograms, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_F_pop_Onder", x="Population", y = "DW_micrograms") + ylim(c(min_DW, max_DW))+geom_text(data = lmers_tukey$DW_F_Onder_lmer_pop$letters, aes(x = Population, label = cld), y = max_DW, vjust = 1) +  droseu_theme
```

```{r echo=FALSE}
p_DW_F_pop_Onder
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_F_pop_Onder.pdf",width=8, height=5)
p_DW_F_pop_Onder
invisible(dev.off())
```

```{r p_DW_M_Onder, include=FALSE}
p_DW_M_Onder <- ggplot(data = filter(droseu$dw, Supervisor.PI=="Onder" & Sex == "M"), aes(x=Population_Lat, y=DW_micrograms, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_M_Onder", x="Population", y = "DW_micrograms")+ ylim(c(min_DW, max_DW))+ droseu_theme
```

```{r echo=FALSE}
p_DW_M_Onder
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_M_Onder.pdf",width=8, height=5)
p_DW_M_Onder
invisible(dev.off())
```

```{r p_DW_M_pop_Onder, include=FALSE}
p_DW_M_pop_Onder <- ggplot(data =  filter(droseu$dw, Supervisor.PI=="Onder" & Sex == "M"),aes(x=Population_Lat, y=DW_micrograms, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_DW_M_pop_Onder", x="Population", y = "DW_micrograms")+ ylim(c(min_DW, max_DW))+geom_text(data = lmers_tukey$DW_M_Onder_lmer_pop$letters, aes(x = Population, label = cld), y = max_DW, vjust = 1) +  droseu_theme
```

```{r echo=FALSE}
p_DW_M_pop_Onder
```

```{r echo=FALSE}
pdf(file="DryWeight/p_DW_M_pop_Onder.pdf",width=8, height=5)
p_DW_M_pop_Onder
invisible(dev.off())
```

### Linear Models / Residuals

#### Colinet Lab

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_F_Colinet_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_F_Colinet_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_M_Colinet_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_M_Colinet_lmer_pop_qq_plot_residuals.png")
```

#### Hoedjes Lab

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_F_Hoedjes_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_F_Hoedjes_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_M_Hoedjes_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_M_Hoedjes_lmer_pop_qq_plot_residuals.png")
```

#### Onder Lab

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_F_Onder_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_F_Onder_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_M_Onder_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/DryWeight/by_lab_lmer_residuals/DW_M_Onder_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations

```{r intersect_DW, include=FALSE}
intersect_Line_DW  <- Reduce(intersect, list(droseu$dw$Line[droseu$dw$Supervisor.PI == "Colinet"],
                                        droseu$dw$Line[droseu$dw$Supervisor.PI == "Hoedjes"],
                                        droseu$dw$Line[droseu$dw$Supervisor.PI == "Onder"]))

write(intersect_Line_DW, file ="DryWeight/intersect_Line_DW.txt", sep = "\t")

intersect_Pop_DW  <- Reduce(intersect, list(droseu$dw$Population[droseu$dw$Supervisor.PI == "Colinet"],
                                        droseu$dw$Population[droseu$dw$Supervisor.PI == "Hoedjes"],
                                        droseu$dw$Population[droseu$dw$Supervisor.PI == "Onder"]))

write(intersect_Pop_DW, file ="DryWeight/intersect_Pop_DW.txt", sep = "\t")

```

There are `r length(intersect_Line_DW) ` populations and `r length(intersect_Pop_DW) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 

_Females_
```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/DryWeight/DW_F_lab_correlations.png")
```

_Males_
```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/DryWeight/DW_M_lab_correlations.png")
```

```{r echo=FALSE}
for (i in dev.list()[1]:dev.list()[length(dev.list())]) {
   invisible(dev.off())
}
# crashed bc tries to save lots -- need to remove some sometimes
```


## Thorax length

```{r include=FALSE}
dir.create(file.path("ThoraxLength"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

### Contributors  

Kozeretska Lab : Svitlana Serga, Alexandra Protsenko, Oleksandr Maistrenko, Iryna Kozeretska

Posnien Lab : Micael Reis, Lennart Hüper

Ritchie Lab : Megan Mcgunnigle, Nicola  Cook, Teresa Abaurrea, Michael Ritchie

Schmidt Lab : Amy Goldfischer, Paul Schmidt

### Data Reformatting

```{r}
str(droseu$tl)
```

### Descriptive Statistics

Descriptive statistics at the line level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$TL, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$TL, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$TL, options = list(pageLength = 10))  %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$TL, options = list(pageLength = 10))  %>% formatRound(5:12, 3)
```

### Plots and Linear Models by Lab  

#### Overview 

```{r}
min_TL <- min(droseu$tl$TL_micrometers)
max_TL <- max(droseu$tl$TL_micrometers)
```

y-axis is scaled by the minimum (`r min_TL`) and maximum (`r max_TL`) values in the full data set. 

```{r p_TL_F, echo=FALSE}
p_TL_F <- ggplot(data = filter(droseu$tl, Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Supervisor.PI)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_F", x="Population", y = "TL_micrometers")+ ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_F
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F.pdf",width=8, height=5)
p_TL_F 
invisible(dev.off())
```

```{r p_TL_M, echo=FALSE}
p_TL_M <- ggplot(data = filter(droseu$tl, Sex == "M"), aes(x=Population_Lat, y=TL_micrometers, fill=Supervisor.PI)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_M", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_M
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_M.pdf",width=8, height=5)
p_TL_M
invisible(dev.off())
```

#### Kozeretska Lab

```{r}
lmers_anova$TL_F_Kozeretska_lmer_pop
```

```{r}
lmers_sum$TL_F_Kozeretska_lmer_pop
```

```{r}
lmers_anova$TL_M_Kozeretska_lmer_pop
```

```{r}
lmers_sum$TL_M_Kozeretska_lmer_pop
```

```{r p_TL_F_Kozeretska, echo=FALSE}
p_TL_F_Kozeretska <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Kozeretska" & Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_F_Kozeretska", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_F_Kozeretska
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F_Kozeretska.pdf",width=8, height=5)
invisible(dev.off())
```

```{r p_TL_F_pop_Kozeretska, echo=FALSE}
p_TL_F_pop_Kozeretska <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Kozeretska" & Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_F_pop_Kozeretska", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+ geom_text(data = lmers_tukey$TL_F_Kozeretska_lmer_pop$letters, aes(x = Population, label = cld), y = max_TL, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_TL_F_pop_Kozeretska
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F_pop_Kozeretska.pdf",width=8, height=5)
p_TL_F_pop_Kozeretska
invisible(dev.off())
```

```{r p_TL_M_Kozeretska, echo=FALSE}
p_TL_M_Kozeretska <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Kozeretska" & Sex == "M"), aes(x=Population_Lat, y=TL_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_M_Kozeretska", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_M_Kozeretska
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_M_Kozeretska.pdf",width=8, height=5)
p_TL_M_Kozeretska
invisible(dev.off())
```

```{r p_TL_M_pop_Kozeretska, include=FALSE}
p_TL_M_pop_Kozeretska <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Kozeretska" & Sex == "M"), aes(x=Population_Lat, y=TL_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_M_pop_Kozeretska", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+geom_text(data = lmers_tukey$TL_M_Kozeretska_lmer_pop$letters, aes(x = Population, label = cld), y = max_TL, vjust = 1)+ droseu_theme
```

```{r echo=FALSE}
p_TL_M_pop_Kozeretska
```

```{r include=FALSE}
pdf(file="ThoraxLength/p_TL_M_pop_Kozeretska.pdf",width=8, height=5)
p_TL_M_pop_Kozeretska
invisible(dev.off())
```

#### Posnien Lab 

```{r}
lmers_anova$TL_F_Posnien_lmer_pop
```

```{r}
lmers_sum$TL_F_Posnien_lmer_pop
```

```{r}
lmers_anova$TL_M_Posnien_lmer_pop
```

```{r}
lmers_sum$TL_M_Posnien_lmer_pop
```

```{r p_TL_F_Posnien, echo=FALSE}
p_TL_F_Posnien <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Posnien" & Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_F_Posnien", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_F_Posnien
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F_Posnien.pdf",width=8, height=5)
p_TL_F_Posnien
invisible(dev.off())
```

```{r p_TL_F_pop_Posnien, echo=FALSE}
p_TL_F_pop_Posnien <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Posnien" & Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_F_pop_Posnien", x="Population", y = "TL_micrometers")+ ylim(c(min_TL, max_TL))+ geom_text(data = lmers_tukey$TL_F_Posnien_lmer_pop$letters, aes(x = Population, label = cld), y = max_TL, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_TL_F_pop_Posnien
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F_pop_Posnien.pdf",width=8, height=5)
p_TL_F_pop_Posnien
invisible(dev.off())
```

```{r p_TL_M_Posnien, echo=FALSE}
p_TL_M_Posnien <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Posnien" & Sex == "M"), aes(x=Population_Lat, y=TL_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_M_Posnien", x="Population", y = "TL_micrometers")+ ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_M_Posnien
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_M_Posnien.pdf",width=8, height=5)
p_TL_M_Posnien
invisible(dev.off())
```

```{r p_TL_M_pop_Posnien, echo=FALSE}
p_TL_M_pop_Posnien <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Posnien" & Sex == "M"), aes(x=Population_Lat, y=TL_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_M_pop_Posnien", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+ geom_text(data = lmers_tukey$TL_M_Posnien_lmer_pop$letters, aes(x = Population, label = cld), y = max_TL, vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_TL_M_pop_Posnien
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_M_pop_Posnien.pdf",width=8, height=5)
p_TL_M_pop_Posnien
invisible(dev.off())
```

#### Ritchie Lab

```{r}
lmers_anova$TL_F_Ritchie_lmer_pop
```

```{r}
lmers_sum$TL_F_Ritchie_lmer_pop
```

```{r}
lmers_anova$TL_M_Ritchie_lmer_pop
```

```{r}
lmers_sum$TL_M_Ritchie_lmer_pop
```

```{r p_TL_F_Ritchie, echo=FALSE}
p_TL_F_Ritchie <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Ritchie" & Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_F_Ritchie", x="Population", y = "TL_micrometers")+ ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_F_Ritchie
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F_Ritchie.pdf",width=8, height=5)
p_TL_F_Ritchie
invisible(dev.off())
```

```{r p_TL_F_pop_Ritchie, echo=FALSE}
p_TL_F_pop_Ritchie <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Ritchie" & Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_TL_F_pop_Ritchie", x="Population", y = "TL_micrometers") + coord_cartesian(ylim =c(min_TL, max_TL))+ geom_text(data = lmers_tukey$TL_F_Ritchie_lmer_pop$letters, aes(x = Population, label = cld), y = max_TL, vjust = 1)+ droseu_theme
invisible(dev.off())
```

```{r echo=FALSE}
p_TL_F_pop_Ritchie 
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F_pop_Ritchie.pdf",width=8, height=5)
p_TL_F_pop_Ritchie
invisible(dev.off())
```

```{r p_TL_M_Ritchie, echo=FALSE}
p_TL_M_Ritchie <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Ritchie" & Sex == "M"), aes(x=Population_Lat, y=TL_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_M_Ritchie", x="Population", y = "TL_micrometers")+ ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_M_Ritchie
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_M_Ritchie.pdf",width=8, height=5)
p_TL_M_Ritchie
invisible(dev.off())
```

```{r p_TL_M_pop_Ritchie, echo=FALSE}
p_TL_M_pop_Ritchie <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Ritchie" & Sex == "M"), aes(x=Population_Lat, y=TL_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_M_pop_Ritchie", x="Population", y = "TL_micrometers")+ ylim(c(min_TL, max_TL))+ geom_text(data = lmers_tukey$TL_M_Posnien_lmer_pop$letters, aes(x = Population, label = cld), y = max_TL, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_TL_M_pop_Ritchie
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_M_Ritchie.pdf",width=8, height=5)
p_TL_M_pop_Ritchie
invisible(dev.off())
```

#### Schmidt Lab

```{r}
lmers_anova$TL_F_Schmidt_lmer_pop
```

```{r}
lmers_sum$TL_F_Schmidt_lmer_pop
```

```{r p_TL_F_Schmidt, echo=FALSE}
p_TL_F_Schmidt <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Schmidt" & Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_F_Schmidt", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+ droseu_theme
```

```{r echo=FALSE}
p_TL_F_Schmidt
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F_Schmidt.pdf",width=8, height=5)
p_TL_F_Schmidt
invisible(dev.off())
```

```{r p_TL_F_pop_Schmidt, echo=FALSE}
p_TL_F_pop_Schmidt <- ggplot(data = filter(droseu$tl,Supervisor.PI=="Schmidt" & Sex == "F"), aes(x=Population_Lat, y=TL_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_TL_F_pop_Schmidt", x="Population", y = "TL_micrometers") + ylim(c(min_TL, max_TL))+ geom_text(data = lmers_tukey$TL_F_Schmidt_lmer_pop$letters, aes(x = Population, label = cld), y = max_TL, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_TL_F_pop_Schmidt
```

```{r echo=FALSE}
pdf(file="ThoraxLength/p_TL_F_pop_Schmidt.pdf",width=8, height=5)
p_TL_F_pop_Schmidt
invisible(dev.off())
```

### Linear Models / Residuals

#### Kozeretska Lab

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_F_Kozeretska_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_F_Kozeretska_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_M_Kozeretska_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_M_Kozeretska_lmer_pop_qq_plot_residuals.png")
```

#### Posnien Lab

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_F_Posnien_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_F_Posnien_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_M_Posnien_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_M_Posnien_lmer_pop_qq_plot_residuals.png")
```

#### Ritchie Lab

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_F_Ritchie_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_F_Ritchie_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_M_Ritchie_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_M_Ritchie_lmer_pop_qq_plot_residuals.png")
```

#### Schmidt Lab

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_F_Schmidt_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ThoraxLength/by_lab_lmer_residuals/TL_F_Schmidt_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations

```{r intersect_TL, echo=FALSE}
intersect_Line_TL  <- Reduce(intersect, list(droseu$tl$Line[droseu$tl$Supervisor.PI == "Kozeretska"],
                                        droseu$tl$Line[droseu$tl$Supervisor.PI == "Posnien"],
                                        droseu$tl$Line[droseu$tl$Supervisor.PI == "Ritchie"],
                                        droseu$tl$Line[droseu$tl$Supervisor.PI == "Schmidt"]))

write(intersect_Line_TL, file ="ThoraxLength/intersect_Line_TL.txt", sep = "\t")

intersect_Pop_TL  <- Reduce(intersect, list(droseu$tl$Population[droseu$tl$Supervisor.PI == "Kozeretska"],
                                        droseu$tl$Population[droseu$tl$Supervisor.PI == "Posnien"],
                                        droseu$tl$Population[droseu$tl$Supervisor.PI == "Ritchie"],
                                        droseu$tl$Population[droseu$tl$Supervisor.PI == "Schmidt"]))

write(intersect_Pop_TL, file ="ThoraxLength/intersect_Pop_TL.txt", sep = "\t")
```

There are `r length(intersect_Line_TL) ` populations and `r length(intersect_Pop_TL) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 

_Females_
```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/ThoraxLength/TL_F_lab_correlations.png")
```

_Males_
```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/ThoraxLength/TL_M_lab_correlations.png")
```

## Wing area

```{r include=FALSE}
dir.create(file.path("WingArea"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

### Contributors  

Onder Lab : Cansu Aksoy, Ekin Demir, Ezgi Cobanoglu, Banu Sebnem Onder

Posnien Lab : Micael Reis, Lennart Hüper, Nico Posnien

Ritchie Lab : Megan Mcgunnigle, Nicola  Cook, Teresa Abaurrea, Marija Tanaskovic, Michael Ritchie

Stamenkovic-Radak Lab : Marija Savic Veselinovic, Marija Tanaskovic, Aleksandra Patenkovic, Filip Filopovski, Mihailo Jelic, Katarina Eric, Pavle Eric, Slobodan Davidovic, Marina Stamenkovic-Radak

### Data Reformatting

```{r d_WA}
str(droseu$wa)
```

### _Left_

```{r echo=FALSE}
for (i in dev.list()[1]:dev.list()[length(dev.list())]) {
   invisible(dev.off())
}
# crashed bc tries to save lots -- need to remove some sometimes
```


### Descriptive Statistics

Descriptive statistics at the line level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$WA_Left, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$WA_Left, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$WA_Left, options = list(pageLength = 10))  %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$WA_Left, options = list(pageLength = 10))  %>% formatRound(5:12, 3)
```


### Plots and Linear Models by Lab  

#### Overview 

```{r}

min_WAL <- min(droseu$wa$CentroidSizeLeft_micrometers)
max_WAL <- max(droseu$wa$CentroidSizeLeft_micrometers)

```

y-axis is scaled by the minimum (`r min_WAL`) and maximum (`r max_WAL`) values in the full data set. 


```{r p_WA_F_L, echo=FALSE}

p_WA_F_L <- ggplot(data = filter(droseu$wa, Sex == "F"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Supervisor.PI)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_L", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ droseu_theme

```


```{r echo=FALSE}
p_WA_F_L
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_L.pdf",width=8, height=5)
p_WA_F_L
invisible(dev.off())
```

```{r p_WA_M_L, echo=FALSE}

p_WA_M_L <- ggplot(data = filter(droseu$wa, Sex == "M"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Supervisor.PI)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_L", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ droseu_theme

```

```{r echo=FALSE}
p_WA_M_L 
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_L.pdf",width=8, height=5)
p_WA_M_L
invisible(dev.off())
```


#### Onder Lab

```{r}
lmers_anova$WA_L_F_Onder_lmer_pop
```

```{r}
lmers_sum$WA_L_F_Onder_lmer_pop
```

```{r}
lmers_anova$WA_L_M_Onder_lmer_pop
```

```{r}
lmers_sum$WA_L_M_Onder_lmer_pop
```

```{r p_WA_F_L_Onder, echo=FALSE}

p_WA_F_L_Onder <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Onder" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_Onder_L", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ droseu_theme

```

```{r echo=FALSE}
p_WA_F_L_Onder
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_Onder_L.pdf",width=8, height=5)
p_WA_F_L_Onder
invisible(dev.off())
```

```{r p_WA_F_L_pop_Onder, echo=FALSE}

p_WA_F_L_pop_Onder <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Onder" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_L_pop_Onder", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ geom_text(data = lmers_tukey$WA_L_F_Onder_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAL, vjust = 1) +
droseu_theme

```

```{r echo=FALSE}
p_WA_F_L_pop_Onder
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_L_pop_Onder.pdf",width=8, height=5)
p_WA_F_L_pop_Onder
invisible(dev.off())
```



```{r p_WA_M_L_Onder, echo=FALSE}
p_WA_M_L_Onder <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Onder" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_Onder_L", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ droseu_theme

```


```{r echo=FALSE}
p_WA_M_L_Onder
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_Onder_L.pdf",width=8, height=5)
p_WA_M_L_Onder
invisible(dev.off())
```

```{r p_WA_M_L_pop_Onder, echo=FALSE}

p_WA_M_L_pop_Onder <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Onder" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_L_pop_Onder", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ geom_text(data = lmers_tukey$WA_L_F_Onder_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAL, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_WA_M_L_pop_Onder
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_L_pop_Onder.pdf",width=8, height=5)
p_WA_M_L_pop_Onder
invisible(dev.off())
```

#### Posnien Lab 

```{r}
lmers_anova$WA_L_F_Posnien_lmer_pop
```

```{r}
lmers_sum$WA_L_F_Posnien_lmer_pop
```

```{r}
lmers_anova$WA_L_M_Posnien_lmer_pop
```

```{r}
lmers_sum$WA_L_M_Posnien_lmer_pop
```


```{r p_WA_F_L_Posnien, echo=FALSE}

p_WA_F_L_Posnien <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Posnien" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_L_Posnien", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ droseu_theme

```


```{r echo=FALSE}
p_WA_F_L_Posnien
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_L_Posnien.pdf",width=8, height=5)
p_WA_F_L_Posnien
invisible(dev.off())
```


```{r p_WA_F_L_pop_Posnien, echo=FALSE}

p_WA_F_L_pop_Posnien <- ggplot(data =  filter(droseu$wa, Supervisor.PI=="Posnien" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_L_pop_Posnien", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ geom_text(data = lmers_tukey$WA_L_F_Posnien_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAL, vjust = 1) +droseu_theme

```


```{r echo=FALSE}
p_WA_F_L_pop_Posnien
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_L_pop_Posnien.pdf",width=8, height=5)
p_WA_F_L_pop_Posnien
invisible(dev.off())
```

```{r p_WA_M_L_Posnien, echo=FALSE}

p_WA_M_L_Posnien <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Posnien" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_L_Posnien", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ droseu_theme

```

```{r echo=FALSE}
p_WA_M_L_Posnien
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_L_Posnien.pdf",width=8, height=5)
p_WA_M_L_Posnien
invisible(dev.off())
```


```{r p_WA_M_L_pop_Posnien, echo=FALSE}
p_WA_M_L_pop_Posnien <- ggplot(data =  filter(droseu$wa, Supervisor.PI=="Posnien" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_L_pop_Posnien", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ geom_text(data = lmers_tukey$WA_L_M_Posnien_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAL, vjust = 1) +droseu_theme

```


```{r echo=FALSE}
p_WA_M_L_pop_Posnien
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_L_pop_Posnien.pdf",width=8, height=5)
p_WA_M_L_pop_Posnien
invisible(dev.off())
```


#### Ritchie Lab

```{r}
lmers_anova$WA_L_F_Ritchie_lmer_pop
```

```{r}
lmers_sum$WA_L_F_Ritchie_lmer_pop
```

```{r}
lmers_anova$WA_L_M_Ritchie_lmer_pop
```

```{r}
lmers_sum$WA_L_M_Ritchie_lmer_pop
```


```{r p_WA_F_L_Ritchie, echo=FALSE}

p_WA_F_L_Ritchie <- ggplot(data =  filter(droseu$wa, Supervisor.PI=="Ritchie" & Sex == "F"),  aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_L_Ritchie", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ droseu_theme

```

```{r echo=FALSE}
p_WA_F_L_Ritchie
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_L_Ritchie.pdf",width=8, height=5)
p_WA_F_L_Ritchie
invisible(dev.off())
```



```{r p_WA_F_L_pop_Ritchie, echo=FALSE}

p_WA_F_L_pop_Ritchie <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Ritchie" & Sex == "F"),  aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_L_pop_Ritchie", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ geom_text(data = lmers_tukey$WA_L_F_Ritchie_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAL, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_WA_F_L_pop_Ritchie
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_L_pop_Ritchie.pdf",width=8, height=5)
p_WA_F_L_pop_Ritchie
invisible(dev.off())
```



```{r p_WA_M_L_Ritchie, echo=FALSE}

p_WA_M_L_Ritchie <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Ritchie" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_L_Ritchie", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ droseu_theme

```

```{r echo=FALSE}
p_WA_M_L_Ritchie
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_L_Ritchie.pdf",width=8, height=5)
p_WA_M_L_Ritchie
invisible(dev.off())
```

```{r p_WA_M_L_pop_Ritchie, echo=FALSE}
p_WA_M_L_pop_Ritchie <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Ritchie" & Sex == "M"),  aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_L_pop_Ritchie", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ geom_text(data = lmers_tukey$WA_L_M_Ritchie_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAL, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_WA_M_L_pop_Ritchie
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_L_pop_Ritchie.pdf",width=8, height=5)
p_WA_M_L_pop_Ritchie
invisible(dev.off())
```

#### Stamenkovic-Radak Lab

```{r}
lmers_anova$WA_L_F_StamenkovicRadak_lmer_pop
```

```{r}
lmers_sum$WA_L_F_StamenkovicRadak_lmer_pop
```

```{r}
lmers_anova$WA_L_M_StamenkovicRadak_lmer_pop
```

```{r}
lmers_sum$WA_L_M_StamenkovicRadak_lmer_pop
```


```{r p_WA_F_L_StamenkovicRadak, echo=FALSE}

p_WA_F_L_StamenkovicRadak <- ggplot(data = (subset(d_WA_F,Supervisor.PI=='StamenkovicRadak')), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_L_StamenkovicRadak", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ droseu_theme

```


```{r echo=FALSE}
p_WA_F_L_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_L_StamenkovicRadak.pdf",width=8, height=5)
p_WA_F_L_StamenkovicRadak
invisible(dev.off())
```


```{r p_WA_F_L_pop_StamenkovicRadak,echo=FALSE}

p_WA_F_L_pop_StamenkovicRadak <- ggplot(data = filter(droseu$wa, Supervisor.PI=="StamenkovicRadak" & Sex == "F"),  aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_L_pop_StamenkovicRadak", x="Population", y = "CentroidSizeLeft_micrometers")+ ylim(c(min_WAL, max_WAL))+ geom_text(data = model_means_WA_F_L_StamenkovicRadak_cld, aes(x = Population, label = cld), y = max_WAL, vjust = 1) +droseu_theme

```


```{r echo=FALSE}
p_WA_F_L_pop_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_L_pop_StamenkovicRadak.pdf",width=8, height=5)
p_WA_F_L_pop_StamenkovicRadak
invisible(dev.off())
```

```{r p_WA_M_L_StamenkovicRadak, echo=FALSE}
p_WA_M_L_StamenkovicRadak <- ggplot(data = filter(droseu$wa, Supervisor.PI=="StamenkovicRadak" & Sex == "M"),  aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_L_StamenkovicRadak", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ droseu_theme

```

```{r echo=FALSE}
p_WA_M_L_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_L_StamenkovicRadak.pdf",width=8, height=5)
p_WA_M_L_StamenkovicRadak
invisible(dev.off())
```

```{r p_WA_M_L_pop_StamenkovicRadak, echo=FALSE}
p_WA_M_L_pop_StamenkovicRadak <- ggplot(data = filter(droseu$wa, Supervisor.PI=="StamenkovicRadak" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeLeft_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_L_pop_StamenkovicRadak", x="Population", y = "CentroidSizeLeft_micrometers") + ylim(c(min_WAL, max_WAL))+ geom_text(data = lmers_tukey$WA_L_M_StamenkovicRadak_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAL, vjust = 1) +droseu_theme

```


```{r echo=FALSE}
p_WA_M_L_pop_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_L_pop_StamenkovicRadak.pdf",width=8, height=5)
p_WA_M_L_pop_StamenkovicRadak
invisible(dev.off())
```

### _Right_

### Descriptive Statistics

Descriptive statistics at the line level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$WA_Right, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$WA_Right, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$WA_Right, options = list(pageLength = 10))  %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$WA_Right, options = list(pageLength = 10))  %>% formatRound(5:12, 3)
```

### Plots and Linear Models by Lab  

#### Overview 

```{r}

min_WAR <- min(droseu$wa$CentroidSizeRight_micrometers)
max_WAR <- max(droseu$wa$CentroidSizeRight_micrometers)

```


```{r p_WA_F_R, echo=FALSE}

p_WA_F_R <- ggplot(data = filter(droseu$wa, Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Supervisor.PI)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_R", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAL, max_WAL))+ droseu_theme

```

```{r echo=FALSE}
p_WA_F_R
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R.pdf",width=8, height=5)
p_WA_F_R
invisible(dev.off())
```


```{r p_WA_M_R, echo=FALSE}

p_WA_M_R <- ggplot(data = filter(droseu$wa, Sex == "M"),  aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Supervisor.PI)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R", x="Population", y = "CentroidSizeRight_micrometers")+ ylim(c(min_WAL, max_WAL))+ droseu_theme

```


```{r echo=FALSE}
p_WA_M_R
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_R.pdf",width=8, height=5)
p_WA_M_R
invisible(dev.off())
```

#### Onder Lab

```{r include=FALSE}
for (i in dev.list()[1]:dev.list()[length(dev.list())]) {
   invisible(dev.off())
}
# crashed bc tries to save lots -- need to remove some 
```


```{r}
lmers_anova$WA_R_F_Onder_lmer_pop
```

```{r}
lmers_sum$WA_R_F_Onder_lmer_pop
```

```{r}
lmers_anova$WA_R_M_Onder_lmer_pop
```

```{r}
lmers_sum$WA_R_M_Onder_lmer_pop
```


```{r p_WA_F_R_Onder, echo=FALSE}

p_WA_F_R_Onder <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Onder" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_Onder_R", x="Population", y = "CentroidSizeRight_micrometers")+ ylim(c(min_WAR, max_WAR))+ droseu_theme

```


```{r echo=FALSE}
p_WA_F_R_Onder
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R_Onder.pdf",width=8, height=5)
p_WA_F_R_Onder
invisible(dev.off())
```

```{r p_WA_F_R_pop_Onder, echo=FALSE}

p_WA_F_R_pop_Onder <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Onder" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_R_pop_Onder", x="Population", y = "CentroidSizeRight_micrometers") + geom_text(data = lmers_tukey$WA_R_F_Onder_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAR, vjust = 1) +
ylim(c(min_WAR, max_WAR))+ droseu_theme

```


```{r echo=FALSE}
p_WA_F_R_pop_Onder
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R_pop_Onder.pdf",width=8, height=5)
p_WA_F_R_pop_Onder
invisible(dev.off())
```


```{r p_WA_M_R_Onder, echo=FALSE}

p_WA_M_R_Onder <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Onder" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R_Onder", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ droseu_theme

```


```{r echo=FALSE}
p_WA_M_R_Onder
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_R_Onder.pdf",width=8, height=5)
p_WA_M_R_Onder
invisible(dev.off())
```

```{r p_WA_M_R_pop_Onder, echo=FALSE}

p_WA_M_R_pop_Onder <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Onder" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R_pop_Onder", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ geom_text(data = lmers_tukey$WA_R_M_Onder_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAR, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_WA_M_R_pop_Onder
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_R_pop_Onder.pdf",width=8, height=5)
p_WA_M_R_pop_Onder
invisible(dev.off())
```


#### Posnien Lab 


```{r}
lmers_anova$WA_R_F_Posnien_lmer_pop
```

```{r}
lmers_sum$WA_R_F_Posnien_lmer_pop
```

```{r}
lmers_anova$WA_R_M_Posnien_lmer_pop
```

```{r}
lmers_sum$WA_R_M_Posnien_lmer_pop
```

```{r p_WA_F_R_Posnien, echo=FALSE}

p_WA_F_R_Posnien <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Posnien" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_R_Posnien", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ droseu_theme

```


```{r echo=FALSE}
p_WA_F_R_Posnien
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R_Posnien.pdf",width=8, height=5)
p_WA_F_R_Posnien
invisible(dev.off())
```


```{r p_WA_F_R_pop_Posnien, echo=FALSE}

p_WA_F_R_pop_Posnien <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Posnien" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_R_pop_Posnien", x="Population", y = "CentroidSizeRight_micrometers")+ ylim(c(min_WAR, max_WAR))+ geom_text(data = lmers_tukey$WA_R_F_Posnien_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAR, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_WA_F_R_pop_Posnien
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R_pop_Posnien.pdf",width=8, height=5)
p_WA_F_R_pop_Posnien
invisible(dev.off())
```

```{r p_WA_M_R_Posnien, echo=FALSE}

p_WA_M_R_Posnien <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Posnien" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R_Posnien", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ droseu_theme

```

```{r echo=FALSE}
p_WA_M_R_Posnien
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_R_Posnien.pdf",width=8, height=5)
p_WA_M_R_Posnien
invisible(dev.off())
```


```{r p_WA_M_R_pop_Posnien, echo=FALSE}

p_WA_M_R_pop_Posnien <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Posnien" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R_pop_Posnien", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ geom_text(data = lmers_tukey$WA_R_M_Posnien_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAR, vjust = 1) +droseu_theme

```


```{r echo=FALSE}
p_WA_M_R_pop_Posnien
```

```{r echo=FALSE}

pdf(file="WingArea/p_WA_M_R_pop_Posnien.pdf",width=8, height=5)
p_WA_M_R_pop_Posnien
invisible(dev.off())
```


#### Ritchie Lab

```{r}
lmers_anova$WA_R_F_Ritchie_lmer_pop
```

```{r}
lmers_sum$WA_R_F_Ritchie_lmer_pop
```

```{r}
lmers_anova$WA_R_M_Ritchie_lmer_pop
```

```{r}
lmers_sum$WA_R_M_Ritchie_lmer_pop
```

```{r p_WA_F_R_Ritchie, echo=FALSE}
p_WA_F_R_Ritchie <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Ritchie" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_R_Ritchie", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ droseu_theme
```

```{r echo=FALSE}
p_WA_F_R_Ritchie
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R_Ritchie.pdf",width=8, height=5)
p_WA_F_R_Ritchie
invisible(dev.off())
```


```{r p_WA_F_R_pop_Ritchie, echo=FALSE}
p_WA_F_R_pop_Ritchie <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Ritchie" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_R_pop_Ritchie", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ geom_text(data = lmers_tukey$WA_R_F_Ritchie_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAR, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_WA_F_R_pop_Ritchie
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R_pop_Ritchie.pdf",width=8, height=5)
p_WA_F_R_pop_Ritchie
invisible(dev.off())
```

```{r p_WA_M_R_Ritchie, echo=FALSE}
p_WA_M_R_Ritchie <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Ritchie" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R_Ritchie", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ droseu_theme

```


```{r echo=FALSE}
p_WA_M_R_Ritchie
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_R_Ritchie.pdf",width=8, height=5)
p_WA_M_R_Ritchie
invisible(dev.off())
```


```{r p_WA_M_R_pop_Ritchie, echo=FALSE}

p_WA_M_R_pop_Ritchie <- ggplot(data = filter(droseu$wa, Supervisor.PI=="Ritchie" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R_pop_Ritchie", x="Population", y = "CentroidSizeRight_micrometers")+ ylim(c(min_WAR, max_WAR))+ geom_text(data = lmers_tukey$WA_R_M_Ritchie_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAR, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_WA_M_R_pop_Ritchie
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_R_pop_Ritchie.pdf",width=8, height=5)
p_WA_M_R_pop_Ritchie
invisible(dev.off())
```



#### Stamenkovic-Radak Lab

```{r}
lmers_anova$WA_R_F_StamenkovicRadak_lmer_pop
```

```{r}
lmers_sum$WA_R_F_StamenkovicRadak_lmer_pop
```

```{r}
lmers_anova$WA_R_M_StamenkovicRadak_lmer_pop
```

```{r}
lmers_sum$WA_R_M_StamenkovicRadak_lmer_pop
```

```{r p_WA_F_R_StamenkovicRadak, echo=FALSE}

p_WA_F_R_StamenkovicRadak <- ggplot(data = filter(droseu$wa, Supervisor.PI=="StamenkovicRadak" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_R_StamenkovicRadak", x="Population", y = "CentroidSizeRight_micrometers")+ ylim(c(min_WAR, max_WAR))+ droseu_theme

```


```{r echo=FALSE}
p_WA_F_R_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R_StamenkovicRadak.pdf",width=8, height=5)
p_WA_F_R_StamenkovicRadak
invisible(dev.off())
```


```{r p_WA_F_R_pop_StamenkovicRadak, echo=FALSE}

p_WA_F_R_pop_StamenkovicRadak <- ggplot(data = filter(droseu$wa, Supervisor.PI=="StamenkovicRadak" & Sex == "F"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_F_R_pop_StamenkovicRadak", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ geom_text(data = lmers_tukey$WA_R_F_StamenkovicRadak_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAR, vjust = 1) +droseu_theme

```


```{r echo=FALSE}
p_WA_F_R_pop_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_F_R_pop_StamenkovicRadak.pdf",width=8, height=5)
p_WA_F_R_pop_StamenkovicRadak
invisible(dev.off())
```

```{r p_WA_M_R_StamenkovicRadak, echo=FALSE}
p_WA_M_R_StamenkovicRadak <- ggplot(data = filter(droseu$wa, Supervisor.PI=="StamenkovicRadak" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Batch)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R_StamenkovicRadak", x="Population", y = "CentroidSizeRight_micrometers")+ ylim(c(min_WAR, max_WAR)) +droseu_theme

```


```{r echo=FALSE}
p_WA_M_R_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_R_StamenkovicRadak.pdf",width=8, height=5)
p_WA_M_R_StamenkovicRadak
invisible(dev.off())
```

```{r p_WA_M_R_pop_StamenkovicRadak, echo=FALSE}
p_WA_M_R_pop_StamenkovicRadak <- ggplot(data = filter(droseu$wa, Supervisor.PI=="StamenkovicRadak" & Sex == "M"), aes(x=Population_Lat, y=CentroidSizeRight_micrometers, fill=Population)) + 
  geom_boxplot(outlier.shape = NA, notch=FALSE) + 
  labs(title="p_WA_M_R_pop_StamenkovicRadak", x="Population", y = "CentroidSizeRight_micrometers") + ylim(c(min_WAR, max_WAR))+ geom_text(data = lmers_tukey$WA_R_M_StamenkovicRadak_lmer_pop$letters, aes(x = Population, label = cld), y = max_WAR, vjust = 1)+ droseu_theme

```

```{r echo=FALSE}
p_WA_M_R_pop_StamenkovicRadak
```

```{r echo=FALSE}
pdf(file="WingArea/p_WA_M_R_pop_StamenkovicRadak.pdf",width=8, height=5)
p_WA_M_R_pop_StamenkovicRadak
invisible(dev.off())
```

### Linear Models / Residuals (Left)

#### Onder Lab

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_Onder_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_Onder_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_M_Onder_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_M_Onder_lmer_pop_qq_plot_residuals.png")
```

#### Posnien Lab

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_Posnien_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_Posnien_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_M_Posnien_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_M_Posnien_lmer_pop_qq_plot_residuals.png")
```

#### Ritchie Lab

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_Ritchie_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_Ritchie_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_M_Ritchie_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_M_Ritchie_lmer_pop_qq_plot_residuals.png")
```

#### Stamenkovic-Radak Lab

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_StamenkovicRadak_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_StamenkovicRadak_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_M_StamenkovicRadak_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_L_F_StamenkovicRadak_lmer_pop_qq_plot_residuals.png")
```

### Linear Models / Residuals (Right)


#### Onder Lab

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_F_Onder_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_F_Onder_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_M_Onder_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_M_Onder_lmer_pop_qq_plot_residuals.png")
```

#### Posnien Lab

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_F_Posnien_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_F_Posnien_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_M_Posnien_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_M_Posnien_lmer_pop_qq_plot_residuals.png")
```

#### Ritchie Lab

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_F_Ritchie_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_F_Ritchie_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_M_Ritchie_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_M_Ritchie_lmer_pop_qq_plot_residuals.png")
```

#### Stamenkovic-Radak Lab

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_F_StamenkovicRadak_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_F_StamenkovicRadak_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_M_StamenkovicRadak_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/WingArea/by_lab_lmer_residuals/WA_R_M_StamenkovicRadak_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations


```{r intesect_WA, echo=FALSE}
intersect_Line_WA  <- Reduce(intersect, list(droseu$wa$Line[droseu$wa$Supervisor.PI == "Onder"],
                                          droseu$wa$Line[droseu$wa$Supervisor.PI == "Posnien"],
                                          droseu$wa$Line[droseu$wa$Supervisor.PI == "Ritchie"],
                                          droseu$wa$Line[droseu$wa$Supervisor.PI == "StamenkovicRadak"]))
write(intersect_Line_WA, file ="WingArea/intersect_Line_WA.txt", sep = "\t")

intersect_Pop_WA  <- Reduce(intersect, list(droseu$wa$Population[droseu$wa$Supervisor.PI == "Onder"],
                                          droseu$wa$Population[droseu$wa$Supervisor.PI == "Posnien"],
                                          droseu$wa$Population[droseu$wa$Supervisor.PI == "Ritchie"],
                                          droseu$wa$Population[droseu$wa$Supervisor.PI == "StamenkovicRadak"]))
write(intersect_Pop_WA, file ="WingArea/intersect_Pop_WA_F.txt", sep = "\t")

```


There are `r length(intersect_Pop_WA) ` populations and `r length(intersect_Line_WA) ` isofemale lines have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 




### _Left_

```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/WingArea/WA_L_F_lab_correlations.png")
```

```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/WingArea/WA_L_M_lab_correlations.png")
```

### _Right_

```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/WingArea/WA_R_F_lab_correlations.png")
```

```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/WingArea/WA_R_M_lab_correlations.png")
```


```{r echo=FALSE}
for (i in dev.list()[1]:dev.list()[length(dev.list())]) {
   invisible(dev.off())
}
# crashed bc tries to save lots -- need to remove some sometimes
```


## Fecundity


```{r include=FALSE}
dir.create(file.path("Fecundity"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _


```{r}
#Note that the trait has been phenotyped only in females.
```

### Contributors  

Billeter Lab : Xiaocui Wang, Tiphaine Bailly, Mario Mira, Jean-Christophe Billeter

Fricke Lab : Claudia Fricke

### Data Reformatting

```{r d_Fec}
str(droseu$fec)
```

### Descriptive Statistics



Descriptive statistics at the line level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$Fec, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics for fecundity at the line level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$Fec, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$Fec, options = list(pageLength = 10))  %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information : 
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$Fec, options = list(pageLength = 10)) %>% formatRound(4:11, 3)
```


### Plots and Linear Models by Lab  

#### Overview 



```{r echo=TRUE}
min_Fec <- min(droseu$fec$NumberOfAdultsEclosed)
max_Fec <- max(droseu$fec$NumberOfAdultsEclosed)
```

y-axis is scaled by the minimum (`r min_Fec`) and maximum (`r max_Fec`) values in the full data set. 

```{r p_Fec_all, echo=FALSE}

p_Fec_all <- ggplot(data = droseu$fec, aes(x=Population_Lat, y=NumberOfAdultsEclosed, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Fec_all", x="Population", y = "DW_micrograms") + ylim(c(min_Fec, max_Fec))+ droseu_theme

```

```{r echo=FALSE}
p_Fec_all
```

```{r echo=FALSE}
pdf(file="Fecundity/p_Fec_all.pdf",width=8, height=5)
p_Fec_all
invisible(dev.off())
```


#### Billeter Lab


```{r}
lmers_anova$Fec_Billeter_lmer_pop
```

```{r}
lmers_sum$Fec_Billeter_lmer_pop
```

```{r p_Fec_Billeter, echo=FALSE}
p_Fec_Billeter <- ggplot(data = filter(droseu$fec, Supervisor.PI=="Billeter"), aes(x=Population_Lat, y=NumberOfAdultsEclosed, fill=Batch)) +  geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Fec_Billeter", x="Population", y = "NumberOfAdultsEclosed") + ylim(c(min_Fec, max_Fec))+ droseu_theme
```

```{r echo=FALSE}
p_Fec_Billeter
```

```{r echo=FALSE}
pdf(file="Fecundity/p_Fec_Billeter.pdf",width=8, height=5)
p_Fec_Billeter
invisible(dev.off())
```

```{r p_Fec_pop_Billeter, echo=FALSE}
p_Fec_pop_Billeter <- ggplot(data = filter(droseu$fec, Supervisor.PI=="Billeter"), aes(x=Population_Lat, y=NumberOfAdultsEclosed, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Fec_pop_Billeter", x="Population", y = "NumberOfAdultsEclosed") + ylim(c(min_Fec, max_Fec))+geom_text(data = lmers_tukey$Fec_Billeter_lmer_pop$letters, aes(x = Population, label = cld), y = max_Fec, vjust = 1) +  droseu_theme
```

```{r echo=FALSE}
p_Fec_pop_Billeter
```

```{r echo=FALSE}
pdf(file="Fecundity/p_Fec_pop_Billeter.pdf",width=8, height=5)
p_Fec_pop_Billeter
invisible(dev.off())
```


#### Fricke Lab 

```{r}
lmers_anova$Fec_Fricke_lmer_pop
```

```{r}
lmers_sum$Fec_Fricke_lmer_pop
```

```{r p_Fec_Fricke, echo=FALSE, message=FALSE, warning=FALSE}

p_Fec_Fricke <- ggplot(data = filter(droseu$fec, Supervisor.PI=="Fricke"), aes(x=Population_Lat, y=NumberOfAdultsEclosed, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Fec_Fricke", x="Population", y = "NumberOfAdultsEclosed") + ylim(c(min_Fec, max_Fec))+ droseu_theme
```

```{r echo=FALSE}
p_Fec_Fricke
```

```{r echo=FALSE}
pdf(file="Fecundity/p_Fec_Fricke.pdf",width=8, height=5)
p_Fec_Fricke
invisible(dev.off())
```


```{r p_Fec_pop_Fricke, echo=FALSE}

p_Fec_pop_Fricke <- ggplot(data = filter(droseu$fec, Supervisor.PI=="Fricke"), aes(x=Population_Lat, y=NumberOfAdultsEclosed, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_Fec_pop_Fricke", x="Population", y = "NumberOfAdultsEclosed") + ylim(c(min_Fec, max_Fec))+ geom_text(data = lmers_tukey$Fec_Fricke_lmer_pop$letters, aes(x = Population, label = cld), y = max_Fec, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_Fec_pop_Fricke
```

```{r echo=FALSE}
pdf(file="Fecundity/p_Fec_pop_Fricke.pdf",width=8, height=5)
p_Fec_pop_Fricke
invisible(dev.off())
```

### Linear Models / Residuals


#### Billeter Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Fecundity/by_lab_lmer_residuals/Fec_Billeter_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Fecundity/by_lab_lmer_residuals/Fec_Billeter_lmer_pop_qq_plot_residuals.png")
```

#### Fricke Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Fecundity/by_lab_lmer_residuals/Fec_Fricke_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Fecundity/by_lab_lmer_residuals/Fec_Fricke_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations

```{r intersect_Fec, echo=FALSE}
intersect_Line_Fec <- intersect(droseu$fec$Line[droseu$fec$Supervisor.PI == "Billeter"], 
                           droseu$fec$Line[droseu$fec$Supervisor.PI == "Fricke"])
write(intersect_Line_Fec, file ="Fecundity/intersect_Line_Fec.txt", sep = "\t")


intersect_Pop_Fec <- intersect(droseu$fec$Line[droseu$fec$Supervisor.PI == "Billeter"], 
                           droseu$fec$Line[droseu$fec$Supervisor.PI == "Fricke"])
write(intersect_Pop_Fec, file ="Fecundity/intersect_Pop_Fec.txt", sep = "\t")


```

There are `r length(intersect_Pop_Fec) ` populations and `r length(intersect_Line_Fec) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 



```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/Fecundity/Fec_F_lab_correlations.png")
```


## Lifespan

```{r echo=FALSE}
dir.create(file.path("Lifespan"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

```{r echo=FALSE}
for (i in dev.list()[1]:dev.list()[length(dev.list())]) {
   invisible(dev.off())
}
# crashed bc tries to save lots -- need to remove some 
```


### Contributors  

Flatt Lab : Esra Durmaz, Envel Kerdaffrec, Thibault Schowing, Virginie Thieu, Marisa Rodrigues, Thomas Flatt

Parsch Lab : Amanda Glaser-Schmitt, Eliza Argyridou, John Parsch

Pasyukova Lab : Natalia Roshina, Alexander Symonenko, Mikhail Trostnikov, Evgenia Tsybul'ko, Ekaterina Veselkina, Olga Rybina, Elena Pasyukova


### Data Reformatting

```{r d_LS_L}
d_LS_L <- read.csv("MasterSheets_Oct21_git/LSL_MasterSheet_Oct21.csv")
```

```{r}
str(droseu$lsl)
```

```{r}
str(droseu$lsp)
```

```{r}
str(droseu$lsm)
```


### Descriptive Statistics

Descriptive statistics at the line level, with batch information :

```{r, echo=FALSE}
DT::datatable(table_line_wbatch$LS_L, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information :

```{r, echo=FALSE}
DT::datatable(table_line_wobatch$LS_L, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$LS_M, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$LS_M, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```


### Plots and Linear Models by Lab  

#### Overview 


```{r}
min_LS <- min(droseu$lsm$LSM_AgeAtDeath_days)
max_LS <- max(droseu$lsm$LSM_AgeAtDeath_days)
```

y-axis is scaled by the minimum (`r min_LS`) and maximum (`r max_LS`) values in the full data set.

```{r p_LS_F, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_F <- ggplot(data = filter(droseu$lsm, Sex == "F"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Supervisor.PI)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_F", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ droseu_theme

```

```{r echo=FALSE}
p_LS_F
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_F.pdf",width=8, height=5)
p_LS_F
invisible(dev.off())
```


```{r p_LS_M, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_M <- ggplot(data = filter(droseu$lsm, Sex == "M"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Supervisor.PI)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_M", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ droseu_theme

```

```{r echo=FALSE}
p_LS_M
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_M.pdf",width=8, height=5)
p_LS_M
invisible(dev.off())
```


#### Flatt Lab

```{r}
lmers_anova$LS_F_Flatt_lmer_pop
```

```{r}
lmers_sum$LS_F_Flatt_lmer_pop
```

```{r}
lmers_anova$LS_M_Flatt_lmer_pop
```

```{r}
lmers_sum$LS_M_Flatt_lmer_pop
```

```{r p_LS_F_Flatt, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_F_Flatt <- ggplot(data = filter(droseu$lsp, Censor == "0" & Sex == "F" & Supervisor.PI == "Flatt"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_F_Flatt", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ geom_text(data = lmers_tukey$LS_F_Flatt_lmer_pop$letters, aes(x = Population, label = cld), y = max_LS, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_LS_F_Flatt
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_F_Flatt.pdf",width=8, height=5)
p_LS_F_Flatt
invisible(dev.off())
```


```{r p_LS_M_Flatt, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_M_Flatt <- ggplot(data = filter(droseu$lsp, Censor == "0" & Sex == "M" & Supervisor.PI == "Flatt"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_M_Flatt", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ geom_text(data = lmers_tukey$LS_M_Flatt_lmer_pop$letters, aes(x = Population, label = cld), y = max_LS, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_LS_M_Flatt
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_M_Flatt.pdf",width=8, height=5)
p_LS_M_Flatt
invisible(dev.off())
```

#### Parsch Lab 


```{r}
lmers_anova$LS_F_Parsch_lmer_pop
```

```{r}
lmers_sum$LS_F_Parsch_lmer_pop
```

```{r}
lmers_anova$LS_M_Parsch_lmer_pop
```

```{r}
lmers_sum$LS_M_Parsch_lmer_pop
```


```{r p_LS_F_Parsch, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_F_Parsch <- ggplot(data = filter(droseu$lsl, Censor == "0" & Sex == "F" & Supervisor.PI == "Parsch"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_F_Parsch", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ droseu_theme

```

```{r echo=FALSE}
p_LS_F_Parsch
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_F_Parsch.pdf",width=8, height=5)
p_LS_F_Parsch
invisible(dev.off())
```

```{r p_LS_F_pop_Parsch, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_F_pop_Parsch <- ggplot(data =  filter(droseu$lsl, Censor == "0" & Sex == "F" & Supervisor.PI == "Parsch"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_F_pop_Parsch", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ 
geom_text(data = lmers_tukey$LS_F_Parsch_lmer_pop$letters, aes(x = Population, label = cld), y = max_LS, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_LS_F_pop_Parsch
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_F_pop_Parsch.pdf",width=8, height=5)
p_LS_F_pop_Parsch
invisible(dev.off())
```



```{r p_LS_M_Parsch, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_M_Parsch <- ggplot(data =  filter(droseu$lsl, Censor == "0" & Sex == "M" & Supervisor.PI == "Parsch"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_M_Parsch", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ droseu_theme

```

```{r echo=FALSE}
p_LS_M_Parsch
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_M_Parsch.pdf",width=8, height=5)
p_LS_M_Parsch
invisible(dev.off())
```

```{r p_LS_M_pop_Parsch, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_M_pop_Parsch <- ggplot(data =  filter(droseu$lsl, Censor == "0" & Sex == "M" & Supervisor.PI == "Parsch"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_M_pop_Parsch", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ geom_text(data = lmers_tukey$LS_M_Parsch_lmer_pop$letters, aes(x = Population, label = cld), y = max_LS, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_LS_M_pop_Parsch
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_M_pop_Parsch",width=8, height=5)
p_LS_M_pop_Parsch
invisible(dev.off())
```

#### Pasyukova Lab

```{r}
lmers_anova$LS_F_Pasyukova_lmer_pop
```

```{r}
lmers_sum$LS_F_Pasyukova_lmer_pop
```

```{r}
lmers_anova$LS_M_Pasyukova_lmer_pop
```

```{r}
lmers_sum$LS_M_Pasyukova_lmer_pop
```


```{r p_LS_F_Pasyukova, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_F_Pasyukova <- ggplot(data = filter(droseu$lsl, Censor == "0" & Sex == "F" & Supervisor.PI == "Pasyukova"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_F_Pasyukova", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ droseu_theme

```

```{r echo=FALSE}
p_LS_F_Pasyukova
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_F_Pasyukova.pdf",width=8, height=5)
p_LS_F_Pasyukova
invisible(dev.off())
```

```{r p_LS_F_pop_Pasyukova, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_F_pop_Pasyukova <- ggplot(data = filter(droseu$lsl, Censor == "0" & Sex == "F" & Supervisor.PI == "Pasyukova"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_F_pop_Pasyukova", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ 
geom_text(data = lmers_tukey$LS_F_Pasyukova_lmer_pop$letters, aes(x = Population, label = cld), y = max_LS, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_LS_F_pop_Pasyukova
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_F_pop_Pasyukova.pdf",width=8, height=5)
p_LS_F_pop_Pasyukova
invisible(dev.off())
```



```{r p_LS_M_Pasyukova, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_M_Pasyukova <- ggplot(data = filter(droseu$lsl, Censor == "0" & Sex == "M" & Supervisor.PI == "Pasyukova"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Batch)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_M_Pasyukova", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+ droseu_theme

```

```{r echo=FALSE}
p_LS_M_Pasyukova
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_M_Pasyukova.pdf",width=8, height=5)
p_LS_M_Pasyukova
invisible(dev.off())
```

```{r p_LS_M_pop_Pasyukova, echo=FALSE, message=FALSE, warning=FALSE}

p_LS_M_pop_Pasyukova <- ggplot(data = filter(droseu$lsl, Censor == "0" & Sex == "M" & Supervisor.PI == "Pasyukova"), aes(x=Population_Lat, y=LSM_AgeAtDeath_days, fill=Population)) + geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_LS_M_pop_Pasyukova", x="Population", y = "LSM_AgeAtDeath_days") + ylim(c(min_LS, max_LS))+  geom_text(data = lmers_tukey$LS_M_Pasyukova_lmer_pop$letters, aes(x = Population, label = cld), y = max_LS, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_LS_M_pop_Pasyukova
```

```{r echo=FALSE}
pdf(file="Lifespan/p_LS_M_pop_Pasyukova",width=8, height=5)
p_LS_M_pop_Pasyukova
invisible(dev.off())
```

### Survival Analyses

```{r d_LS_M_prop, eval=FALSE, message=FALSE, include=FALSE}
d_LS_prop_pop <- d_LS_M %>%
  filter(Censor != 1) %>%
  rename(AgeAtDeath = LSM_AgeAtDeath_days) %>%
  group_split(Supervisor.PI, Population, Sex) %>%
  lapply(., survProp, cuts = seq(0, 120, 5)) %>%
  bind_rows()
```


#### Flatt Lab

```{r}
knitr::include_graphics("SurvivalAnalyses/Lifespan/p_LS_F_Flatt_survival_curves.pdf")
```

```{r}
knitr::include_graphics("SurvivalAnalyses/Lifespan/p_LS_M_Flatt_survival_curves.pdf")
```

#### Parsch Lab

```{r}
knitr::include_graphics("SurvivalAnalyses/Lifespan/p_LS_F_Parsch_survival_curves.pdf")
```

```{r}
knitr::include_graphics("SurvivalAnalyses/Lifespan/p_LS_M_Parsch_survival_curves.pdf")
```

#### Pasyukova Lab


```{r}
knitr::include_graphics("SurvivalAnalyses/Lifespan/p_LS_F_Pasyukova_survival_curves.pdf")
```


```{r}
knitr::include_graphics("SurvivalAnalyses/Lifespan/p_LS_F_Pasyukova_survival_curves.pdf")
```

##### coxme outputs

```{r}
#to be added here
```

### Linear Models / Residuals

#### Flatt Lab

```{r}
knitr::include_graphics("LinearModelsPop/Lifespan/by_lab_lmer_residuals/LS_F_Flatt_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Lifespan/by_lab_lmer_residuals/LS_F_Flatt_lmer_pop_qq_plot_residuals.png")
```

#### Parsch Lab

```{r}
knitr::include_graphics("LinearModelsPop/Lifespan/by_lab_lmer_residuals/LS_F_Parsch_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Lifespan/by_lab_lmer_residuals/LS_M_Parsch_lmer_pop_hist_residuals.png")
```

#### Pasyukova Lab

```{r}
knitr::include_graphics("LinearModelsPop/Lifespan/by_lab_lmer_residuals/LS_F_Pasyukova_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Lifespan/by_lab_lmer_residuals/LS_M_Pasyukova_lmer_pop_qq_plot_residuals.png")
```


### Lab correlations

```{r intersect_LS, echo=FALSE}
intersect_Line_LS <- intersect(droseu$lsl$Line[droseu$lsl$Supervisor.PI == "Parsch"], 
                           droseu$lsl$Line[droseu$lsl$Supervisor.PI == "Pasyukova"])

write(intersect_Line_LS, file ="Lifespan/intersect_Line_LS", sep = "\t")

intersect_Pop_LS <- Reduce(intersect, list(droseu$lsm$Population[droseu$lsm$Supervisor.PI == "Parsch"],
                              droseu$lsm$Line[droseu$lsm$Supervisor.PI == "Flatt"],
                           droseu$lsm$Population[droseu$lsm$Supervisor.PI == "Pasyukova"]))

write(intersect_Pop_LS, file ="Lifespan/intersect_Pop_LS", sep = "\t")


```

There are `r length(intersect_Pop_LS) ` populations and `r length(intersect_Line_LS) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 


```{r}
knitr::include_graphics("LabCorrelations/Lifespan/LS_F_lab_correlations.png")
```


```{r}
knitr::include_graphics("LabCorrelations/Lifespan/LS_M_lab_correlations.png")
```


## Cold-shock mortality

```{r echo=FALSE}
dir.create(file.path("ColdShock"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

### Contributors  

Gonzalez Lab : Llewellyn Green, Josefa Gonzalez, Miriam Merenciano

Kozeretska Lab : Svitlana Serga, Alexandra Protsenko, Oleksandr Maistrenko, Iryna Kozeretska

Vieira Lab : Jorge Vieira, Cristina P. Vieira, Pedro Duque, Tânia Dias


### Data Reformatting

```{r}
str(droseu$csm)
```

### Descriptive Statistics


Descriptive statistics at the line level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$CSM, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$CSM, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$CSM, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$CSM, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```


### Plots and Linear Models by Lab  

#### Overview 

```{r min_max_CSM, echo=TRUE}
min_CSM <- min(droseu$csm$CSM_PropDead_ED)
max_CSM <- max(droseu$csm$CSM_PropDead_ED)
```

y-axis is scaled by the minimum (`r min_CSM`) and maximum (`r max_CSM`) values in the full data set.


```{r p_CSM_F, echo=FALSE}
p_CSM_F <- ggplot(data = filter(droseu$csm, Sex == "F"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_F", x="Population", y = "Prop_dead") + ylim(c(min_CSM, max_CSM))+ droseu_theme
```

```{r echo=FALSE}
p_CSM_F
```

```{r, echo=FALSE}
pdf(file="ColdShock/p_CSM_F.pdf",width=8, height=5)
p_CSM_F
invisible(dev.off())
```


```{r p_CSM_M, echo=FALSE}
p_CSM_M <- ggplot(data = filter(droseu$csm, Sex == "M"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_M", x="Population", y = "Prop_dead") +  ylim(c(min_CSM, max_CSM))+ droseu_theme
```

```{r echo=FALSE}
p_CSM_M
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_M.pdf",width=8, height=5)
p_CSM_M
invisible(dev.off())
```


#### Gonzalez Lab


```{r}
lmers_anova$CSM_F_Gonzalez_lmer_pop
```

```{r}
lmers_sum$CSM_F_Gonzalez_lmer_pop
```

```{r}
lmers_anova$CSM_M_Gonzalez_lmer_pop
```

```{r}
lmers_sum$CSM_M_Gonzalez_lmer_pop
```

```{r p_CSM_F_Gonzalez, echo=FALSE}
p_CSM_F_Gonzalez <- ggplot(data = filter(droseu$csm, Sex == "F" & Supervisor.PI == "Gonzalez"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_F_Gonzalez", x="Population", y = "CSM (PropSur)") + ylim(c(min_CSM, max_CSM))+ droseu_theme
```

```{r echo=FALSE}
p_CSM_F_Gonzalez
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_F_Gonzalez.pdf",width=8, height=5)
p_CSM_F_Gonzalez
invisible(dev.off())
```


```{r p_CSM_F_pop_Gonzalez, echo=FALSE}
p_CSM_F_pop_Gonzalez <- ggplot(data = filter(droseu$csm, Sex == "F" & Supervisor.PI == "Gonzalez"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_F_pop_Gonzalez", x="Population", y = "CSM (PropSur)") +  ylim(c(min_CSM, max_CSM))+ geom_text(data = lmers_tukey$CSM_F_Gonzalez_lmer_pop$letters, aes(x = Population, label = cld), y = max_CSM, vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_CSM_F_pop_Gonzalez
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_F_pop_Gonzalez.pdf",width=8, height=5)
p_CSM_F_pop_Gonzalez
invisible(dev.off())
```

```{r p_CSM_M_Gonzalez, echo=FALSE}
p_CSM_M_Gonzalez <- ggplot(data = filter(droseu$csm, Sex == "M" & Supervisor.PI == "Gonzalez"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_M_Gonzalez", x="Population", y = "CSM (PropSur)") + ylim(c(min_CSM, max_CSM))+ droseu_theme
```

```{r echo=FALSE}
p_CSM_M_Gonzalez
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_M_Gonzalez.pdf",width=8, height=5)
p_CSM_M_Gonzalez
invisible(dev.off())
```

```{r p_CSM_M_pop_Gonzalez, echo=FALSE}
p_CSM_M_pop_Gonzalez <- ggplot(data = filter(droseu$csm, Sex == "M" & Supervisor.PI == "Gonzalez"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_M_pop_Gonzalez", x="Population", y = "CSM (PropSur)") + ylim(c(min_CSM, max_CSM))+ geom_text(data = lmers_tukey$CSM_M_Gonzalez_lmer_pop$letters, aes(x = Population, label = cld), y = max_CSM, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_CSM_M_pop_Gonzalez
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_M_pop_Gonzalez.pdf",width=8, height=5)
p_CSM_M_pop_Gonzalez
invisible(dev.off())
```

#### Kozeretska Lab 

```{r}
lmers_anova$CSM_F_Kozeretska_lmer_pop
```

```{r}
lmers_sum$CSM_F_Kozeretska_lmer_pop
```

```{r}
lmers_anova$CSM_M_Kozeretska_lmer_pop
```

```{r}
lmers_sum$CSM_M_Kozeretska_lmer_pop
```

```{r p_CSM_F_Kozeretska, echo=FALSE}

p_CSM_F_Kozeretska <- ggplot(data = filter(droseu$csm, Sex == "F" & Supervisor.PI == "Kozeretska"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_F_Kozeretska", x="Population", y = "CSM (PropSur)") + ylim(c(min_CSM, max_CSM))+ droseu_theme

```

```{r echo=FALSE}
p_CSM_F_Kozeretska
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_F_Kozeretska.pdf",width=8, height=5)
p_CSM_F_Kozeretska
invisible(dev.off())
```


```{r p_CSM_F_pop_Kozeretska, echo=FALSE}

p_CSM_F_pop_Kozeretska <- ggplot(data = filter(droseu$csm, Sex == "F" & Supervisor.PI == "Kozeretska"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_F_pop_Kozeretska", x="Population", y = "CSM (PropSur)") +  ylim(c(min_CSM, max_CSM))+ 
geom_text(data = lmers_tukey$CSM_F_Kozeretska_lmer_pop$letters, aes(x = Population, label = cld), y = max_CSM, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_CSM_F_pop_Kozeretska
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_F_pop_Kozeretska.pdf",width=8, height=5)
p_CSM_F_pop_Kozeretska
invisible(dev.off())
```



```{r p_CSM_M_Kozeretska, echo=FALSE}
p_CSM_M_Kozeretska <- ggplot(data = filter(droseu$csm, Sex == "M" & Supervisor.PI == "Kozeretska"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_M_Kozeretska", x="Population", y = "CSM (PropSur)")  +  ylim(c(min_CSM, max_CSM))+ droseu_theme

```

```{r echo=FALSE}
p_CSM_M_Kozeretska
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_M_Kozeretska.pdf",width=8, height=5)
p_CSM_M_Kozeretska
invisible(dev.off())
```


```{r p_CSM_M_pop_Kozeretska, echo=FALSE}
p_CSM_M_pop_Kozeretska <- ggplot(data = filter(droseu$csm, Sex == "M" & Supervisor.PI == "Kozeretska"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_M_pop_Kozeretska", x="Population", y = "CSM (PropSur)")  +  ylim(c(min_CSM, max_CSM))+ geom_text(data = lmers_tukey$CSM_M_Kozeretska_lmer_pop$letters, aes(x = Population, label = cld), y = max_CSM, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_CSM_M_pop_Kozeretska
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_M_pop_Kozeretska.pdf",width=8, height=5)
p_CSM_M_pop_Kozeretska
invisible(dev.off())
```


#### Vieira Lab

```{r}
lmers_anova$CSM_F_Vieira_lmer_pop
```

```{r}
lmers_sum$CSM_F_Vieira_lmer_pop
```

```{r}
lmers_anova$CSM_M_Vieira_lmer_pop
```

```{r}
lmers_sum$CSM_M_Vieira_lmer_pop
```

```{r p_CSM_F_Vieira, echo=FALSE}
p_CSM_F_Vieira <- ggplot(data = filter(droseu$csm, Sex == "F" & Supervisor.PI == "Vieira"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_F_Vieira", x="Population", y = "CSM (PropSur)") +  ylim(c(min_CSM, max_CSM))+ droseu_theme

```

```{r echo=FALSE}
p_CSM_F_Vieira
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_F_Vieira.pdf",width=8, height=5)
p_CSM_F_Vieira
invisible(dev.off())
```


```{r p_CSM_F_pop_Vieira, echo=FALSE}
p_CSM_F_pop_Vieira <- ggplot(data = filter(droseu$csm, Sex == "F" & Supervisor.PI == "Vieira"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_F_pop_Vieira", x="Population", y = "CSM (PropSur)") +  ylim(c(min_CSM, max_CSM))+ geom_text(data = lmers_tukey$CSM_F_Vieira_lmer_pop$letters, aes(x = Population, label = cld), y = max_CSM, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_CSM_F_pop_Vieira
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_F_pop_Vieira.pdf",width=8, height=5)
p_CSM_F_pop_Vieira
invisible(dev.off())
```



```{r p_CSM_M_Vieira, echo=FALSE}
p_CSM_M_Vieira <- ggplot(data = filter(droseu$csm, Sex == "M" & Supervisor.PI == "Vieira"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_M_Vieira", x="Population", y = "CSM (PropSur)") +  ylim(c(min_CSM, max_CSM))+ droseu_theme

```

```{r echo=FALSE}
p_CSM_M_Vieira
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_M_Vieira.pdf",width=8, height=5)
p_CSM_M_Vieira
invisible(dev.off())
```


```{r p_CSM_M_pop_Vieira, echo=FALSE}


p_CSM_M_pop_Vieira <- ggplot(data = filter(droseu$csm, Sex == "M" & Supervisor.PI == "Vieira"), aes(x=Population_Lat, y=CSM_PropDead_ED, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CSM_M_pop_Vieira", x="Population", y = "CSM (PropSur)") +  ylim(c(min_CSM, max_CSM))+ geom_text(data = lmers_tukey$CSM_M_Vieira_lmer_pop$letters, aes(x = Population, label = cld), y = max_CSM, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_CSM_M_pop_Vieira
```

```{r echo=FALSE}
pdf(file="ColdShock/p_CSM_M_pop_Vieira.pdf",width=8, height=5)
p_CSM_M_pop_Vieira
invisible(dev.off())
```

### Linear Models / Residuals

#### Gonzalez Lab
```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_F_Gonzalez_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_F_Gonzalez_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_M_Gonzalez_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_M_Gonzalez_lmer_pop_qq_plot_residuals.png")
```

#### Kozeretska Lab

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_F_Kozeretska_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_F_Kozeretska_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_M_Kozeretska_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_M_Kozeretska_lmer_pop_qq_plot_residuals.png")
```


#### Vieira Lab
```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_F_Vieira_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_F_Vieira_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_M_Vieira_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/ColdShock/by_lab_lmer_residuals/CSM_M_Vieira_lmer_pop_qq_plot_residuals.png")
```


### Lab correlations

```{r intersect_CSM, echo=FALSE}
intersect_Line_CSM  <- Reduce(intersect, list(droseu$csm$Line[droseu$csm$Supervisor.PI == "Gonzalez"],
                                         droseu$csm$Line[droseu$csm$Supervisor.PI == "Vieira"],
                                         droseu$csm$Line[droseu$csm$Supervisor.PI == "Kozeretska"]))
write(intersect_Line_CSM, file ="ColdShock/intersect_Line_CSM.txt", sep = "\t")

intersect_Pop_CSM  <- Reduce(intersect, list(droseu$csm$Population[droseu$csm$Supervisor.PI == "Gonzalez"],
                                         droseu$csm$Population[droseu$csm$Supervisor.PI == "Vieira"],
                                         droseu$csm$Population[droseu$csm$Supervisor.PI == "Kozeretska"]))
write(intersect_Pop_CSM, file ="ColdShock/intersect_Pop_CSM.txt", sep = "\t")
```

There are `r length(intersect_Pop_CSM) ` populations and `r length(intersect_Line_CSM) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 

```{r}
knitr::include_graphics("LabCorrelations/ColdShock/CSM_F_lab_correlations.png")
```

```{r}
knitr::include_graphics("LabCorrelations/ColdShock/CSM_M_lab_correlations.png")
```


```{r echo=FALSE}
for (i in dev.list()[1]:dev.list()[length(dev.list())]) {
   invisible(dev.off())
}
# crashed bc tries to save lots -- need to remove some sometimes
```



## Chill-coma recovery time

```{r include=FALSE}
dir.create(file.path("ChillComa"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _


### Contributors  

Mensh Lab : Florencia Putero, Lucas Kreiman, Julian Mensh

Vieira Lab : Jorge Vieira, Cristina P. Vieira, Pedro Duque, Tânia Dias


### Data Reformatting

```{r}
str(droseu$ccrt)
```

### Descriptive Statistics

Descriptive statistics at the line level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$CCRT, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$CCRT, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$CCRT, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$CCRT, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```



### Plots and Linear Models by Lab  


#### Overview 


```{r min_max_CCRT, echo=TRUE}
min_CCRT <- min(droseu$ccrt$CCRT_seconds)
max_CCRT <- max(droseu$ccrt$CCRT_seconds)
```

y-axis is scaled by the minimum (`r min_CSM`) and maximum (`r max_CSM`) values in the full data set.

#### Mensh Lab 

#### Vieira Lab

```{r}
lmers_anova$CCRT_F_Vieira_lmer_pop
```

```{r}
lmers_sum$CCRT_F_Vieira_lmer_pop
```


```{r}
lmers_anova$CCRT_M_Vieira_lmer_pop
```

```{r}
lmers_sum$CCRT_M_Vieira_lmer_pop
```


```{r p_CCRT_F_Vieira, echo=FALSE}
p_CCRT_F_Vieira <- ggplot(data = filter(droseu$ccrt, Censor == "0" & Supervisor.PI == "Vieira" & Sex == "F"), aes(x=Population_Lat, y=CCRT_seconds, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CCRT_F_Vieira", x="Population", y = "CCRT_seconds") + ylim(c(min_CCRT, max_CCRT))+ droseu_theme

```

```{r echo=FALSE}
p_CCRT_F_Vieira
```

```{r echo=FALSE}
pdf(file="ChillComa/p_CCRT_F_Vieira.pdf",width=8, height=5)
p_CCRT_F_Vieira
invisible(dev.off())
```



```{r p_CCRT_F_pop_Vieira, echo=FALSE}

p_CCRT_F_pop_Vieira <- ggplot(data = filter(droseu$ccrt, Censor == "0" & Supervisor.PI == "Vieira" & Sex == "F"), aes(x=Population_Lat, y=CCRT_seconds, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CCRT_F_pop_Vieira", x="Population", y = "CCRT_seconds")+ ylim(c(min_CCRT, max_CCRT))+ geom_text(data = lmers_tukey$CCRT_F_Vieira_lmer_pop$letters, aes(x = Population, label = cld), y = max_CCRT, vjust = 1) + droseu_theme


```

```{r echo=FALSE}
p_CCRT_F_pop_Vieira
```

```{r echo=FALSE}
pdf(file="ChillComa/p_CCRT_F_pop_Vieira.pdf",width=8, height=5)
p_CCRT_F_pop_Vieira
invisible(dev.off())
```



```{r echo=FALSE}
p_CCRT_M_Vieira <- ggplot(data = filter(droseu$ccrt, Censor == "0" & Supervisor.PI == "Vieira" & Sex == "M"), aes(x=Population_Lat, y=CCRT_seconds, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CCRT_M", x="Population", y = "CCRT_seconds") + ylim(c(min_CCRT, max_CCRT))+ droseu_theme

```

```{r echo=FALSE}
p_CCRT_M_Vieira
```

```{r echo=FALSE}
pdf(file="ChillComa/p_CCRT_M_Vieira.pdf",width=8, height=5)
p_CCRT_M_Vieira
invisible(dev.off())
```


```{r p_CCRT_M_pop_Vieira, echo=FALSE}
p_CCRT_M_pop_Vieira <- ggplot(data = filter(droseu$ccrt, Censor == "0" & Supervisor.PI == "Vieira" & Sex == "M"), aes(x=Population_Lat, y=CCRT_seconds, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_CCRT_M_pop_Vieira", x="Population", y = "CCRT_seconds") + ylim(c(min_CCRT, max_CCRT))+ geom_text(data = lmers_tukey$CCRT_M_Vieira_lmer_pop$letters, aes(x = Population, label = cld), y = max_CCRT, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_CCRT_M_pop_Vieira
```

```{r echo=FALSE}
pdf(file="ChillComa/p_CCRT_M_pop_Vieira.pdf",width=8, height=5)
p_CCRT_M_pop_Vieira
invisible(dev.off())
```



### Survival Analyses

#### Mensh Lab

#### Vieira Lab

```{r, echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/ChillComa/p_CCRT_F_Vieira_survival_curves.pdf")
```

```{r, echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/ChillComa/p_CCRT_M_Vieira_survival_curves.pdf")
```

#### coxme

```{r}
#Please refer to "_SurvivalAnalyses_" folder
```

### Linear Models / Residuals

#### Mensh Lab

#### Vieira Lab
```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/ChillComa/by_lab_lmer_residuals/CCRT_F_Vieira_lmer_pop_hist_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/ChillComa/by_lab_lmer_residuals/CCRT_F_Vieira_lmer_pop_qq_plot_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/ChillComa/by_lab_lmer_residuals/CCRT_M_Vieira_lmer_pop_hist_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/ChillComa/by_lab_lmer_residuals/CCRT_M_Vieira_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations

```{r}
#waiting for Mensh Lab's data 
```


## Heat-shock mortality

```{r include=FALSE}
dir.create(file.path("HeatShock"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _


### Contributors  

Parsch Lab : Eliza Argyridou, Amanda Glaser-Schmitt, John Parsch

Vieira Lab : Jorge Vieira, Cristina P. Vieira, Pedro Duque, Tânia Dias

### Data Reformatting


```{r}
str(droseu$hsm)
```

### Descriptive Statistics

Descriptive statistics at the line level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$HSM, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$HSM, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$HSM, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$HSM, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```


### Plots and Linear Models by Lab  

#### Overview 

```{r}
min_HSM <- min(droseu$hsm$TimeDeath_min)
max_HSM <- max(droseu$hsm$TimeDeath_min)
```

y-axis is scaled by the minimum (`r min_HSM`) and maximum (`r max_HSM`) values in the full data set.

```{r p_HSM_F, echo=FALSE}
p_HSM_F <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "F"), aes(x=Population_Lat, y=TimeDeath_min, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_F", x="Population", y = "TimeDeath(min)")  + ylim(c(min_HSM, max_HSM))+ droseu_theme
```

```{r echo=FALSE}
p_HSM_F
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_F.pdf",width=8, height=5)
p_HSM_F
invisible(dev.off())
```


```{r p_HSM_M, echo=FALSE}

p_HSM_M <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "M"), aes(x=Population_Lat, y=TimeDeath_min, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_M", x="Population", y = "TimeDeath(min)")  + ylim(c(min_HSM, max_HSM))+ droseu_theme
```

```{r echo=FALSE}
p_HSM_M
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_M.pdf",width=8, height=5)
p_HSM_M
invisible(dev.off())
```

#### Parsch Lab

```{r}
lmers_anova$HSM_F_Parsch_lmer_pop
```

```{r}
lmers_sum$HSM_F_Parsch_lmer_pop
```

```{r}
lmers_anova$HSM_M_Parsch_lmer_pop
```

```{r}
lmers_sum$HSM_M_Parsch_lmer_pop
```

```{r p_HSM_F_Parsch, echo=FALSE}
p_HSM_F_Parsch <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "F" & Supervisor.PI == "Parsch"), aes(x=Population_Lat, y=TimeDeath_min, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_F_Parsch", x="Population", y = "TimeDeath_min") + ylim(c(min_HSM, max_HSM))+ droseu_theme
```

```{r echo=FALSE}
p_HSM_F_Parsch
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_F_Parsch.pdf",width=8, height=5)
p_HSM_F_Parsch
invisible(dev.off())
```


```{r p_HSM_F_pop_Parsch, echo=FALSE}
p_HSM_F_pop_Parsch <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "F" & Supervisor.PI == "Parsch"), aes(x=Population_Lat, y=TimeDeath_min, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_F_pop_Parsch", x="Population", y = "TimeDeath_min") + ylim(c(min_HSM, max_HSM))+  geom_text(data = lmers_tukey$HSM_F_Parsch_lmer_pop$letters, aes(x = Population, label = cld), y = max_HSM, vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_HSM_F_pop_Parsch
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_F_pop_Parsch.pdf",width=8, height=5)
p_HSM_F_pop_Parsch
invisible(dev.off())
```


```{r p_HSM_M_Parsch}
p_HSM_M_Parsch <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "M" & Supervisor.PI == "Parsch"), aes(x=Population_Lat, y=TimeDeath_min, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_M_Parsch", x="Population", y = "TimeDeath_min") + ylim(c(min_HSM, max_HSM))+ droseu_theme
```

```{r}
p_HSM_M_Parsch
```

```{r}
pdf(file="HeatShock/p_HSM_M_Parsch.pdf",width=8, height=5)
p_HSM_M_Parsch
invisible(dev.off())
```


```{r p_HSM_M_pop_Parsch, echo=FALSE}
p_HSM_M_pop_Parsch <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "M" & Supervisor.PI == "Parsch"), aes(x=Population_Lat, y=TimeDeath_min, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_M_pop_Parsch", x="Population", y = "TimeDeath_min") + ylim(c(min_HSM, max_HSM))+ geom_text(data = lmers_tukey$HSM_M_Parsch_lmer_pop$letters, aes(x = Population, label = cld), y = max_HSM, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_HSM_M_pop_Parsch
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_M_pop_Parsch.pdf",width=8, height=5)
p_HSM_M_pop_Parsch
invisible(dev.off())
```



#### Vieira Lab 

```{r}
lmers_anova$HSM_F_Vieira_lmer_pop
```

```{r}
lmers_sum$HSM_F_Vieira_lmer_pop
```

```{r}
lmers_anova$HSM_M_Vieira_lmer_pop
```

```{r}
lmers_sum$HSM_M_Vieira_lmer_pop
```


```{r p_HSM_F_Vieira, echo=FALSE}
p_HSM_F_Vieira <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "F" & Supervisor.PI == "Vieira"), aes(x=Population_Lat, y=TimeDeath_min, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_F_Vieira", x="Population", y = "TimeDeath_min") + ylim(c(min_HSM, max_HSM))+ droseu_theme
```

```{r echo=FALSE}
p_HSM_F_Vieira
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_F_Vieira.pdf",width=8, height=5)
p_HSM_F_Vieira
invisible(dev.off())
```


```{r p_HSM_F_pop_Vieira, echo=FALSE}
p_HSM_F_pop_Vieira <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "F" & Supervisor.PI == "Vieira"), aes(x=Population_Lat, y=TimeDeath_min, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_F_pop_Vieira", x="Population", y = "TimeDeath_min") + ylim(c(min_HSM, max_HSM))+  geom_text(data = lmers_tukey$HSM_F_Vieira_lmer_pop$letters, aes(x = Population, label = cld), y = max_HSM, vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_HSM_F_pop_Vieira
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_F_pop_Vieira.pdf",width=8, height=5)
p_HSM_F_pop_Vieira
invisible(dev.off())
```


```{r p_HSM_M_Vieira, echo=FALSE}
p_HSM_M_Vieira <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "M" & Supervisor.PI == "Vieira"), aes(x=Population_Lat, y=TimeDeath_min, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_M_Vieira", x="Population", y = "TimeDeath_min") + ylim(c(min_HSM, max_HSM))+ droseu_theme
```

```{r echo=FALSE}
p_HSM_M_Vieira
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_M_Vieira.pdf",width=8, height=5)
p_HSM_M_Vieira
invisible(dev.off())
```


```{r p_HSM_M_pop_Vieira, echo=FALSE}
p_HSM_M_pop_Vieira <- ggplot(data = filter(droseu$hsm, Censor == "0" & Sex == "M" & Supervisor.PI == "Vieira"), aes(x=Population_Lat, y=TimeDeath_min, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_HSM_M_pop_Vieira", x="Population", y = "TimeDeath_min") + ylim(c(min_HSM, max_HSM))+ geom_text(data = lmers_tukey$HSM_M_Vieira_lmer_pop$letters, aes(x = Population, label = cld), y = max_HSM, vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_HSM_M_pop_Vieira
```

```{r echo=FALSE}
pdf(file="HeatShock/p_HSM_M_pop_Vieira.pdf",width=8, height=5)
p_HSM_M_pop_Vieira
invisible(dev.off())
```


### Survival Analyses

#### Parsch Lab

```{r, echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/HeatShock/p_HSM_F_Parsch_survival_curves.pdf")
```

```{r, echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/HeatShock/p_HSM_M_Parsch_survival_curves.pdf")
```

#### Vieira Lab

```{r, echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/HeatShock/p_HSM_F_Vieira_survival_curves.pdf")
```

```{r, echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/HeatShock/p_HSM_M_Vieira_survival_curves.pdf")
```

#### coxme


```{r}
#Please refer to "_SurvivalAnalyses_" folder
```

### Linear Models / Residuals

#### Parsch Lab

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/HeatShock/by_lab_lmer_residuals/HSM_F_Parsch_lmer_pop_hist_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/HeatShock/by_lab_lmer_residuals/HSM_F_Parsch_lmer_pop_qq_plot_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/HeatShock/by_lab_lmer_residuals/HSM_M_Parsch_lmer_pop_hist_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/HeatShock/by_lab_lmer_residuals/HSM_M_Parsch_lmer_pop_qq_plot_residuals.png")
```


#### Vieria Lab
```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/HeatShock/by_lab_lmer_residuals/HSM_F_Vieira_lmer_pop_hist_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/HeatShock/by_lab_lmer_residuals/HSM_F_Vieira_lmer_pop_qq_plot_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/HeatShock/by_lab_lmer_residuals/HSM_M_Vieira_lmer_pop_hist_residuals.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LinearModelsPop/HeatShock/by_lab_lmer_residuals/HSM_M_Vieira_lmer_pop_qq_plot_residuals.png")
```


### Lab correlations

```{r intersect_HSM, echo=FALSE}

intersect_Line_HSM  <- Reduce(intersect, list(droseu$hsm$Line[droseu$hsm$Supervisor.PI == "Parsch"],
                                         droseu$hsm$Line[droseu$hsm$Supervisor.PI == "Vieira"]))
write(intersect_Line_HSM, file ="HeatShock/intersect_Line_HSM.txt", sep = "\t")

intersect_Pop_HSM  <- Reduce(intersect, list(droseu$hsm$Population[droseu$hsm$Supervisor.PI == "Parsch"],
                                         droseu$hsm$Population[droseu$hsm$Supervisor.PI == "Vieira"]))
write(intersect_Pop_HSM, file ="HeatShock/intersect_Pop_HSM.txt", sep = "\t")

```

There are `r length(intersect_Pop_HSM) ` populations and `r length(intersect_Line_HSM) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 

```{r, echo=FALSE}
knitr::include_graphics("LabCorrelations/HeatShock/HSM_F_lab_correlations.png")
```

```{r, echo=FALSE}
knitr::include_graphics("LabCorrelations/HeatShock/HSM_M_lab_correlations.png")
```


## Diapause


```{r include=FALSE}
dir.create(file.path("Diapause"), showWarnings = FALSE)
```


_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

### Contributors  

Bergland Lab : Liam Miller, Alan Bergland, Priscilla Erickson

Flatt Lab : Esra Durmaz, Envel Kerdaffrec, Thibault Schowing, Virginie Thieu, Marisa Rodrigues, Thomas Flatt

Schlotterer Lab : Manolis Lyrakis, Christian Schlötterer

### Data Reformatting

```{r}
str(droseu$dia)
```


### Descriptive Statistics

Descriptive statistics at the line level for Max Stage 9, with batch information :
```{r echo=FALSE}
DT::datatable(table_line_wbatch$Dia_Prop_Max_Stage9, options = list(pageLength = 10)) %>% formatRound(5, 3)
```

Descriptive statistics at the line level for Max Stage 9, without batch information :
```{r echo=FALSE}
DT::datatable(table_line_wobatch$Dia_Prop_Max_Stage9, options = list(pageLength = 10)) %>% formatRound(4, 3)
```

Descriptive statistics at the population level, with batch information :
```{r echo=FALSE}
DT::datatable(table_pop_wbatch$Dia_Prop_Max_Stage9, options = list(pageLength = 10)) %>% formatRound(4:11, 3)
```

Descriptive statistics at the population level, without batch information :
```{r echo=FALSE}
DT::datatable(table_pop_wobatch$Dia_Prop_Max_Stage9, options = list(pageLength = 10)) %>% formatRound(3:10, 3)
```


### Plots and Linear Models by Lab

#### Bergland Lab
```{r}
lmers_anova$Dia_Bergland_lmer_pop
```

```{r}
lmers_sum$Dia_Bergland_lmer_pop
```

```{r}
min_Dia <- min(d_Dia_Line_wbatch$Max_Stage9_Prop)
max_Dia <- max(d_Dia_Line_wbatch$Max_Stage9_Prop)
```

```{r echo=FALSE}
p_Dia_Pop_Bergland_wbatch <- d_Dia_Line_wbatch %>% 
  filter(Supervisor.PI == 'Bergland') %>%
  ggplot(aes(x = Population_Lat, y = Max_Stage9_Prop, fill = Batch)) + 
  geom_boxplot(outlier.shape = NA, notch = FALSE) +  
  labs(title = "p_Dia_Bergland", x = "Population", y = "Max_Stage9_Prop") +
  droseu_theme +
  ylim(c(min_Dia, max_Dia))
p_Dia_Pop_Bergland_wbatch
```
```{r echo=FALSE}
pdf(file="Diapause/p_Dia_Pop_Bergland_wbatch.pdf",width=8, height=5)
p_Dia_Pop_Bergland_wbatch
invisible(dev.off())
```
```{r echo=FALSE}
p_Dia_Pop_Bergland_wobatch <- d_Dia_Line_wobatch %>% 
  filter(Supervisor.PI == 'Bergland') %>%
  inner_join(model_means_Bergland_cld) %>%
  ggplot(aes(x = Population_Lat, y = Max_Stage9_Prop, fill = Population)) + 
  geom_boxplot(outlier.shape = NA, notch = FALSE) +  
  geom_text(data = model_means_Bergland_cld, aes(x = Population, label = cld), y = max_Dia, vjust = -1) +
  labs(title = "p_Dia_Bergland", x = "Population", y = "Max_Stage9_Prop") +
  droseu_theme +
  ylim(c(min_Dia, max_Dia))
p_Dia_Pop_Bergland_wobatch
```
```{r echo=FALSE}
pdf(file="Diapause/p_Dia_Pop_Bergland_wobatch.pdf",width=8, height=5)
p_Dia_Pop_Bergland_wobatch
invisible(dev.off())
```

#### Flatt Lab 

```{r}
lmers_anova$Dia_Flatt_lm_pop
```

```{r}
lmers_sum$Dia_Flatt_lm_pop
```

```{r echo=FALSE}
p_Dia_Pop_Flatt_wbatch <- d_Dia_Line_wbatch %>% 
  filter(Supervisor.PI == 'Flatt') %>%
  ggplot(aes(x = Population_Lat, y = Max_Stage9_Prop, fill = Batch)) + 
  geom_boxplot(outlier.shape = NA, notch = FALSE) +  
  labs(title = "p_Dia_Flatt", x = "Population", y = "Max_Stage9_Prop") +
  droseu_theme +
  ylim(c(min_Dia, max_Dia))
p_Dia_Pop_Flatt_wbatch
```

```{r  echo=FALSE}
pdf(file="Diapause/p_Dia_Pop_Flatt_wbatch.pdf", width=8, height=5)
p_Dia_Pop_Flatt_wbatch
invisible(dev.off())
```

```{r echo=FALSE}
p_Dia_Pop_Flatt_wobatch <- d_Dia_Line_wobatch %>% 
  filter(Supervisor.PI == 'Flatt') %>%
  ggplot(aes(x = Population_Lat, y = Max_Stage9_Prop, fill = Population)) + 
  geom_boxplot(outlier.shape = NA, notch = FALSE) +  
  geom_text(data = model_means_Flatt_cld, aes(x = Population, label = cld), y = max_Dia, vjust = -1) +
  labs(title = "p_Dia_Flatt", x = "Population", y = "Max_Stage9_Prop") +
  droseu_theme +
  ylim(c(min_Dia, max_Dia))
p_Dia_Pop_Flatt_wobatch
```

```{r  echo=FALSE}
pdf(file="Diapause/p_Dia_Pop_Flatt_wobatch.pdf", width=8, height=5)
p_Dia_Pop_Flatt_wobatch
invisible(dev.off())
```




#### Schlotterer Lab


```{r}
lmers_anova$Dia_Schlotterer_lmer_pop
```

```{r}
lmers_sum$Dia_Schlotterer_lmer_pop
```

```{r echo=FALSE}
model_means_Schlotterer <- emmeans(object = Dia_lmer_Schlotterer, specs = ~ Population)
model_means_Schlotterer_cld <- cld(object = model_means_Schlotterer, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```
```{r echo=FALSE}
p_Dia_Pop_Schlotterer_wbatch <- d_Dia_Line_wbatch %>% 
  filter(Supervisor.PI == 'Schlotterer') %>%
  ggplot(aes(x = Population_Lat, y = Max_Stage9_Prop, fill = Batch)) + 
  geom_boxplot(outlier.shape = NA, notch = FALSE) +  
  labs(title = "p_Dia_Schlotterer", x = "Population", y = "Max_Stage9_Prop") +
  droseu_theme +
  ylim(c(min_Dia, max_Dia))
p_Dia_Pop_Schlotterer_wbatch
```
```{r echo=FALSE}
pdf(file="Diapause/p_Dia_Pop_Schlotterer_wbatch.pdf", width=8, height=5)
p_Dia_Pop_Schlotterer_wbatch
invisible(dev.off())
```
```{r echo=FALSE}
p_Dia_Pop_Schlotterer_wobatch <- d_Dia_Line_wobatch %>% 
  filter(Supervisor.PI == 'Schlotterer') %>%
  ggplot(aes(x = Population_Lat, y = Max_Stage9_Prop, fill = Population)) + 
  geom_boxplot(outlier.shape = NA, notch = FALSE) +  
  geom_text(data = model_means_Schlotterer_cld, aes(x = Population, label = cld), y = max_Dia, vjust = -0.7) +
  labs(title = "p_Dia_Schlotterer", x = "Population", y = "Max_Stage9_Prop") +
  droseu_theme +
  ylim(c(min_Dia, max_Dia))
p_Dia_Pop_Schlotterer_wobatch
```

```{r  echo=FALSE}
pdf(file="Diapause/p_Dia_Pop_Schlotterer_wobatch.pdf", width=8, height=5)
p_Dia_Pop_Schlotterer_wobatch
invisible(dev.off())
```


### Linear Models / Residuals

#### Bergland Lab

```{r}
knitr::include_graphics("LinearModelsPop/Diapause/by_lab_lmer_residuals/Dia_Bergland_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Diapause/by_lab_lmer_residuals/Dia_Bergland_lmer_pop_qq_plot_residuals.png")
```


#### Flatt Lab

```{r}
knitr::include_graphics("LinearModelsPop/Diapause/by_lab_lmer_residuals/Dia_Flatt_lm_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Diapause/by_lab_lmer_residuals/Dia_Flatt_lm_pop_qq_plot_residuals.png")
```

#### Schlotterer Lab

```{r}
knitr::include_graphics("LinearModelsPop/Diapause/by_lab_lmer_residuals/Dia_Schlotterer_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Diapause/by_lab_lmer_residuals/Dia_Schlotterer_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations


```{r intersect_Dia, echo=FALSE}
intersect_Line_Dia  <- Reduce(intersect, list(droseu$dia$Line[droseu$dia$Supervisor.PI == "Flatt"],
                                         droseu$dia$Line[droseu$dia$Supervisor.PI == "Bergland"],
                                         droseu$dia$Line[droseu$dia$Supervisor.PI == "Schlotterer"]))

write(intersect_Line_Dia, file ="Diapause/intersect_Line_Dia.txt", sep = "\t")

intersect_Pop_Dia  <- Reduce(intersect, list(droseu$dia$Population[droseu$dia$Supervisor.PI == "Flatt"],
                                         droseu$dia$Population[droseu$dia$Supervisor.PI == "Bergland"],
                                         droseu$dia$Population[droseu$dia$Supervisor.PI == "Schlotterer"]))

write(intersect_Pop_Dia, file ="Diapause/intersect_Pop_Dia.txt", sep = "\t")

```

There are `r length(intersect_Pop_Dia) ` populations and `r length(intersect_Line_Dia) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 

```{r}
knitr::include_graphics("LabCorrelations/Diapause/Dia_F_lab_correlations.png")
```


## Circadian eclosion timing


```{r include=FALSE}
dir.create(file.path("CircadianEclosion"), showWarnings = FALSE)
```



_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

```{r}
#Note that the trait has been phenotyped as mixed sex. 
```

### Contributors  

Wegener Lab : Susanne Klühspies, Christian Wegener

### Data Reformatting

```{r}
str(droseu$cets)
```

### Descriptive Statistics

#### 18_LD_DD

Descriptive statistics for CET_hours_MESA :
```{r table_CET_18LDDD_CET_MESA, echo=FALSE}
table_CET_18LDDD_CET_MESA <- write.csv(d_CET_18_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                    summarise_at(vars(CET_hours_MESA), 
                                                 list(Mean = mean)), 
                            file = "CircadianEclosion/table_CET_18LDDD_CET_MESA.csv", row.names = T)
table_CET_18LDDD_CET_MESA <- read.csv("CircadianEclosion/table_CET_18LDDD_CET_MESA.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDDD_CET_MESA, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for CET_hours_LSPR :
```{r table_CET_18LDDD_CET_LSPR, echo=FALSE}
table_CET_18LDDD_CET_LSPR <- write.csv(d_CET_18_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                    summarise_at(vars(CET_hours_LSPR), 
                                                 list(Mean = mean)), 
                            file = "CircadianEclosion/table_CET_18LDDD_CET_LSPR.csv", row.names = T)
table_CET_18LDDD_CET_LSPR <- read.csv("CircadianEclosion/table_CET_18LDDD_CET_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDDD_CET_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```



Descriptive statistics for Period_MESA :
```{r table_CET_18LDDD_Period_MESA, echo=FALSE}
table_CET_18LDDD_Period_MESA <- write.csv(d_CET_18_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                    summarise_at(vars(Period_MESA), 
                                                 list(Mean = mean)), 
                            file = "CircadianEclosion/table_CET_18LDDD_Period_MESA.csv", row.names = T)
table_CET_18LDDD_Period_MESA <- read.csv("CircadianEclosion/table_CET_18LDDD_Period_MESA.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDDD_Period_MESA, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```


Descriptive statistics for Period_LSPR :
```{r table_CET_18LDDD_Period_LSPR, echo=FALSE}
table_CET_18LDDD_Period_LSPR <- write.csv(d_CET_18_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                    summarise_at(vars(Period_LSPR), 
                                                 list(Mean = mean)), 
                            file = "CircadianEclosion/table_CET_18LDDD_Period_LSPR.csv", row.names = T)
table_CET_18LDDD_Period_LSPR <- read.csv("CircadianEclosion/table_CET_18LDDD_Period_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDDD_Period_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```


Descriptive statistics for Rhythmicity_LSPR_amplitude :
```{r table_CET_18LDDD_Rhy_LSPR, echo=FALSE}
table_CET_18LDDD_Rhy_LSPR <- write.csv(d_CET_18_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                    summarise_at(vars(Rhythmicity_LSPR_amplitude), 
                                                 list(Mean = mean)), 
                            file = "CircadianEclosion/table_CET_18LDDD_Rhy_LSPR.csv", row.names = T)
table_CET_18LDDD_Rhy_LSPR <- read.csv("CircadianEclosion/table_CET_18LDDD_Rhy_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDDD_Rhy_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Rhythmicity_JTK_p_BH_corrected :
```{r table_CET_18LDDD_Rhy_JTK, echo=FALSE}
table_CET_18LDDD_Rhy_JTK <- write.csv(d_CET_18_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                    summarise_at(vars(Rhythmicity_JTK_p_BH_corrected), 
                                                 list(Mean = mean)), 
                            file = "CircadianEclosion/table_CET_18LDDD_Rhy_JTK.csv", row.names = T)
table_CET_18LDDD_Rhy_JTK <- read.csv("CircadianEclosion/table_CET_18LDDD_Rhy_JTK.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDDD_Rhy_JTK, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```


#### 18_LD_LD

Descriptive statistics for CET_hours_MESA :
```{r table_CET_18LDLD_CET_MESA, echo=FALSE}
table_CET_18LDLD_CET_MESA <- write.csv(d_CET_18_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(CET_hours_MESA), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_18LDLD_CET_MESA.csv", row.names = T)
table_CET_18LDLD_CET_MESA <- read.csv("CircadianEclosion/table_CET_18LDLD_CET_MESA.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDLD_CET_MESA, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for CET_hours_LSPR :
```{r table_CET_18LDLD_CET_LSPR, echo=FALSE}
table_CET_18LDLD_CET_LSPR <- write.csv(d_CET_18_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(CET_hours_LSPR), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_18LDLD_CET_LSPR.csv", row.names = T)
table_CET_18LDLD_CET_LSPR <- read.csv("CircadianEclosion/table_CET_18LDLD_CET_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDLD_CET_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Period_MESA :
```{r table_CET_18LDLD_Period_MESA, echo=FALSE}
table_CET_18LDLD_Period_MESA <- write.csv(d_CET_18_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                            summarise_at(vars(Period_MESA), 
                                                         list(Mean = mean)), 
                                          file = "CircadianEclosion/table_CET_18LDLD_Period_MESA.csv", row.names = T)
table_CET_18LDLD_Period_MESA <- read.csv("CircadianEclosion/table_CET_18LDLD_Period_MESA.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDLD_Period_MESA, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Period_LSPR :
```{r table_CET_18LDLD_Period_LSPR, echo=FALSE}
table_CET_18LDLD_Period_LSPR <- write.csv(d_CET_18_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                            summarise_at(vars(Period_LSPR), 
                                                         list(Mean = mean)), 
                                          file = "CircadianEclosion/table_CET_18LDLD_Period_LSPR.csv", row.names = T)
table_CET_18LDLD_Period_LSPR <- read.csv("CircadianEclosion/table_CET_18LDLD_Period_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDLD_Period_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Rhythmicity_LSPR_amplitude :
```{r table_CET_18LDLD_Rhy_LSPR, echo=FALSE}
table_CET_18LDLD_Rhy_LSPR <- write.csv(d_CET_18_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(Rhythmicity_LSPR_amplitude), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_18LDLD_Rhy_LSPR.csv", row.names = T)
table_CET_18LDLD_Rhy_LSPR <- read.csv("CircadianEclosion/table_CET_18LDLD_Rhy_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_18LDLD_Rhy_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Rhythmicity_JTK_p_BH_corrected :
```{r table_CET_18LDLD_Rhy_JTK, echo=FALSE}
table_CET_18LDLD_Rhy_JTK <- write.csv(d_CET_18_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                        summarise_at(vars(Rhythmicity_JTK_p_BH_corrected), 
                                                     list(Mean = mean)), 
                                      file = "CircadianEclosion/table_CET_18LDLD_Rhy_JTK.csv", row.names = T)
table_CET_18LDLD_Rhy_JTK <- read.csv("CircadianEclosion/table_CET_18LDLD_Rhy_JTK.csv")

```

```{r, echo=FALSE}
DT::datatable(table_CET_18LDLD_Rhy_JTK, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```


#### 29_LD_DD

Descriptive statistics for CET_hours_MESA :
```{r table_CET_29LDDD_CET_MESA, echo=FALSE}
table_CET_29LDDD_CET_MESA <- write.csv(d_CET_29_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(CET_hours_MESA), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_29LDDD_CET_MESA.csv", row.names = T)
table_CET_29LDDD_CET_MESA <- read.csv("CircadianEclosion/table_CET_29LDDD_CET_MESA.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDDD_CET_MESA, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for CET_hours_LSPR :
```{r table_CET_29LDDD_CET_LSPR, echo=FALSE}
table_CET_29LDDD_CET_LSPR <- write.csv(d_CET_29_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(CET_hours_LSPR), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_29LDDD_CET_LSPR.csv", row.names = T)
table_CET_29LDDD_CET_LSPR <- read.csv("CircadianEclosion/table_CET_29LDDD_CET_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDDD_CET_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Period_MESA :
```{r table_CET_29LDDD_Period_MESA, echo=FALSE}
table_CET_29LDDD_Period_MESA <- write.csv(d_CET_29_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                            summarise_at(vars(Period_MESA), 
                                                         list(Mean = mean)), 
                                          file = "CircadianEclosion/table_CET_29LDDD_Period_MESA.csv", row.names = T)
table_CET_29LDDD_Period_MESA <- read.csv("CircadianEclosion/table_CET_29LDDD_Period_MESA.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDDD_Period_MESA, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Period_LSPR :
```{r table_CET_29LDDD_Period_LSPR, echo=FALSE}
table_CET_29LDDD_Period_LSPR <- write.csv(d_CET_29_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                            summarise_at(vars(Period_LSPR), 
                                                         list(Mean = mean)), 
                                          file = "CircadianEclosion/table_CET_29LDDD_Period_LSPR.csv", row.names = T)
table_CET_29LDDD_Period_LSPR <- read.csv("CircadianEclosion/table_CET_29LDDD_Period_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDDD_Period_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Rhythmicity_LSPR_amplitude :
```{r table_CET_29LDDD_Rhy_LSPR, echo=FALSE}
table_CET_29LDDD_Rhy_LSPR <- write.csv(d_CET_29_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(Rhythmicity_LSPR_amplitude), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_29LDDD_Rhy_LSPR.csv", row.names = T)
table_CET_29LDDD_Rhy_LSPR <- read.csv("CircadianEclosion/table_CET_29LDDD_Rhy_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDDD_Rhy_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Rhythmicity_JTK_p_BH_corrected :
```{r table_CET_29LDDD_Rhy_JTK}
table_CET_29LDDD_Rhy_JTK <- write.csv(d_CET_29_LD_DD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                        summarise_at(vars(Rhythmicity_JTK_p_BH_corrected), 
                                                     list(Mean = mean)), 
                                      file = "CircadianEclosion/table_CET_29LDDD_Rhy_JTK.csv", row.names = T)
table_CET_29LDDD_Rhy_JTK <- read.csv("CircadianEclosion/table_CET_29LDDD_Rhy_JTK.csv")

```

```{r}
DT::datatable(table_CET_29LDDD_Rhy_JTK, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

#### 29_LD_LD

Descriptive statistics for CET_hours_MESA :
```{r table_CET_29LDLD_CET_MESA, echo=FALSE}
table_CET_29LDLD_CET_MESA <- write.csv(d_CET_29_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(CET_hours_MESA), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_29LDLD_CET_MESA.csv", row.names = T)
table_CET_29LDLD_CET_MESA <- read.csv("CircadianEclosion/table_CET_29LDLD_CET_MESA.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDLD_CET_MESA, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for CET_hours_LSPR :
```{r table_CET_29LDLD_CET_LSPR, echo=FALSE}
table_CET_29LDLD_CET_LSPR <- write.csv(d_CET_29_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(CET_hours_LSPR), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_29LDLD_CET_LSPR.csv", row.names = T)
table_CET_29LDLD_CET_LSPR <- read.csv("CircadianEclosion/table_CET_29LDLD_CET_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDLD_CET_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Period_MESA :
```{r table_CET_29LDLD_Period_MESA, echo=FALSE}
table_CET_29LDLD_Period_MESA <- write.csv(d_CET_29_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                            summarise_at(vars(Period_MESA), 
                                                         list(Mean = mean)), 
                                          file = "CircadianEclosion/table_CET_29LDLD_Period_MESA.csv", row.names = T)
table_CET_29LDLD_Period_MESA <- read.csv("CircadianEclosion/table_CET_29LDLD_Period_MESA.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDLD_Period_MESA, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Period_LSPR :
```{r table_CET_29LDLD_Period_LSPR, echo=FALSE}
table_CET_29LDLD_Period_LSPR <- write.csv(d_CET_29_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                            summarise_at(vars(Period_LSPR), 
                                                         list(Mean = mean)), 
                                          file = "CircadianEclosion/table_CET_29LDLD_Period_LSPR.csv", row.names = T)
table_CET_29LDLD_Period_LSPR <- read.csv("CircadianEclosion/table_CET_29LDLD_Period_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDLD_Period_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Rhythmicity_LSPR_amplitude :
```{r table_CET_29LDLD_Rhy_LSPR, echo=FALSE}
table_CET_29LDLD_Rhy_LSPR <- write.csv(d_CET_29_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                         summarise_at(vars(Rhythmicity_LSPR_amplitude), 
                                                      list(Mean = mean)), 
                                       file = "CircadianEclosion/table_CET_29LDLD_Rhy_LSPR.csv", row.names = T)
table_CET_29LDLD_Rhy_LSPR <- read.csv("CircadianEclosion/table_CET_29LDLD_Rhy_LSPR.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDLD_Rhy_LSPR, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```

Descriptive statistics for Rhythmicity_JTK_p_BH_corrected :
```{r table_CET_29LDLD_Rhy_JTK, echo=FALSE}
table_CET_29LDLD_Rhy_JTK <- write.csv(d_CET_29_LD_LD %>% group_by(Supervisor.PI, Condition, Population) %>% 
                                        summarise_at(vars(Rhythmicity_JTK_p_BH_corrected), 
                                                     list(Mean = mean)), 
                                      file = "CircadianEclosion/table_CET_29LDLD_Rhy_JTK.csv", row.names = T)
table_CET_29LDLD_Rhy_JTK <- read.csv("CircadianEclosion/table_CET_29LDLD_Rhy_JTK.csv")

```

```{r echo=FALSE}
DT::datatable(table_CET_29LDLD_Rhy_JTK, options = list(pageLength = 10)) %>% formatRound(5:5, 3)
```


### Plots and Linear Models by Lab  

#### Overview 

#### Wegener Lab

From Chris :

```{r echo=FALSE}
knitr::include_graphics("CircadianEclosion/FromChris/Graphs_plate_MESA.png")
```


```{r echo=FALSE}
knitr::include_graphics("CircadianEclosion/FromChris/Graphs_DrosEU_eclosion/Slide1.png")
```


```{r echo=FALSE}
knitr::include_graphics("CircadianEclosion/FromChris/Graphs_DrosEU_eclosion/Slide2.png")
```


```{r echo=FALSE}
knitr::include_graphics("CircadianEclosion/FromChris/Graphs_DrosEU_eclosion/Slide3.png")
```


From Eran :

```{r}
knitr::include_graphics("CircadianEclosion/FromEran/eclosion_data_Christian_Page_1.png")
```

```{r}
knitr::include_graphics("CircadianEclosion/FromEran/eclosion_data_Christian_Page_2.png")
```


## Locomotor activity

```{r include=FALSE}
dir.create(file.path("Locomotor"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _

```{r}
#Note that the trait has been phenotyped only in males.
```


### Contributors  

Tauber Lab : Bettina Fishman, Eran Tauber

### Data Reformatting

```{r d_LA}
d_LA <- read.csv("MasterSheets_Oct21_git/LA_MasterSheet_Oct21.csv")
```


```{r}
str(d_LA)
```


```{r}
d_LA$Supervisor.PI <- as.factor(d_LA$Supervisor.PI)
d_LA$Diet <- as.factor(d_LA$Diet)
d_LA$Batch <- as.factor(d_LA$Batch)
d_LA$Population_Lat <- factor(d_LA$Population, levels= c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))
d_LA$Population_Lon <- factor(d_LA$Population, levels= c("RE","GI","KA","MU","MA","AK","UM","YE","VA"))
d_LA$Population_Alt <- factor(d_LA$Population, levels= c("KA","AK","GI","RE","UM","VA","MU","MA","YE"))
d_LA$Line <- as.factor(d_LA$Line)
d_LA$ReplicateChamber <- as.factor(d_LA$ReplicateChamber)
d_LA$Period <- as.numeric(d_LA$Period)
d_LA$CircPhase <- as.numeric(d_LA$CircPhase)
d_LA$AbsPhase <- as.numeric(d_LA$AbsPhase)
d_LA$Experiment <- as.numeric(d_LA$Experiment)
d_LA$ND <- as.numeric(d_LA$ND)
d_LA$Activity <- as.numeric(d_LA$Activity)
```


```{r}
str(d_LA)
```

### Descriptive Statistics

#### LA - Activity

Descriptive statistics at the line level, with batch information :
```{r table_LA_Act_Line_wbatch, echo=FALSE}
table_LA_Act_Line_wbatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Batch, Sex, Population, Line) %>% 
                                    summarise_at(vars(Activity), 
                                                 list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                      SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                            file = "Locomotor/table_LA_Act_Line_wbatch.csv", row.names = T)
table_LA_Act_Line_wbatch <- read.csv("Locomotor/table_LA_Act_Line_wbatch.csv")

```

```{r, echo=FALSE}
DT::datatable(table_LA_Act_Line_wbatch, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information :
```{r table_LA_Act_Line_wobatch, echo=FALSE}
table_LA_Act_Line_wobatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Sex, Population, Line) %>% 
                                    summarise_at(vars(Activity), 
                                                 list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                      SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                            file = "Locomotor/table_LA_Act_Line_wobatch.csv", row.names = T)
table_LA_Act_Line_wobatch <- read.csv("Locomotor/table_LA_Act_Line_wobatch.csv")

```

```{r, echo=FALSE}
DT::datatable(table_LA_Act_Line_wobatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information :
```{r table_LA_Act_Pop_wbatch, echo=FALSE}
table_LA_Act_Pop_wbatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Batch, Sex, Population) %>% 
                                   summarise_at(vars(Activity), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Locomotor/table_LA_Act_Pop_wbatch.csv", row.names = T)
table_LA_Act_Pop_wbatch <- read.csv("Locomotor/table_LA_Act_Pop_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_LA_Act_Pop_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information :
```{r table_LA_Act_Pop_wobatch, echo=FALSE}
table_LA_Act_Pop_wobatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Sex, Population) %>% 
                                   summarise_at(vars(Activity), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Locomotor/table_LA_Act_Pop_wobatch.csv", row.names = T)
table_LA_Act_Pop_wobatch <- read.csv("Locomotor/table_LA_Act_Pop_wobatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_LA_Act_Pop_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

#### LA - ND ratio

Descriptive statistics at the line level, with batch information :
```{r table_LA_ND_Line_wbatch, echo=FALSE}
table_LA_ND_Line_wbatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Batch, Sex, Population, Line) %>% 
                                        summarise_at(vars(ND), 
                                                     list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                          SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                                      file = "Locomotor/table_LA_ND_Line_wbatch.csv", row.names = T)
table_LA_ND_Line_wbatch <- read.csv("Locomotor/table_LA_ND_Line_wbatch.csv")

```

```{r, echo=FALSE}
DT::datatable(table_LA_ND_Line_wbatch, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information :
```{r table_LA_ND_Line_wobatch, echo=FALSE}
table_LA_ND_Line_wobatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Sex, Population, Line) %>% 
                                         summarise_at(vars(ND), 
                                                      list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                           SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                                       file = "Locomotor/table_LA_ND_Line_wobatch.csv", row.names = T)
table_LA_ND_Line_wobatch <- read.csv("Locomotor/table_LA_ND_Line_wobatch.csv")

```

```{r, echo=FALSE}
DT::datatable(table_LA_ND_Line_wobatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information :
```{r table_LA_ND_Pop_wbatch, echo=FALSE}
table_LA_ND_Pop_wbatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Batch, Sex, Population) %>% 
                                       summarise_at(vars(ND), 
                                                    list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                         SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                                     file = "Locomotor/table_LA_ND_Pop_wbatch.csv", row.names = T)
table_LA_ND_Pop_wbatch <- read.csv("Locomotor/table_LA_ND_Pop_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_LA_ND_Pop_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information :
```{r table_LA_ND_Pop_wobatch}
table_LA_ND_Pop_wobatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Sex, Population) %>% 
                                        summarise_at(vars(ND), 
                                                     list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                          SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                                      file = "Locomotor/table_LA_ND_Pop_wobatch.csv", row.names = T)
table_LA_ND_Pop_wobatch <- read.csv("Locomotor/table_LA_ND_Pop_wobatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_LA_ND_Pop_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```


#### LA - Circadian Phase

Descriptive statistics at the line level, with batch information :
```{r table_LA_CircPhase_Line_wbatch, echo=FALSE}
table_LA_CircPhase_Line_wbatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Batch, Sex, Population, Line) %>% 
                                        summarise_at(vars(CircPhase), 
                                                     list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                          SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                                      file = "Locomotor/table_LA_CircPhase_Line_wbatch.csv", row.names = T)
table_LA_CircPhase_Line_wbatch <- read.csv("Locomotor/table_LA_CircPhase_Line_wbatch.csv")

```

```{r, echo=FALSE}
DT::datatable(table_LA_CircPhase_Line_wbatch, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information :
```{r table_LA_CircPhase_Line_wobatch, echo=FALSE}
table_LA_CircPhase_Line_wobatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Sex, Population, Line) %>% 
                                         summarise_at(vars(CircPhase), 
                                                      list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                           SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                                       file = "Locomotor/table_LA_CircPhase_Line_wobatch.csv", row.names = T)
table_LA_CircPhase_Line_wobatch <- read.csv("Locomotor/table_LA_CircPhase_Line_wobatch.csv")

```

```{r, echo=FALSE}
DT::datatable(table_LA_CircPhase_Line_wobatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information :
```{r table_LA_CircPhase_Pop_wbatch, echo=FALSE}
table_LA_CircPhase_Pop_wbatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Batch, Sex, Population) %>% 
                                       summarise_at(vars(CircPhase), 
                                                    list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                         SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                                     file = "Locomotor/table_LA_CircPhase_Pop_wbatch.csv", row.names = T)
table_LA_CircPhase_Pop_wbatch <- read.csv("Locomotor/table_LA_CircPhase_Pop_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_LA_CircPhase_Pop_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information :
```{r table_LA_CircPhase_Pop_wobatch, echo=FALSE}
table_LA_CircPhase_Pop_wobatch <- write.csv(d_LA %>% group_by(Supervisor.PI, Sex, Population) %>% 
                                        summarise_at(vars(CircPhase), 
                                                     list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                          SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                                      file = "Locomotor/table_LA_CircPhase_Pop_wobatch.csv", row.names = T)
table_LA_CircPhase_Pop_wobatch <- read.csv("Locomotor/table_LA_CircPhase_Pop_wobatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_LA_CircPhase_Pop_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```


### Plots and Linear Models by Lab  

#### Overview 

#### Tauber Lab

```{r, echo=FALSE}
d_LA_trans <- d_LA %>% mutate(ND = log2(ND))
```

```{r LA_lmer_ND_Tauber, echo=FALSE} 
LA_lmer_ND_Tauber <- lmer(ND ~ Population  + (1|Line:Population) +
                          (1|Batch), data = d_LA)
capture.output(summary(LA_lmer_ND_Tauber),file = "Locomotor/LA_lmer_ND_Tauber_sum.txt")
capture.output(anova(LA_lmer_ND_Tauber),file = "Locomotor/LA_lmer_ND_Tauber.txt")
capture.output(emmeans(LA_lmer_ND_Tauber, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Locomotor/LA_lmer_ND_Tauber_tk.txt")
```

```{r}
anova(LA_lmer_ND_Tauber)
```

```{r}
summary(LA_lmer_ND_Tauber)
```


_CircPhase_

```{r LA_lmer_CircPhase_Tauber, echo=FALSE}
LA_lmer_CircPhase_Tauber <- lmer(CircPhase ~ Population  + (1|Line:Population) +
                          (1|Batch) , data = (d_LA))
capture.output(summary(LA_lmer_CircPhase_Tauber),file = "Locomotor/LA_lmer_CircPhase_Tauber_sum.txt")
capture.output(anova(LA_lmer_CircPhase_Tauber),file = "Locomotor/LA_lmer_CircPhase_Tauber.txt")
capture.output(emmeans(LA_lmer_CircPhase_Tauber, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Locomotor/LA_lmer_CircPhase_Tauber_tk.txt")
```

```{r}
anova(LA_lmer_CircPhase_Tauber)
```

```{r}
summary(LA_lmer_CircPhase_Tauber)
```


_Activity_

```{r LA_lmer_Activity_Tauber, echo=FALSE}
LA_lmer_Activity_Tauber <- lmer(Activity ~ Population  + (1|Line:Population) +
                          (1|Batch), data = (d_LA))
capture.output(summary(LA_lmer_Activity_Tauber),file = "Locomotor/LA_lmer_Activity_Tauber_sum.txt")
capture.output(anova(LA_lmer_Activity_Tauber),file = "Locomotor/LA_lmer_Activity_Tauber.txt")
capture.output(emmeans(LA_lmer_Activity_Tauber, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Locomotor/LA_lmer_Activity_Tauber_tk.txt")
```

```{r}
anova(LA_lmer_Activity_Tauber)
```

```{r}
summary(LA_lmer_Activity_Tauber)
```


From Eran :

Red letters denote significant difference by Multiple Comparisons of Means: Tukey Contrasts

```{r}
knitr::include_graphics("Locomotor/FromEran/pheno2022_Page_1.png")
```

```{r}
knitr::include_graphics("Locomotor/FromEran/pheno2022_Page_2.png")
```

```{r}
knitr::include_graphics("Locomotor/FromEran/pheno2022_Page_3.png")
```

Phase in DD represented in angle. The mean and 95 confidence limits are shown. CI limits in red indicate uniform distribution (low concentration. i.e. Rayleigh test not significant).

```{r}
knitr::include_graphics("Locomotor/FromEran/pheno2022_Page_4.png")
```

Oriana 3 (Stacked circular).   

```{r}
knitr::include_graphics("Locomotor/FromEran/pheno2022_Page_5.png")
```


## Starvation resistance


```{r include=FALSE}
dir.create(file.path("Starvation"), showWarnings = FALSE)
```


_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _


### Contributors  

Gonzalez Lab : Llewellyn Green, Josefa Gonzalez, Miriam Merenciano

Onder Lab : Seda Coskun, Ekin Demir, Senel Selin Senkal, Cansu Aksoy

Pasyukova Lab : Alexander Symonenko, Natalia Roshina, Mikhail Trostnokov, Ekaterina Veselkina, Evgenia Tsybul'ko, Olga Rybina, Elena Pasyukova


### Data Reformatting

```{r}
str(droseu$sr)
```

### Descriptive Statistics

Descriptive statistics at the line level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wbatch$SR, options = list(pageLength = 10)) %>% formatRound(7:14, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_line_wobatch$SR, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wbatch$SR, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
DT::datatable(table_pop_wobatch$SR, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```


### Plots and Linear Models by Lab  

#### Overview 


```{r}
min_SR <- min(droseu$sr$AgeAtDeath_hours)
max_SR <- max(droseu$sr$AgeAtDeath_hours)
```

```{r p_SR_F, echo=FALSE}

p_SR_F <- ggplot(data = filter(droseu$sr, Sex == "F"), aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_SR_F", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ droseu_theme

```


```{r echo=FALSE}
p_SR_F
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_F.pdf",width=8, height=5)
p_SR_F
invisible(dev.off())
```

```{r p_SR_M, echo=FALSE}

p_SR_M <- ggplot(data = filter(droseu$sr, Sex == "M"), aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Supervisor.PI)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + labs(title="p_SR_M", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ droseu_theme

```

```{r echo=FALSE}
p_SR_M
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_M.pdf",width=8, height=5)
p_SR_M
invisible(dev.off())
```



#### Gonzalez Lab


```{r}
lmers_anova$SR_F_Gonzalez_lmer_pop
```

```{r}
lmers_sum$SR_F_Gonzalez_lmer_pop
```

```{r}
lmers_anova$SR_M_Gonzalez_lmer_pop
```

```{r}
lmers_sum$SR_M_Gonzalez_lmer_pop
```


```{r p_SR_F_Gonzalez, echo=FALSE}

p_SR_F_Gonzalez <- ggplot(data = filter(droseu$sr, Sex == "F" & Supervisor.PI=='Gonzalez'), aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_F_Gonzalez", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ droseu_theme
```

```{r echo=FALSE}
p_SR_F_Gonzalez
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_F_Gonzalez.pdf",width=8, height=5)
p_SR_F_Gonzalez
invisible(dev.off())
```

```{r p_SR_F_pop_Gonzalez, echo=FALSE}

p_SR_F_pop_Gonzalez <- ggplot(data = filter(droseu$sr, Sex == "F" & Supervisor.PI=='Gonzalez'), aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_F_pop_Gonzalez", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ geom_text(data = lmers_tukey$SR_F_Gonzalez_lmer_pop$letters, aes(x = Population, label = cld), y = max_SR, vjust = 1) +droseu_theme

```

```{r echo=FALSE}
p_SR_F_pop_Gonzalez
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_F_pop_Gonzalez.pdf",width=8, height=5)
p_SR_F_pop_Gonzalez
invisible(dev.off())
```


```{r p_SR_M_Gonzalez, echo=FALSE}

p_SR_M_Gonzalez <- ggplot(data = filter(droseu$sr, Sex == "M" & Supervisor.PI=='Gonzalez'), aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_M_Gonzalez", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ droseu_theme

```

```{r echo=FALSE}
p_SR_M_Gonzalez
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_M_Gonzalez.pdf",width=8, height=5)
p_SR_M_Gonzalez
invisible(dev.off())
```

```{r p_SR_M_pop_Gonzalez, echo=FALSE}

p_SR_M_pop_Gonzalez <- ggplot(data = filter(droseu$sr, Sex == "M" & Supervisor.PI=='Gonzalez'), aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_M_pop_Gonzalez", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ geom_text(data = lmers_tukey$SR_M_Gonzalez_lmer_pop$letters, aes(x = Population, label = cld), y = max_SR, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_SR_M_pop_Gonzalez
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_M_pop_Gonzalez.pdf",width=8, height=5)
p_SR_M_pop_Gonzalez
invisible(dev.off())
```


#### Onder Lab 


```{r}
lmers_anova$SR_F_Onder_lmer_pop
```

```{r}
lmers_sum$SR_F_Onder_lmer_pop
```

```{r}
lmers_anova$SR_M_Onder_lmer_pop
```

```{r}
lmers_sum$SR_M_Onder_lmer_pop
```

```{r p_SR_F_Onder, echo=FALSE}

p_SR_F_Onder <- ggplot(data = filter(droseu$sr, Sex == "F" & Supervisor.PI=="Onder"),  aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_F_Onder", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ droseu_theme

```

```{r echo=FALSE}
p_SR_F_Onder
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_F_Onder.pdf", width=8, height=5)
p_SR_F_Onder
invisible(dev.off())
```

```{r p_SR_F_pop_Onder, echo=FALSE}

p_SR_F_pop_Onder <- ggplot(data = filter(droseu$sr, Sex == "F" & Supervisor.PI=="Onder"), aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_F_pop_Onder", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ geom_text(data = lmers_tukey$SR_F_Onder_lmer_pop$letters, aes(x = Population, label = cld), y = max_SR, vjust = 1) + droseu_theme

```


```{r echo=FALSE}
p_SR_F_pop_Onder
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_F_pop_Onder.pdf",width=8, height=5)
p_SR_F_pop_Onder
invisible(dev.off())
```

```{r p_SR_M_Onder, echo=FALSE}

p_SR_M_Onder <- ggplot(data = filter(droseu$sr, Sex == "M" & Supervisor.PI=="Onder"),  aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_M_Onder", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ droseu_theme

```

```{r echo=FALSE}
p_SR_M_Onder
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_M_Onder.pdf",width=8, height=5)
p_SR_M_Onder
invisible(dev.off())
```

```{r p_SR_M_pop_Onder, echo=FALSE}

p_SR_M_pop_Onder <- ggplot(data = filter(droseu$sr, Sex == "M" & Supervisor.PI=="Onder"),  aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_M_pop_Onder", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ geom_text(data = lmers_tukey$SR_M_Onder_lmer_pop$letters, aes(x = Population, label = cld), y = max_SR, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_SR_M_pop_Onder
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_M_pop_Onder.pdf",width=8, height=5)
p_SR_M_pop_Onder
invisible(dev.off())
```

#### Pasyukova Lab

```{r}
lmers_anova$SR_F_Pasyukova_lmer_pop
```

```{r}
lmers_sum$SR_F_Pasyukova_lmer_pop
```

```{r}
lmers_anova$SR_M_Pasyukova_lmer_pop
```

```{r}
lmers_sum$SR_M_Pasyukova_lmer_pop
```


```{r p_SR_F_Pasyukova, echo=FALSE}

p_SR_F_Pasyukova <- ggplot(data = filter(droseu$sr, Sex == "F" & Supervisor.PI=="Pasyukova"),  aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_F_Pasyukova", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ droseu_theme

```

```{r echo=FALSE}
p_SR_F_Pasyukova
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_F_Pasyukova.pdf",width=8, height=5)
p_SR_F_Pasyukova
invisible(dev.off())
```


```{r p_SR_F_pop_Pasyukova, echo=FALSE}

p_SR_F_pop_Pasyukova <- ggplot(data = filter(droseu$sr, Sex == "F" & Supervisor.PI=="Pasyukova"),  aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_F_pop_Pasyukova", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ geom_text(data = lmers_tukey$SR_F_Pasyukova_lmer_pop$letters, aes(x = Population, label = cld), y = max_SR, vjust = 1) + droseu_theme

```


```{r echo=FALSE}
p_SR_F_pop_Pasyukova
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_F_pop_Pasyukova.pdf",width=8, height=5)
p_SR_F_pop_Pasyukova
invisible(dev.off())
```

```{r p_SR_M_Pasyukova, echo=FALSE}

p_SR_M_Pasyukova <- ggplot(data = filter(droseu$sr, Sex == "M" & Supervisor.PI=="Pasyukova"),  aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_M_Pasyukova", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR)) +droseu_theme
```

```{r echo=FALSE}
p_SR_M_Pasyukova
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_M_Pasyukova.pdf",width=8, height=5)
p_SR_M_Pasyukova
invisible(dev.off())
```

```{r p_SR_M_pop_Pasyukova, echo=FALSE}

p_SR_M_pop_Pasyukova <- ggplot(data = filter(droseu$sr, Sex == "M" & Supervisor.PI=="Pasyukova"),  aes(x=Population_Lat, y=AgeAtDeath_hours, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_SR_M_pop_Pasyukova", x="Population", y = "AgeAtDeath_hours") + ylim(c(min_SR, max_SR))+ geom_text(data = lmers_tukey$SR_M_Pasyukova_lmer_pop$letters, aes(x = Population, label = cld), y = max_SR, vjust = 1) + droseu_theme

```

```{r echo=FALSE}
p_SR_M_pop_Pasyukova
```

```{r echo=FALSE}
pdf(file="Starvation/p_SR_M_pop_Pasyukova.pdf",width=8, height=5)
p_SR_M_pop_Pasyukova
invisible(dev.off())
```



### Survival Analyses


#### Gonzalez Lab

```{r echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/Starvation/p_SR_F_Gonzalez_survival_curves.pdf")
```

```{r echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/Starvation/p_SR_M_Gonzalez_survival_curves.pdf")
```

#### Onder Lab

```{r echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/Starvation/p_SR_F_Onder_survival_curves.pdf")
```

```{r echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/Starvation/p_SR_M_Onder_survival_curves.pdf")
```

#### Pasyukova Lab

```{r echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/Starvation/p_SR_F_Pasyukova_survival_curves.pdf")
```

```{r echo=FALSE}
knitr::include_graphics("SurvivalAnalyses/Starvation/p_SR_M_Pasyukova_survival_curves.pdf")
```

#### coxme


```{r}
#Please refer to "_SurvivalAnalyses_" folder
```

### Linear Models / Residuals



#### Gonzalez Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_F_Gonzalez_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_F_Gonzalez_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_M_Gonzalez_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_M_Gonzalez_lmer_pop_qq_plot_residuals.png")
```

#### Onder Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_F_Onder_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_F_Onder_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_M_Onder_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_M_Onder_lmer_pop_qq_plot_residuals.png")
```

#### Pasyukova Lab

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_F_Pasyukova_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_F_Pasyukova_lmer_pop_qq_plot_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_M_Pasyukova_lmer_pop_hist_residuals.png")
```

```{r echo=FALSE}
knitr::include_graphics("LinearModelsPop/Starvation/by_lab_lmer_residuals/SR_M_Pasyukova_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations

```{r intersect_SR, echo=FALSE}
intersect_Line_SR  <- Reduce(intersect, list(droseu$sr$Line[droseu$sr$Supervisor.PI == "Gonzalez"],
                                        droseu$sr$Line[droseu$sr$Supervisor.PI == "Pasyukova"],
                                        droseu$sr$Line[droseu$sr$Supervisor.PI == "Onder"]))

write(intersect_Line_SR, file ="Starvation/intersect_Line_SR.txt", sep = "\t")

intersect_Pop_SR  <- Reduce(intersect, list(droseu$sr$Population[droseu$sr$Supervisor.PI == "Gonzalez"],
                                        droseu$sr$Population[droseu$sr$Supervisor.PI == "Pasyukova"],
                                        droseu$sr$Line[droseu$sr$Supervisor.PI == "Onder"]))
write(intersect_Pop_SR, file ="Starvation/intersect_Pop_SR.txt", sep = "\t")

```

There are `r length(intersect_Pop_SR) ` populations and `r length(intersect_Line_SR) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 


```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/Starvation/SR_F_lab_correlations.png")
```

```{r echo=FALSE}
knitr::include_graphics("LabCorrelations/Starvation/SR_M_lab_correlations.png")
```



## Pigmentation

```{r include=FALSE}
dir.create(file.path("Pigmentation"), showWarnings = FALSE)
```

_For a detailed description of tables, plots, linear models and outputs, please refer [here](#anchor) _


### Contributors  

Abbott Lab : Jessica Abbott, Qinyang Li, Shahzad Khan

Gibert Lab :Cristina Vieira, Laurence Mouton, Natacha Kremer, Sonia Martinez, Camille Mermet, Patricia Gibert

Schmidt Lab : Amy Goldfischer, Paul Schmidt 

### Data Reformatting

```{r d_Pgm}
d_Pgm <- read.csv("MasterSheets_Oct21_git/PGM_MasterSheet_Jan22.csv")
```

```{r}
str(d_Pgm)
```


```{r}
d_Pgm$Supervisor.PI <- as.factor(d_Pgm$Supervisor.PI)
d_Pgm$Diet <- as.factor(d_Pgm$Diet)
d_Pgm$Batch <- as.factor(d_Pgm$Batch)
d_Pgm$Population_Lat <- factor(d_Pgm$Population, levels= c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))
d_Pgm$Population_Lon <- factor(d_Pgm$Population, levels= c("RE","GI","KA","MU","MA","AK","UM","YE","VA"))
d_Pgm$Population_Alt <- factor(d_Pgm$Population, levels= c("KA","AK","GI","RE","UM","VA","MU","MA","YE"))
d_Pgm$Line <- as.factor(d_Pgm$Line)
d_Pgm$AreaT4 <- as.numeric(d_Pgm$AreaT4)
d_Pgm$AreaT5 <- as.numeric(d_Pgm$AreaT5)
d_Pgm$AreaT6 <- as.numeric(d_Pgm$AreaT6)
d_Pgm$PercT4 <- as.numeric(d_Pgm$PercT4)
d_Pgm$PercT5 <- as.numeric(d_Pgm$PercT5)
d_Pgm$PercT6 <- as.numeric(d_Pgm$PercT6)
d_Pgm$TotalArea <- as.numeric(d_Pgm$TotalArea)
d_Pgm$TotalBlack <- as.numeric(d_Pgm$TotalBlack)
d_Pgm$Latitude <- as.numeric(d_Pgm$Latitude)
d_Pgm$Longitude <- as.numeric(d_Pgm$Latitude)
d_Pgm$Altitude <- as.numeric(d_Pgm$Altitude)
```

```{r}
str(d_Pgm)
```

```{r d_Pgm2}
d_Pgm2 <- read.csv("MasterSheets_Oct21_git/PGM2_MasterSheet_Oct21.csv")
```

```{r}
str(d_Pgm2)
```

```{r}
d_Pgm2$Supervisor.PI <- as.factor(d_Pgm2$Supervisor.PI)
d_Pgm2$Diet <- as.factor(d_Pgm2$Diet)
d_Pgm2$Batch <- as.factor(d_Pgm2$Batch)
d_Pgm2$Population_Lat <- factor(d_Pgm2$Population, levels= c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))
d_Pgm2$Population_Lon <- factor(d_Pgm2$Population, levels= c("RE","GI","KA","MU","MA","AK","UM","YE","VA"))
d_Pgm2$Population_Alt <- factor(d_Pgm2$Population, levels= c("KA","AK","GI","RE","UM","VA","MU","MA","YE"))
d_Pgm2$Line <- as.factor(d_Pgm2$Line)
d_Pgm2$Tergite8 <- as.numeric(d_Pgm2$Tergite8)
d_Pgm2$Tergite9 <- as.numeric(d_Pgm2$Tergite9)
d_Pgm2$Tergite10 <- as.numeric(d_Pgm2$Tergite10)
d_Pgm2$Total <- as.numeric(d_Pgm2$Total)
```


```{r}
str(d_Pgm2)
```

### Descriptive Statistics


### _T4_


Descriptive statistics at the line level, with batch information :
```{r table_Pgm_Line_PercT4_wbatch, echo=FALSE}
table_Pgm_Line_PercT4_wbatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(PercT4), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_PercT4_wbatch.csv", row.names = T)

table_Pgm_Line_PercT4_wbatch <- read.csv("Pigmentation/table_Pgm_Line_PercT4_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_Pgm_Line_PercT4_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
table_Pgm_Line_PercT4_wobatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Population, Line) %>% summarise_at(vars(PercT4), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_PercT4_wobatch.csv", row.names = T)
table_Pgm_Line_PercT4_wobatch <- read.csv("Pigmentation/table_Pgm_Line_PercT4_wobatch.csv")
```

```{r table_Pgm_Line_PercT4_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Line_PercT4_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
table_Pgm_Pop_PercT4_wbatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Batch, Population) %>% 
                                   summarise_at(vars(PercT4), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_PercT4_wbatch.csv", row.names = T)

table_Pgm_Pop_PercT4_wbatch <- read.csv("Pigmentation/table_Pgm_Pop_PercT4_wbatch.csv")
```

```{r table_Pgm_Pop_PercT4_wbatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_PercT4_wbatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
table_Pgm_Pop_PercT4_wobatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Population) %>% 
                                   summarise_at(vars(PercT4), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_PercT4_wobatch.csv", row.names = T)

table_Pgm_Pop_PercT4_wobatch <- read.csv("Pigmentation/table_Pgm_Pop_PercT4_wobatch.csv")
```

```{r table_Pgm_Pop_PercT4_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_PercT4_wobatch, options = list(pageLength = 10))%>% formatRound(4:11, 3)
```


### _T5_

Descriptive statistics at the line level, with batch information :
```{r table_Pgm_Line_PercT5_wbatch, echo=FALSE}
table_Pgm_Line_PercT5_wbatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(PercT5), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_PercT5_wbatch.csv", row.names = T)

table_Pgm_Line_PercT5_wbatch <- read.csv("Pigmentation/table_Pgm_Line_PercT5_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_Pgm_Line_PercT5_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
table_Pgm_Line_PercT5_wobatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Population, Line) %>% summarise_at(vars(PercT5), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_PercT5_wobatch.csv", row.names = T)
table_Pgm_Line_PercT5_wobatch <- read.csv("Pigmentation/table_Pgm_Line_PercT5_wobatch.csv")
```

```{r table_Pgm_Line_PercT5_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Line_PercT5_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
table_Pgm_Pop_PercT5_wbatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Batch, Population) %>% 
                                   summarise_at(vars(PercT5), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_PercT5_wbatch.csv", row.names = T)

table_Pgm_Pop_PercT5_wbatch <- read.csv("Pigmentation/table_Pgm_Pop_PercT5_wbatch.csv")
```

```{r table_Pgm_Pop_PercT5_wbatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_PercT5_wbatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
table_Pgm_Pop_PercT5_wobatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Population) %>% 
                                   summarise_at(vars(PercT5), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_PercT5_wobatch.csv", row.names = T)

table_Pgm_Pop_PercT5_wobatch <- read.csv("Pigmentation/table_Pgm_Pop_PercT5_wobatch.csv")
```

```{r table_Pgm_Pop_PercT5_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_PercT5_wobatch, options = list(pageLength = 10))%>% formatRound(4:11, 3)
```

### _T6_

Descriptive statistics at the line level, with batch information :
```{r table_Pgm_Line_PercT6_wbatch, echo=FALSE}
table_Pgm_Line_PercT6_wbatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(PercT6), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_PercT6_wbatch.csv", row.names = T)

table_Pgm_Line_PercT6_wbatch <- read.csv("Pigmentation/table_Pgm_Line_PercT6_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_Pgm_Line_PercT6_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
table_Pgm_Line_PercT6_wobatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Population, Line) %>% summarise_at(vars(PercT6), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_PercT6_wobatch.csv", row.names = T)
table_Pgm_Line_PercT6_wobatch <- read.csv("Pigmentation/table_Pgm_Line_PercT6_wobatch.csv")
```

```{r table_Pgm_Line_PercT6_wobatch}
DT::datatable(table_Pgm_Line_PercT6_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
table_Pgm_Pop_PercT6_wbatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Batch, Population) %>% 
                                   summarise_at(vars(PercT6), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_PercT6_wbatch.csv", row.names = T)

table_Pgm_Pop_PercT6_wbatch <- read.csv("Pigmentation/table_Pgm_Pop_PercT6_wbatch.csv")
```

```{r table_Pgm_Pop_PercT6_wbatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_PercT6_wbatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
table_Pgm_Pop_PercT6_wobatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Population) %>% 
                                   summarise_at(vars(PercT6), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_PercT6_wobatch.csv", row.names = T)

table_Pgm_Pop_PercT6_wobatch <- read.csv("Pigmentation/table_Pgm_Pop_PercT6_wobatch.csv")
```

```{r table_Pgm_Pop_PercT6_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_PercT6_wobatch, options = list(pageLength = 10))%>% formatRound(4:11, 3)
```

### _Total_

Descriptive statistics at the line level, with batch information :
```{r table_Pgm_Line_TotalPerc_wbatch, echo=FALSE}
table_Pgm_Line_TotalPerc_wbatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(TotalPerc), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_TotalPerc_wbatch.csv", row.names = T)

table_Pgm_Line_TotalPerc_wbatch <- read.csv("Pigmentation/table_Pgm_Line_TotalPerc_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_Pgm_Line_TotalPerc_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
table_Pgm_Line_TotalPerc_wobatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Population, Line) %>% summarise_at(vars(TotalPerc), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_TotalPerc_wobatch.csv", row.names = T)
table_Pgm_Line_TotalPerc_wobatch <- read.csv("Pigmentation/table_Pgm_Line_TotalPerc_wobatch.csv")
```

```{r table_Pgm_Line_TotalPerc_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Line_TotalPerc_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
table_Pgm_Pop_TotalPerc_wbatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Batch, Population) %>% 
                                   summarise_at(vars(TotalPerc), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_TotalPerc_wbatch.csv", row.names = T)

table_Pgm_Pop_TotalPerc_wbatch <- read.csv("Pigmentation/table_Pgm_Pop_TotalPerc_wbatch.csv")
```

```{r table_Pgm_Pop_TotalPerc_wbatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_TotalPerc_wbatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
table_Pgm_Pop_TotalPerc_wobatch <- write.csv(d_Pgm %>% group_by(Supervisor.PI, Population) %>% 
                                   summarise_at(vars(TotalPerc), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_TotalPerc_wobatch.csv", row.names = T)

table_Pgm_Pop_TotalPerc_wobatch <- read.csv("Pigmentation/table_Pgm_Pop_TotalPerc_wobatch.csv")
```

```{r table_Pgm_Pop_TotalPerc_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_TotalPerc_wobatch, options = list(pageLength = 10))%>% formatRound(4:11, 3)
```

### _T8_

Descriptive statistics at the line level, with batch information :
```{r table_Pgm_Line_Tergite8_wbatch, echo=FALSE}
table_Pgm_Line_Tergite8_wbatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(Tergite8), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_Tergite8_wbatch.csv", row.names = T)

table_Pgm_Line_Tergite8_wbatch <- read.csv("Pigmentation/table_Pgm_Line_Tergite8_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_Pgm_Line_Tergite8_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
table_Pgm_Line_Tergite8_wobatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Population, Line) %>% summarise_at(vars(Tergite8), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_Tergite8_wobatch.csv", row.names = T)
table_Pgm_Line_Tergite8_wobatch <- read.csv("Pigmentation/table_Pgm_Line_Tergite8_wobatch.csv")
```

```{r table_Pgm_Line_Tergite8_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Line_Tergite8_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
table_Pgm_Pop_Tergite8_wbatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Batch, Population) %>% 
                                   summarise_at(vars(Tergite8), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_Tergite8_wbatch.csv", row.names = T)

table_Pgm_Pop_Tergite8_wbatch <- read.csv("Pigmentation/table_Pgm_Pop_Tergite8_wbatch.csv")
```

```{r table_Pgm_Pop_Tergite8_wbatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_Tergite8_wbatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
table_Pgm_Pop_Tergite8_wobatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Population) %>% 
                                   summarise_at(vars(Tergite8), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_Tergite8_wobatch.csv", row.names = T)

table_Pgm_Pop_Tergite8_wobatch <- read.csv("Pigmentation/table_Pgm_Pop_Tergite8_wobatch.csv")
```

```{r table_Pgm_Pop_Tergite8_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_Tergite8_wobatch, options = list(pageLength = 10))%>% formatRound(4:11, 3)
```

### _T9_

Descriptive statistics at the line level, with batch information :
```{r table_Pgm_Line_Tergite9_wbatch, echo=FALSE}
table_Pgm_Line_Tergite9_wbatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(Tergite9), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_Tergite9_wbatch.csv", row.names = T)

table_Pgm_Line_Tergite9_wbatch <- read.csv("Pigmentation/table_Pgm_Line_Tergite9_wbatch.csv")
```

```{r}
DT::datatable(table_Pgm_Line_Tergite9_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information :
```{r}
table_Pgm_Line_Tergite9_wobatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Population, Line) %>% summarise_at(vars(Tergite9), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_Tergite9_wobatch.csv", row.names = T)
table_Pgm_Line_Tergite9_wobatch <- read.csv("Pigmentation/table_Pgm_Line_Tergite9_wobatch.csv")
```

```{r table_Pgm_Line_Tergite9_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Line_Tergite9_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
table_Pgm_Pop_Tergite9_wbatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Batch, Population) %>% 
                                   summarise_at(vars(Tergite9), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_Tergite9_wbatch.csv", row.names = T)

table_Pgm_Pop_Tergite9_wbatch <- read.csv("Pigmentation/table_Pgm_Pop_Tergite9_wbatch.csv")
```

```{r table_Pgm_Pop_Tergite9_wbatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_Tergite9_wbatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
table_Pgm_Pop_Tergite9_wobatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Population) %>% 
                                   summarise_at(vars(Tergite9), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_Tergite9_wobatch.csv", row.names = T)

table_Pgm_Pop_Tergite9_wobatch <- read.csv("Pigmentation/table_Pgm_Pop_Tergite9_wobatch.csv")
```

```{r table_Pgm_Pop_Tergite9_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_Tergite9_wobatch, options = list(pageLength = 10))%>% formatRound(4:11, 3)
```

### _T10_


Descriptive statistics at the line level, with batch information :
```{r table_Pgm_Line_Tergite10_wbatch, echo=FALSE}
table_Pgm_Line_Tergite10_wbatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(Tergite10), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_Tergite10_wbatch.csv", row.names = T)

table_Pgm_Line_Tergite10_wbatch <- read.csv("Pigmentation/table_Pgm_Line_Tergite10_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_Pgm_Line_Tergite10_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
table_Pgm_Line_Tergite10_wobatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Population, Line) %>% summarise_at(vars(Tergite10), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_Tergite10_wobatch.csv", row.names = T)
table_Pgm_Line_Tergite10_wobatch <- read.csv("Pigmentation/table_Pgm_Line_Tergite10_wobatch.csv")
```

```{r table_Pgm_Line_Tergite10_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Line_Tergite10_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
table_Pgm_Pop_Tergite10_wbatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Batch, Population) %>% 
                                   summarise_at(vars(Tergite10), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_Tergite10_wbatch.csv", row.names = T)

table_Pgm_Pop_Tergite10_wbatch <- read.csv("Pigmentation/table_Pgm_Pop_Tergite10_wbatch.csv")
```

```{r table_Pgm_Pop_Tergite10_wbatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_Tergite10_wbatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
table_Pgm_Pop_Tergite10_wobatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Population) %>% 
                                   summarise_at(vars(Tergite10), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_Tergite10_wobatch.csv", row.names = T)

table_Pgm_Pop_Tergite10_wobatch <- read.csv("Pigmentation/table_Pgm_Pop_Tergite10_wobatch.csv")
```

```{r table_Pgm_Pop_Tergite10_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_Tergite10_wobatch, options = list(pageLength = 10))%>% formatRound(4:11, 3)
```

### _Total2_


Descriptive statistics at the line level, with batch information :
```{r table_Pgm_Line_Total_wbatch, echo=FALSE}
table_Pgm_Line_Total_wbatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Batch, Population, Line) %>% summarise_at(vars(Total), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_Total_wbatch.csv", row.names = T)

table_Pgm_Line_Total_wbatch <- read.csv("Pigmentation/table_Pgm_Line_Total_wbatch.csv")
```

```{r, echo=FALSE}
DT::datatable(table_Pgm_Line_Total_wbatch, options = list(pageLength = 10)) %>% formatRound(6:13, 3)
```

Descriptive statistics at the line level, without batch information :
```{r, echo=FALSE}
table_Pgm_Line_Total_wobatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Population, Line) %>% summarise_at(vars(Total), list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, SE = std_err, CV = coef_var, Mode = estimate_mode)), file = "Pigmentation/table_Pgm_Line_Total_wobatch.csv", row.names = T)
table_Pgm_Line_Total_wobatch <- read.csv("Pigmentation/table_Pgm_Line_Total_wobatch.csv")
```

```{r table_Pgm_Line_Total_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Line_Total_wobatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, with batch information :
```{r, echo=FALSE}
table_Pgm_Pop_Total_wbatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Batch, Population) %>% 
                                   summarise_at(vars(Total), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_Total_wbatch.csv", row.names = T)

table_Pgm_Pop_Total_wbatch <- read.csv("Pigmentation/table_Pgm_Pop_Total_wbatch.csv")
```

```{r table_Pgm_Pop_Total_wbatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_Total_wbatch, options = list(pageLength = 10)) %>% formatRound(5:12, 3)
```

Descriptive statistics at the population level, without batch information :
```{r, echo=FALSE}
table_Pgm_Pop_Total_wobatch <- write.csv(d_Pgm2 %>% group_by(Supervisor.PI, Population) %>% 
                                   summarise_at(vars(Total), 
                                                list(Mean = mean, SD= sd, Median = median, Min = min, Max = max, 
                                                     SE = std_err, CV = coef_var, Mode = estimate_mode)), 
                           file = "Pigmentation/table_Pgm_Pop_Total_wobatch.csv", row.names = T)

table_Pgm_Pop_Total_wobatch <- read.csv("Pigmentation/table_Pgm_Pop_Total_wobatch.csv")
```

```{r table_Pgm_Pop_Total_wobatch, echo=FALSE}
DT::datatable(table_Pgm_Pop_Total_wobatch, options = list(pageLength = 10))%>% formatRound(4:11, 3)
```

#### Overview 

### Plots and Linear Models by Lab  

#### Abbott Lab

_PercT4_

```{r Pgm_lmer_Abbott_PercT4, echo=FALSE}
Pgm_lmer_Abbott_PercT4 <- lmer(PercT4 ~ Population + (1|Line:Population), 
                               data = (subset(d_Pgm,Supervisor.PI=='Abbott')))
capture.output(summary(Pgm_lmer_Abbott_PercT4),file = "Pigmentation/Pgm_lmer_Abbott_PercT4_sum.txt")
capture.output(anova(Pgm_lmer_Abbott_PercT4),file = "Pigmentation/Pgm_lmer_Abbott_PercT4.txt")
capture.output(emmeans(Pgm_lmer_Abbott_PercT4, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm_lmer_Abbott_PercT4_tk.txt")
```



```{r include=FALSE}
model_means_Pgm_PercT4_Abbott <- emmeans(object = Pgm_lmer_Abbott_PercT4, specs = ~ Population)
model_means_Pgm_PercT4_Abbott_cld <- cld(object = model_means_Pgm_PercT4_Abbott, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```


```{r}
anova(Pgm_lmer_Abbott_PercT4)
```

```{r}
summary(Pgm_lmer_Abbott_PercT4)
```


```{r p_Pgm_Abbott_PercT4, echo=FALSE}

p_Pgm_Abbott_PercT4 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Abbott')), aes(x=Population_Lat, y=PercT4, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_Abbott_PercT4", x="Population", y = "PercT4") + ylim(c(min(d_Pgm$PercT4) , max(d_Pgm$PercT4)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm_Abbott_PercT4
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_Abbott_PercT4.pdf",width=8, height=5)
p_Pgm_Abbott_PercT4
invisible(dev.off())
```


```{r p_Pgm_pop_Abbott_PercT4, echo=FALSE}

p_Pgm_pop_Abbott_PercT4 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Abbott')), aes(x=Population_Lat, y=PercT4, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_pop_Abbott_PercT4", x="Population", y = "PercT4") + ylim(c(min(d_Pgm$PercT4) , max(d_Pgm$PercT4)))+ geom_text(data = model_means_Pgm_PercT4_Abbott_cld, aes(x = Population, label = cld), y = max(d_Pgm$PercT4), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm_pop_Abbott_PercT4
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_pop_Abbott_PercT4.pdf",width=8, height=5)
p_Pgm_pop_Abbott_PercT4
invisible(dev.off())
```



_PercT5_

```{r Pgm_lmer_Abbott_PercT5, echo=FALSE}
Pgm_lmer_Abbott_PercT5 <- lmer(PercT5 ~ Population  + (1|Line:Population) , data = (subset(d_Pgm,Supervisor.PI=='Abbott')))
capture.output(summary(Pgm_lmer_Abbott_PercT5),file = "Pigmentation/Pgm_lmer_Abbott_PercT5_sum.txt")
capture.output(anova(Pgm_lmer_Abbott_PercT5),file = "Pigmentation/Pgm_lmer_Abbott_PercT5.txt")
capture.output(emmeans(Pgm_lmer_Abbott_PercT5, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm_lmer_Abbott_PercT5_tk.txt")
```

```{r include=FALSE}
model_means_Pgm_PercT5_Abbott <- emmeans(object = Pgm_lmer_Abbott_PercT5, specs = ~ Population)
model_means_Pgm_PercT5_Abbott_cld <- cld(object = model_means_Pgm_PercT5_Abbott, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```



```{r}
anova(Pgm_lmer_Abbott_PercT5)
```

```{r}
summary(Pgm_lmer_Abbott_PercT5)
```


```{r p_Pgm_Abbott_PercT5, echo=FALSE}

p_Pgm_Abbott_PercT5 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Abbott')), aes(x=Population_Lat, y=PercT5, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_Abbott_PercT5", x="Population", y = "PercT5") + ylim(c(min(d_Pgm$PercT5) , max(d_Pgm$PercT5)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm_Abbott_PercT5
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_Abbott_PercT5.pdf",width=8, height=5)
p_Pgm_Abbott_PercT5
invisible(dev.off())
```


```{r p_Pgm_pop_Abbott_PercT5, echo=FALSE}

p_Pgm_pop_Abbott_PercT5 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Abbott')), aes(x=Population_Lat, y=PercT5, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_pop_Abbott_PercT5", x="Population", y = "PercT5") + ylim(c(min(d_Pgm$PercT5) , max(d_Pgm$PercT5)))+ geom_text(data = model_means_Pgm_PercT5_Abbott_cld, aes(x = Population, label = cld), y = max(d_Pgm$PercT5), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm_pop_Abbott_PercT5
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_pop_Abbott_PercT5.pdf",width=8, height=5)
p_Pgm_pop_Abbott_PercT5
invisible(dev.off())
```


_PercT6_

```{r Pgm_lmer_Abbott_PercT6, echo=FALSE}
Pgm_lmer_Abbott_PercT6 <- lmer(PercT6 ~ Population  + (1|Line:Population) , data = (subset(d_Pgm,Supervisor.PI=='Abbott')))
capture.output(summary(Pgm_lmer_Abbott_PercT6),file = "Pigmentation/Pgm_lmer_Abbott_PercT6_sum.txt")
capture.output(anova(Pgm_lmer_Abbott_PercT6),file = "Pigmentation/Pgm_lmer_Abbott_PercT6.txt")
capture.output(emmeans(Pgm_lmer_Abbott_PercT6, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm_lmer_Abbott_PercT6_tk.txt")
```


```{r include=FALSE}
model_means_Pgm_PercT6_Abbott <- emmeans(object = Pgm_lmer_Abbott_PercT6, specs = ~ Population)
model_means_Pgm_PercT6_Abbott_cld <- cld(object = model_means_Pgm_PercT6_Abbott, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```


```{r}
anova(Pgm_lmer_Abbott_PercT6)
```

```{r}
summary(Pgm_lmer_Abbott_PercT6)
```


```{r p_Pgm_Abbott_PercT6, echo=FALSE}

p_Pgm_Abbott_PercT6 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Abbott')), aes(x=Population_Lat, y=PercT6, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_Abbott_PercT6", x="Population", y = "PercT6") + ylim(c(min(d_Pgm$PercT6) , max(d_Pgm$PercT6)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm_Abbott_PercT6
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_Abbott_PercT6.pdf",width=8, height=5)
p_Pgm_Abbott_PercT6
invisible(dev.off())
```


```{r p_Pgm_pop_Abbott_PercT6, echo=FALSE}

p_Pgm_pop_Abbott_PercT6 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Abbott')), aes(x=Population_Lat, y=PercT6, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_pop_Abbott_PercT6", x="Population", y = "PercT6") + ylim(c(min(d_Pgm$PercT6) , max(d_Pgm$PercT6)))+ geom_text(data = model_means_Pgm_PercT6_Abbott_cld, aes(x = Population, label = cld), y = max(d_Pgm$PercT6), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm_pop_Abbott_PercT6
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_pop_Abbott_PercT6.pdf",width=8, height=5)
p_Pgm_pop_Abbott_PercT6
invisible(dev.off())
```


_TotalPerc_

```{r Pgm_lmer_Abbott_TotalPerc, echo=FALSE}
Pgm_lmer_Abbott_TotalPerc <- lmer(TotalPerc ~ Population  + (1|Line:Population)  , data = (subset(d_Pgm,Supervisor.PI=='Abbott')))
capture.output(summary(Pgm_lmer_Abbott_TotalPerc),file = "Pigmentation/Pgm_lmer_Abbott_TotalPerc_sum.txt")
capture.output(anova(Pgm_lmer_Abbott_TotalPerc),file = "Pigmentation/Pgm_lmer_Abbott_TotalPerc.txt")
capture.output(emmeans(Pgm_lmer_Abbott_TotalPerc, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm_lmer_Abbott_TotalPerc_tk.txt")
```


```{r include=FALSE}
model_means_Pgm_TotalPerc_Abbott <- emmeans(object = Pgm_lmer_Abbott_TotalPerc, specs = ~ Population)
model_means_Pgm_TotalPerc_Abbott_cld <- cld(object = model_means_Pgm_TotalPerc_Abbott, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```



```{r}
anova(Pgm_lmer_Abbott_TotalPerc)
```

```{r}
summary(Pgm_lmer_Abbott_TotalPerc)
```


```{r p_Pgm_Abbott_TotalPerc, echo=FALSE}

p_Pgm_Abbott_TotalPerc <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Abbott')), aes(x=Population_Lat, y=TotalPerc, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_Abbott_TotalPerc", x="Population", y = "TotalPerc") + ylim(c(min(d_Pgm$TotalPerc) , max(d_Pgm$TotalPerc)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm_Abbott_TotalPerc
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_Abbott_TotalPerc.pdf",width=8, height=5)
p_Pgm_Abbott_TotalPerc
invisible(dev.off())
```


```{r p_Pgm_pop_Abbott_TotalPerc, echo=FALSE}

p_Pgm_pop_Abbott_TotalPerc <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Abbott')), aes(x=Population_Lat, y=TotalPerc, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_pop_Abbott_TotalPerc", x="Population", y = "TotalPerc") + ylim(c(min(d_Pgm$TotalPerc) , max(d_Pgm$TotalPerc)))+ geom_text(data = model_means_Pgm_TotalPerc_Abbott_cld, aes(x = Population, label = cld), y = max(d_Pgm$TotalPerc), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm_pop_Abbott_TotalPerc
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_pop_Abbott_TotalPerc.pdf",width=8, height=5)
p_Pgm_pop_Abbott_TotalPerc
invisible(dev.off())
```


#### Gibert Lab 


_PercT4_

```{r Pgm_lmer_Gibert_PercT4, echo=FALSE}
Pgm_lmer_Gibert_PercT4 <- lmer(PercT4 ~ Population  + (1|Line:Population) +
                                    (1|Batch), data = (subset(d_Pgm,Supervisor.PI=='Gibert')))
capture.output(summary(Pgm_lmer_Gibert_PercT4),file = "Pigmentation/Pgm_lmer_Gibert_PercT4_sum.txt")
capture.output(anova(Pgm_lmer_Gibert_PercT4),file = "Pigmentation/Pgm_lmer_Gibert_PercT4.txt")
capture.output(emmeans(Pgm_lmer_Gibert_PercT4, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm_lmer_Gibert_PercT4_tk.txt")
```


```{r include=FALSE}
model_means_Pgm_PercT4_Gibert <- emmeans(object = Pgm_lmer_Gibert_PercT4, specs = ~ Population)
model_means_Pgm_PercT4_Gibert_cld <- cld(object = model_means_Pgm_PercT4_Gibert, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```



```{r}
anova(Pgm_lmer_Gibert_PercT4)
```

```{r}
summary(Pgm_lmer_Gibert_PercT4)
```


```{r p_Pgm_Gibert_PercT4, echo=FALSE}

p_Pgm_Gibert_PercT4 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Gibert')), aes(x=Population_Lat, y=PercT4, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_Gibert_PercT4", x="Population", y = "PercT4") + ylim(c(min(d_Pgm$PercT4) , max(d_Pgm$PercT4)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm_Gibert_PercT4
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_Gibert_PercT4.pdf",width=8, height=5)
p_Pgm_Gibert_PercT4
invisible(dev.off())
```


```{r p_Pgm_pop_Gibert_PercT4, echo=FALSE}

p_Pgm_pop_Gibert_PercT4 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Gibert')), aes(x=Population_Lat, y=PercT4, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_pop_Gibert_PercT4", x="Population", y = "PercT4") + ylim(c(min(d_Pgm$PercT4) , max(d_Pgm$PercT4)))+ geom_text(data = model_means_Pgm_PercT4_Gibert_cld, aes(x = Population, label = cld), y = max(d_Pgm$PercT4), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm_pop_Gibert_PercT4
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_pop_Gibert_PercT4.pdf",width=8, height=5)
p_Pgm_pop_Gibert_PercT4
invisible(dev.off())
```



_PercT5_

```{r Pgm_lmer_Gibert_PercT5, echo=FALSE}
Pgm_lmer_Gibert_PercT5 <- lmer(PercT5 ~ Population  + (1|Line:Population), data = (subset(d_Pgm,Supervisor.PI=='Gibert')))
capture.output(summary(Pgm_lmer_Gibert_PercT5),file = "Pigmentation/Pgm_lmer_Gibert_PercT5_sum.txt")
capture.output(anova(Pgm_lmer_Gibert_PercT5),file = "Pigmentation/Pgm_lmer_Gibert_PercT5.txt")
capture.output(emmeans(Pgm_lmer_Gibert_PercT5, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm_lmer_Gibert_PercT5_tk.txt")
```

```{r include=FALSE}
model_means_Pgm_PercT5_Gibert <- emmeans(object = Pgm_lmer_Gibert_PercT5, specs = ~ Population)
model_means_Pgm_PercT5_Gibert_cld <- cld(object = model_means_Pgm_PercT5_Gibert, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```


```{r}
anova(Pgm_lmer_Gibert_PercT5)
```

```{r}
summary(Pgm_lmer_Gibert_PercT5)
```


```{r p_Pgm_Gibert_PercT5, echo=FALSE}

p_Pgm_Gibert_PercT5 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Gibert')), aes(x=Population_Lat, y=PercT5, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_Gibert_PercT5", x="Population", y = "PercT5") + ylim(c(min(d_Pgm$PercT5) , max(d_Pgm$PercT5)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm_Gibert_PercT5
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_Gibert_PercT5.pdf",width=8, height=5)
p_Pgm_Gibert_PercT5
invisible(dev.off())
```


```{r p_Pgm_pop_Gibert_PercT5, echo=FALSE}

p_Pgm_pop_Gibert_PercT5 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Gibert')), aes(x=Population_Lat, y=PercT5, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_pop_Gibert_PercT5", x="Population", y = "PercT5") + ylim(c(min(d_Pgm$PercT5) , max(d_Pgm$PercT5)))+ geom_text(data = model_means_Pgm_PercT5_Gibert_cld, aes(x = Population, label = cld), y = max(d_Pgm$PercT5), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm_pop_Gibert_PercT5
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_pop_Gibert_PercT5.pdf",width=8, height=5)
p_Pgm_pop_Gibert_PercT5
invisible(dev.off())
```


_PercT6_

```{r Pgm_lmer_Gibert_PercT6, echo=FALSE}
Pgm_lmer_Gibert_PercT6 <- lmer(PercT6 ~ Population  + (1|Line:Population) +
                                    (1|Batch), data = (subset(d_Pgm,Supervisor.PI=='Gibert')))
capture.output(summary(Pgm_lmer_Gibert_PercT6),file = "Pigmentation/Pgm_lmer_Gibert_PercT6_sum.txt")
capture.output(anova(Pgm_lmer_Gibert_PercT6),file = "Pigmentation/Pgm_lmer_Gibert_PercT6.txt")
capture.output(emmeans(Pgm_lmer_Gibert_PercT6, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm_lmer_Gibert_PercT6_tk.txt")
```


```{r include=FALSE}
model_means_Pgm_PercT6_Gibert <- emmeans(object = Pgm_lmer_Gibert_PercT6, specs = ~ Population)
model_means_Pgm_PercT6_Gibert_cld <- cld(object = model_means_Pgm_PercT6_Gibert, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```



```{r}
anova(Pgm_lmer_Gibert_PercT6)
```

```{r}
summary(Pgm_lmer_Gibert_PercT6)
```


```{r p_Pgm_Gibert_PercT6, echo=FALSE}

p_Pgm_Gibert_PercT6 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Gibert')), aes(x=Population_Lat, y=PercT6, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_Gibert_PercT6", x="Population", y = "PercT6") + ylim(c(min(d_Pgm$PercT6) , max(d_Pgm$PercT6)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm_Gibert_PercT6
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_Gibert_PercT6.pdf",width=8, height=5)
p_Pgm_Gibert_PercT6
invisible(dev.off())
```


```{r p_Pgm_pop_Gibert_PercT6, echo=FALSE}

p_Pgm_pop_Gibert_PercT6 <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Gibert')), aes(x=Population_Lat, y=PercT6, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_pop_Gibert_PercT6", x="Population", y = "PercT6") + ylim(c(min(d_Pgm$PercT6) , max(d_Pgm$PercT6)))+ geom_text(data = model_means_Pgm_PercT6_Gibert_cld, aes(x = Population, label = cld), y = max(d_Pgm$PercT6), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm_pop_Gibert_PercT6
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_pop_Gibert_PercT6.pdf",width=8, height=5)
p_Pgm_pop_Gibert_PercT6
invisible(dev.off())
```


_TotalPerc_

```{r Pgm_lmer_Gibert_TotalPerc, echo=FALSE}
Pgm_lmer_Gibert_TotalPerc <- lmer(TotalPerc ~ Population  + (1|Line:Population) +
                                    (1|Batch), data = (subset(d_Pgm,Supervisor.PI=='Gibert')))
capture.output(summary(Pgm_lmer_Gibert_TotalPerc),file = "Pigmentation/Pgm_lmer_Gibert_TotalPerc_sum.txt")
capture.output(anova(Pgm_lmer_Gibert_TotalPerc),file = "Pigmentation/Pgm_lmer_Gibert_TotalPerc.txt")
capture.output(emmeans(Pgm_lmer_Gibert_TotalPerc, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm_lmer_Gibert_TotalPerc_tk.txt")
```

```{r include=FALSE}
model_means_Pgm_TotalPerc_Gibert <- emmeans(object = Pgm_lmer_Gibert_TotalPerc, specs = ~ Population)
model_means_Pgm_TotalPerc_Gibert_cld <- cld(object = model_means_Pgm_TotalPerc_Gibert, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```




```{r}
anova(Pgm_lmer_Gibert_TotalPerc)
```

```{r}
summary(Pgm_lmer_Gibert_TotalPerc)
```


```{r p_Pgm_Gibert_TotalPerc, echo=FALSE}

p_Pgm_Gibert_TotalPerc <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Gibert')), aes(x=Population_Lat, y=TotalPerc, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_Gibert_TotalPerc", x="Population", y = "TotalPerc") + ylim(c(min(d_Pgm$TotalPerc) , max(d_Pgm$TotalPerc)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm_Gibert_TotalPerc
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_Gibert_TotalPerc.pdf",width=8, height=5)
p_Pgm_Gibert_TotalPerc
invisible(dev.off())
```


```{r p_Pgm_pop_Gibert_TotalPerc, echo=FALSE}

p_Pgm_pop_Gibert_TotalPerc <- ggplot(data = (subset(d_Pgm,Supervisor.PI=='Gibert')), aes(x=Population_Lat, y=TotalPerc, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm_pop_Gibert_TotalPerc", x="Population", y = "TotalPerc") + ylim(c(min(d_Pgm$TotalPerc) , max(d_Pgm$TotalPerc)))+ geom_text(data = model_means_Pgm_TotalPerc_Gibert_cld, aes(x = Population, label = cld), y = max(d_Pgm$TotalPerc), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm_pop_Gibert_TotalPerc
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm_pop_Gibert_TotalPerc.pdf",width=8, height=5)
p_Pgm_pop_Gibert_TotalPerc
invisible(dev.off())
```


#### Schmidt Lab


_Tergite8_

```{r Pgm2_lmer_Schmidt_Tergite8, echo=FALSE}
Pgm2_lmer_Schmidt_Tergite8 <- lmer(Tergite8 ~ Population  + (1|Line:Population), data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')))
capture.output(summary(Pgm2_lmer_Schmidt_Tergite8),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite8_sum.txt")
capture.output(anova(Pgm2_lmer_Schmidt_Tergite8),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite8.txt")
capture.output(emmeans(Pgm2_lmer_Schmidt_Tergite8, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite8_tk.txt")
```

```{r include=FALSE}
model_means_Pgm2_Tergite8_Schmidt <- emmeans(object = Pgm2_lmer_Schmidt_Tergite8, specs = ~ Population)
model_means_Pgm2_Tergite8_Schmidt_cld <- cld(object = model_means_Pgm2_Tergite8_Schmidt, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```


```{r}
anova(Pgm2_lmer_Schmidt_Tergite8)
```

```{r}
summary(Pgm2_lmer_Schmidt_Tergite8)
```


```{r p_Pgm2_Schmidt_Tergite8, echo=FALSE}

p_Pgm2_Schmidt_Tergite8 <- ggplot(data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')), aes(x=Population_Lat, y=Tergite8, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm2_Schmidt_Tergite8", x="Population", y = "Tergite8") + ylim(c(min(d_Pgm2$Tergite8) , max(d_Pgm2$Tergite8)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm2_Schmidt_Tergite8
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm2_Schmidt_Tergite8.pdf",width=8, height=5)
p_Pgm2_Schmidt_Tergite8
invisible(dev.off())
```


```{r p_Pgm2_pop_Schmidt_Tergite8, echo=FALSE}

p_Pgm2_pop_Schmidt_Tergite8 <- ggplot(data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')), aes(x=Population_Lat, y=Tergite8, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm2_pop_Schmidt_Tergite8", x="Population", y = "Tergite8") + ylim(c(min(d_Pgm2$Tergite8) , max(d_Pgm2$Tergite8)))+ geom_text(data = model_means_Pgm2_Tergite8_Schmidt_cld, aes(x = Population, label = cld), y = max(d_Pgm2$Tergite8), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm2_pop_Schmidt_Tergite8
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm2_pop_Schmidt_Tergite8.pdf",width=8, height=5)
p_Pgm2_pop_Schmidt_Tergite8
invisible(dev.off())
```



_Tergite9_

```{r Pgm2_lmer_Schmidt_Tergite9, echo=FALSE}
Pgm2_lmer_Schmidt_Tergite9 <- lmer(Tergite9 ~ Population  + (1|Line:Population) , data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')))
capture.output(summary(Pgm2_lmer_Schmidt_Tergite9),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite9_sum.txt")
capture.output(anova(Pgm2_lmer_Schmidt_Tergite9),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite9.txt")
capture.output(emmeans(Pgm2_lmer_Schmidt_Tergite9, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite9_tk.txt")
```


```{r include=FALSE}
model_means_Pgm2_Tergite9_Schmidt <- emmeans(object = Pgm2_lmer_Schmidt_Tergite9, specs = ~ Population)
model_means_Pgm2_Tergite9_Schmidt_cld <- cld(object = model_means_Pgm2_Tergite9_Schmidt, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```


```{r}
anova(Pgm2_lmer_Schmidt_Tergite9)
```

```{r}
summary(Pgm2_lmer_Schmidt_Tergite9)
```


```{r p_Pgm2_Schmidt_Tergite9, echo=FALSE}

p_Pgm2_Schmidt_Tergite9 <- ggplot(data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')), aes(x=Population_Lat, y=Tergite9, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm2_Schmidt_Tergite9", x="Population", y = "Tergite9") + ylim(c(min(d_Pgm2$Tergite9) , max(d_Pgm2$Tergite9)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm2_Schmidt_Tergite9
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm2_Schmidt_Tergite9.pdf",width=8, height=5)
p_Pgm2_Schmidt_Tergite9
invisible(dev.off())
```


```{r p_Pgm2_pop_Schmidt_Tergite9, echo=FALSE}

p_Pgm2_pop_Schmidt_Tergite9 <- ggplot(data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')), aes(x=Population_Lat, y=Tergite9, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm2_pop_Schmidt_Tergite9", x="Population", y = "Tergite9") + ylim(c(min(d_Pgm2$Tergite9) , max(d_Pgm2$Tergite9)))+ geom_text(data = model_means_Pgm2_Tergite9_Schmidt_cld, aes(x = Population, label = cld), y = max(d_Pgm2$Tergite9), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm2_pop_Schmidt_Tergite9
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm2_pop_Schmidt_Tergite9.pdf",width=8, height=5)
p_Pgm2_pop_Schmidt_Tergite9
invisible(dev.off())
```


_Tergite10_

```{r Pgm2_lmer_Schmidt_Tergite10, echo=FALSE}
Pgm2_lmer_Schmidt_Tergite10 <- lmer(Tergite10 ~ Population  + (1|Line:Population), data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')))
capture.output(summary(Pgm2_lmer_Schmidt_Tergite10),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite10_sum.txt")
capture.output(anova(Pgm2_lmer_Schmidt_Tergite10),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite10.txt")
capture.output(emmeans(Pgm2_lmer_Schmidt_Tergite10, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm2_lmer_Schmidt_Tergite10_tk.txt")
```

```{r include=FALSE}
model_means_Pgm2_Tergite10_Schmidt <- emmeans(object = Pgm2_lmer_Schmidt_Tergite10, specs = ~ Population)
model_means_Pgm2_Tergite10_Schmidt_cld <- cld(object = model_means_Pgm2_Tergite10_Schmidt, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```


```{r}
anova(Pgm2_lmer_Schmidt_Tergite10)
```

```{r}
summary(Pgm2_lmer_Schmidt_Tergite10)
```


```{r p_Pgm2_Schmidt_Tergite10, echo=FALSE}

p_Pgm2_Schmidt_Tergite10 <- ggplot(data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')), aes(x=Population_Lat, y=Tergite10, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm2_Schmidt_Tergite10", x="Population", y = "Tergite10") + ylim(c(min(d_Pgm2$Tergite10) , max(d_Pgm2$Tergite10)))+ droseu_theme
```

```{r echo=FALSE}
p_Pgm2_Schmidt_Tergite10
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm2_Schmidt_Tergite10.pdf",width=8, height=5)
p_Pgm2_Schmidt_Tergite10
invisible(dev.off())
```


```{r p_Pgm2_pop_Schmidt_Tergite10, echo=FALSE}

p_Pgm2_pop_Schmidt_Tergite10 <- ggplot(data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')), aes(x=Population_Lat, y=Tergite10, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm2_pop_Schmidt_Tergite10", x="Population", y = "Tergite10") + ylim(c(min(d_Pgm2$Tergite10) , max(d_Pgm2$Tergite10)))+ geom_text(data = model_means_Pgm2_Tergite10_Schmidt_cld, aes(x = Population, label = cld), y = max(d_Pgm2$Tergite10), vjust = 1) + droseu_theme
```

```{r echo=FALSE}
p_Pgm2_pop_Schmidt_Tergite10
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm2_pop_Schmidt_Tergite10.pdf",width=8, height=5)
p_Pgm2_pop_Schmidt_Tergite10
invisible(dev.off())
```


_Total_

```{r Pgm2_lmer_Schmidt_Total, echo=FALSE}
Pgm2_lmer_Schmidt_Total <- lmer(Total ~ Population  + (1|Line:Population), data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')))
capture.output(summary(Pgm2_lmer_Schmidt_Total),file = "Pigmentation/Pgm2_lmer_Schmidt_Total_sum.txt")
capture.output(anova(Pgm2_lmer_Schmidt_Total),file = "Pigmentation/Pgm2_lmer_Schmidt_Total.txt")
capture.output(emmeans(Pgm2_lmer_Schmidt_Total, list(pairwise ~ Population), adjust = "tukey", mode = "asymp"),file = "Pigmentation/Pgm2_lmer_Schmidt_Total_tk.txt")
```

```{r}
anova(Pgm2_lmer_Schmidt_Total)
```

```{r}
summary(Pgm2_lmer_Schmidt_Total)
```


```{r p_Pgm2_Schmidt_Total, echo=FALSE}

p_Pgm2_Schmidt_Total <- ggplot(data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')), aes(x=Population_Lat, y=Total, fill=Batch)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm2_Schmidt_Total", x="Population", y = "Total") + ylim(c(min(d_Pgm2$Total) , max(d_Pgm2$Total)))+ droseu_theme
```

```{r include=FALSE}
model_means_Pgm2_Total_Schmidt <- emmeans(object = Pgm2_lmer_Schmidt_Total, specs = ~ Population)
model_means_Pgm2_Total_Schmidt_cld <- cld(object = model_means_Pgm2_Total_Schmidt, adjust = "Tukey", Letters = letters, alpha = 0.05) %>% as.data.frame %>% mutate(Population = factor(Population, levels = c("YE","RE","GI","MU","MA","UM","KA","VA","AK"))) %>% arrange(Population) %>% dplyr::rename(cld = .group)
```


```{r echo=FALSE}
p_Pgm2_Schmidt_Total
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm2_Schmidt_Total.pdf",width=8, height=5)
p_Pgm2_Schmidt_Total
invisible(dev.off())
```


```{r p_Pgm2_pop_Schmidt_Total, echo=FALSE}

p_Pgm2_pop_Schmidt_Total <- ggplot(data = (subset(d_Pgm2,Supervisor.PI=='Schmidt')), aes(x=Population_Lat, y=Total, fill=Population)) + 
        geom_boxplot(outlier.shape = NA, notch=FALSE) + 
        labs(title="p_Pgm2_pop_Schmidt_Total", x="Population", y = "Total") + ylim(c(min(d_Pgm2$Total) , max(d_Pgm2$Total)))+ geom_text(data = model_means_Pgm2_Total_Schmidt_cld, aes(x = Population, label = cld), y = max(d_Pgm2$Total), vjust = 1) +droseu_theme
```

```{r echo=FALSE}
p_Pgm2_pop_Schmidt_Total
```

```{r echo=FALSE}
pdf(file="Pigmentation/p_Pgm2_pop_Schmidt_Total.pdf",width=8, height=5)
p_Pgm2_pop_Schmidt_Total
invisible(dev.off())
```

### Linear Models / Residuals

#### Abbott Lab
```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T4_Abbott_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T4_Abbott_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T5_Abbott_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T5_Abbott_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T6_Abbott_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T6_Abbott_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_Total_Abbott_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_Total_Abbott_lmer_pop_qq_plot_residuals.png")
```


#### Gibert Lab

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T4_Gibert_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T4_Gibert_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T5_Gibert_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T5_Gibert_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T6_Gibert_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T6_Gibert_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_Total_Gibert_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_Total_Gibert_lmer_pop_qq_plot_residuals.png")
```


#### Schmidt Lab

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T4_Schmidt_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T4_Schmidt_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T5_Schmidt_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T5_Schmidt_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T6_Schmidt_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_T6_Schmidt_lmer_pop_qq_plot_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_Total_Schmidt_lmer_pop_hist_residuals.png")
```

```{r}
knitr::include_graphics("LinearModelsPop/Pigmentation/by_lab_lmer_residuals/Pgm_Total_Schmidt_lmer_pop_qq_plot_residuals.png")
```

### Lab correlations

```{r intersect_Pgm, echo=FALSE}

intersect_Line_Pgm  <- Reduce(intersect, list(droseu$pgm$Line[droseu$pgm$Supervisor.PI == "Abbott"],
                                         droseu$pgm2$Line[droseu$pgm2$Supervisor.PI == "Schmidt"],
                                         droseu$pgm$Line[droseu$pgm$Supervisor.PI == "Gibert"]))

write(intersect_Line_Pgm, file ="Pigmentation/intersect_Line_Pgm.txt", sep = "\t")


intersect_Pop_Pgm  <- Reduce(intersect, list(droseu$pgm$Population[droseu$pgm$Supervisor.PI == "Abbott"],
                                         droseu$pgm2$Population[droseu$pgm2$Supervisor.PI == "Schmidt"],
                                         droseu$pgm$Population[droseu$pgm$Supervisor.PI == "Gibert"]))

write(intersect_Pop_Pgm, file ="Pigmentation/intersect_Pop_Pgm", sep = "\t")

```

There are `r length(intersect_Pop_Pgm) ` populations and `r length(intersect_Line_Pgm) ` isofemale lines that have been phenotyped across the labs. Please refer to tables named intersect_Trait_level (i.e. intersect_Via_pop) for details. 


```{r}
knitr::include_graphics("LabCorrelations/Pigmentation/Pgm_T4_F_lab_correlations.png")
```


```{r}
knitr::include_graphics("LabCorrelations/Pigmentation/Pgm_T5_F_lab_correlations.png")
```


```{r}
knitr::include_graphics("LabCorrelations/Pigmentation/Pgm_T6_F_lab_correlations.png")
```


```{r}
knitr::include_graphics("LabCorrelations/Pigmentation/Pgm_Total_F_lab_correlations.png")
```


# Results II : Meta-Analyses

## Viability

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/Viability/Via_meta_summary_effect.png")
```

## Development Time

_Females_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/DevelopmentTime/DT_A_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/DevelopmentTime/DT_A_M_meta_summary_effect.png")
```


## Dry weight

_Females_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/DryWeight/DW_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/DryWeight/DW_M_meta_summary_effect.png")
```

## Thorax length

_Females_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/ThoraxLength/TL_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/ThoraxLength/TL_M_meta_summary_effect.png")
```

## Wing area

_Left_
_Females_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/WingArea/WA_L_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/WingArea/WA_L_M_meta_summary_effect.png")
```

_Right_
_Females_
```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/WingArea/WA_R_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/WingArea/WA_R_M_meta_summary_effect.png")
```

## Fecundity

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/Fecundity/Fec_meta_summary_effect.png")
```

## Lifespan

_Females_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/Lifespan/LS_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/Lifespan/LS_M_meta_summary_effect.png")
```

## Cold-shock mortality

_Females_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/ColdShock/CSM_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/ColdShock/CSM_M_meta_summary_effect.png")
```

## Heat-shock mortality

_Females_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/HeatShock/HSM_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/HeatShock/HSM_M_meta_summary_effect.png")
```

## Diapause


```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/Diapause/Dia_meta_summary_effect.png")
```

## Starvation resistance

_Females_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/Starvation/SR_F_meta_summary_effect.png")
```

_Males_

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/Starvation/SR_M_meta_summary_effect.png")
```

## Pigmentation

```{r echo=FALSE}
knitr::include_graphics("MetaAnalyses/Pigmentation/Pgm_Total_meta_summary_effect.png")
```


# Results III : Environmental Variables  

```{r}
lmers_lat <- readRDS("LinearModelsLat/all_lmers_list_lat_anova.rds")
lmers_lon <- readRDS("LinearModelsLon/all_lmers_list_lon_anova.rds")
lmers_alt <- readRDS("LinearModelsAlt/all_lmers_list_alt_anova.rds")
```

## Geography - Latitude 

### Viability

#### Gibert Lab

```{r}
lmers_lat$Via_Gibert_lmer_lat
```

#### Grath Lab

```{r}
lmers_lat$Via_Grath_lmer_lat
```

#### Hoedjes Lab

```{r}
anova(Via_lmer_Lat_Hoedjes)
```

#### Schmidt Lab

```{r}
anova(Via_lmer_Lat_Schmidt) 
```

#### Stamenkovic-Radak Lab

```{r}
anova(Via_lmer_Lat_StamenkovicRadak)
```

#### Zwaan Lab

```{r}
anova(Via_lmer_Lat_Zwaan)
```

### Developmental time


### Egg-to-pupa developmental time

##### Schmidt Lab

```{r}
anova(DT_P_lmer_Lat)
```

### Egg-to-adult developmental time


##### Gibert Lab

```{r}
anova(DT_A_F_lmer_Lat_Gibert)
```

```{r}
anova(DT_A_M_lmer_Lat_Gibert)
```

##### Grath Lab


```{r}
anova(DT_A_F_lmer_Lat_Grath)
```

```{r}
anova(DT_A_M_lmer_Lat_Grath)
```

##### Hoedjes Lab

```{r}
anova(DT_A_F_lmer_Lat_Hoedjes)
```

```{r}
anova(DT_A_M_lmer_Lat_Hoedjes)
```

##### Schmidt Lab

```{r}
anova(DT_A_F_lmer_Lat_Schmidt)
```


```{r}
anova(DT_A_M_lmer_Lat_Schmidt)
```

##### Stamenkovic-Radak Lab

```{r}
anova(DT_A_F_lmer_Lat_StamenkovicRadak)
```

```{r}
anova(DT_A_M_lmer_Lat_StamenkovicRadak)
```


##### Zwaan Lab

```{r}
anova(DT_A_F_lmer_Lat_Zwaan)
```


```{r}
anova(DT_A_M_lmer_Lat_Zwaan)
```

### Dry weight

#### Colinet Lab

```{r}
anova(DW_F_lmer_Lat_Colinet)
```

```{r}
anova(DW_M_lmer_Lat_Colinet)
```

#### Hoedjes Lab 
```{r}
anova(DW_F_lmer_Lat_Hoedjes)
```

```{r}
anova(DW_M_lmer_Lat_Hoedjes)
```

#### Onder Lab


```{r}
anova(DW_F_lmer_Lat_Onder)
```

```{r}
anova(DW_M_lmer_Lat_Onder)
```

### Thorax length

#### Kozeretska Lab

```{r}
anova(TL_F_lmer_Lat_Kozeretska)
```

```{r}
anova(TL_M_lmer_Lat_Kozeretska)
```

#### Posnien Lab 

```{r}
anova(TL_F_lmer_Lat_Posnien)
```

```{r}
anova(TL_M_lmer_Lat_Posnien)
```

#### Ritchie Lab

```{r}
anova(TL_F_lmer_Lat_Ritchie)
```

```{r}
anova(TL_M_lmer_Lat_Ritchie)
```

#### Schmidt Lab


```{r}
anova(TL_F_lmer_Lat_Schmidt)
```

### Wing area

### _Left_

#### Onder Lab

```{r}
anova(WA_Lat_F_L_Onder_lmer)
```

```{r}
anova(WA_Lat_M_L_Onder_lmer)
```

#### Posnien Lab 

```{r}
anova(WA_Lat_F_L_Posnien_lmer)
```

```{r}
anova(WA_Lat_M_L_Posnien_lmer)
```

#### Ritchie Lab


```{r}
anova(WA_Lat_F_L_Ritchie_lmer)
```

```{r}
anova(WA_Lat_M_L_Ritchie_lmer)
```

#### Stamenkovic-Radak Lab


```{r}
anova(WA_Lat_F_L_StamenkovicRadak_lmer)
```

```{r}
anova(WA_Lat_M_L_StamenkovicRadak_lmer)
```

### _Right_

#### Onder Lab

```{r}
anova(WA_Lat_F_R_Onder_lmer)
```


```{r}
anova(WA_Lat_M_R_Onder_lmer)
```

#### Posnien Lab 

```{r}
anova(WA_Lat_F_R_Posnien_lmer)
```


```{r}
anova(WA_Lat_M_R_Posnien_lmer)
```

#### Ritchie Lab

```{r}
anova(WA_Lat_F_R_Ritchie_lmer)
```

```{r}
anova(WA_Lat_M_R_Ritchie_lmer)
```

#### Stamenkovic-Radak Lab

```{r}
anova(WA_Lat_F_R_StamenkovicRadak_lmer)
```

```{r}
anova(WA_Lat_M_R_StamenkovicRadak_lmer)
```


### Fecundity


#### Billeter Lab


```{r}
anova(Fec_lmer_Lat_Billeter)
```

#### Fricke Lab 

```{r}
anova(Fec_lmer_Lat_Fricke)
```

### Lifespan

#### Flatt Lab

```{r}
anova(LS_Lat_P_F_Flatt_lmer)
```


```{r}
anova(LS_Lat_P_M_Flatt_lmer)
```

#### Parsch Lab 

```{r}
anova(LS_Lat_L_F_Parsch_lmer)
```

```{r}
anova(LS_Lat_L_M_Parsch_lmer)
```

#### Pasyukova Lab

```{r}
anova(LS_Lat_L_F_Pasyukova_lmer)
```


```{r}
anova(LS_Lat_L_M_Pasyukova_lmer)
```


### Cold-shock mortality

#### Gonzalez Lab

```{r}
anova(CSM_F_lmer_Lat_Gonzalez)
```

```{r}
anova(CSM_M_lmer_Lat_Gonzalez)
```

#### Kozeretska Lab 

```{r}
anova(CSM_F_lmer_Lat_Kozeretska)
```


```{r}
anova(CSM_M_lmer_Lat_Kozeretska)
```

#### Vieira Lab


```{r}
anova(CSM_F_lmer_Lat_Vieira)
```


```{r}
anova(CSM_M_lmer_Lat_Vieira)
```

### Chill-coma recovery time

```{r}
anova(CCRT_F_lmer_Lat_Vieira)
```

```{r}
anova(CCRT_M_lmer_Lat_Vieira)
```

#### Mensh Lab 


### Heat-shock mortality

#### Parsch Lab


```{r}
anova(HSM_F_lmer_Lat_Parsch)
```


```{r}
anova(HSM_M_lmer_Lat_Parsch)
```

#### Vieira Lab 


```{r}
anova(HSM_F_lmer_Lat_Vieira)
```
``

```{r}
anova(HSM_M_lmer_Lat_Vieira)
```

### Diapause


#### Bergland Lab
```{r}
anova(Dia_lmer_Lat_Bergland)
```

#### Flatt Lab 

```{r}
anova(Dia_lmer_Lat_Flatt)
```

#### Schlotterer Lab

```{r}
anova(Dia_lmer_Lat_Schlotterer)
```

### Circadian eclosion timing


### Locomotor activity

#### Tauber Lab

```{r}
anova(LA_lmer_Lat_ND_Tauber)
```

_CircPhase_


```{r}
anova(LA_lmer_Lat_CircPhase_Tauber)
```

_Activity_


```{r}
anova(LA_lmer_Lat_Activity_Tauber)
```

### Starvation resistance



#### Gonzalez Lab


```{r}
anova(SR_F_lmer_Lat_Gonzalez)
```


```{r}
anova(SR_M_lmer_Lat_Gonzalez)
```

#### Onder Lab 


```{r}
anova(SR_F_lmer_Lat_Onder)
```


```{r}
anova(SR_M_lmer_Lat_Onder)
```

#### Pasyukova Lab


```{r}
anova(SR_F_lmer_Lat_Pasyukova)
```

```{r}
anova(SR_M_lmer_Lat_Pasyukova)
```

### Pigmentation

#### Abbott Lab
_PercT4_


```{r}
anova(Pgm_lmer_Lat_Abbott_PercT4)
```

_PercT5_

```{r}
anova(Pgm_lmer_Lat_Abbott_PercT5)
```

_PercT6_


```{r}
anova(Pgm_lmer_Lat_Abbott_PercT6)
```

_TotalPerc_


```{r}
anova(Pgm_lmer_Lat_Abbott_TotalPerc)
```

#### Gibert Lab 

_PercT4_

```{r}
anova(Pgm_lmer_Lat_Gibert_PercT4)
```

_PercT5_


```{r}
anova(Pgm_lmer_Lat_Gibert_PercT5)
```

_PercT6_


```{r}
anova(Pgm_lmer_Lat_Gibert_PercT6)
```

_TotalPerc_

```{r}
anova(Pgm_lmer_Lat_Gibert_TotalPerc)
```

#### Schmidt Lab

_Tergite8_


```{r}
anova(Pgm2_lmer_Lat_Schmidt_Tergite8)
```

_Tergite9_

```{r}
anova(Pgm2_lmer_Lat_Schmidt_Tergite9)
```

_Tergite10_

```{r}
anova(Pgm2_lmer_Lat_Schmidt_Tergite10)
```

_Total_

```{r}
anova(Pgm2_lmer_Lat_Schmidt_Total)
```


## Geography - Longitude

### Viability

#### Gibert Lab

```{r}
anova(Via_lmer_Long_Gibert)
```

#### Grath Lab


```{r}
anova(Via_lmer_Long_Grath)
```

#### Hoedjes Lab

```{r}
anova(Via_lmer_Long_Hoedjes)
```

#### Schmidt Lab

```{r}
anova(Via_lmer_Long_Schmidt) 
```

#### Stamenkovic-Radak Lab

```{r}
anova(Via_lmer_Long_StamenkovicRadak)
```

#### Zwaan Lab

```{r}
anova(Via_lmer_Long_Zwaan)
```

### Developmental time

#### Egg-to-pupa developmental time

##### Schmidt Lab


```{r}
anova(DT_P_lmer_Long)
```

#### Egg-to-adult developmental time

##### Gibert Lab

```{r}
anova(DT_A_F_lmer_Long_Gibert)
```

```{r}
anova(DT_A_M_lmer_Long_Gibert)
```

##### Grath Lab

```{r}
anova(DT_A_F_lmer_Long_Grath)
```


```{r}
anova(DT_A_M_lmer_Long_Grath)
```

##### Hoedjes Lab


```{r}
anova(DT_A_F_lmer_Long_Hoedjes)
```


```{r}
anova(DT_A_M_lmer_Long_Hoedjes)
```

##### Schmidt Lab

```{r}
anova(DT_A_F_lmer_Long_Schmidt)
```


```{r}
anova(DT_A_M_lmer_Long_Schmidt)
```

##### Stamenkovic-Radak Lab


```{r}
anova(DT_A_F_lmer_Long_StamenkovicRadak)
```

```{r}
anova(DT_A_M_lmer_Long_StamenkovicRadak)
```


##### Zwaan Lab


```{r}
anova(DT_A_F_lmer_Long_Zwaan)
```

```{r}
anova(DT_A_M_lmer_Long_Zwaan)
```

### Dry weight

#### Colinet Lab

```{r}
anova(DW_F_lmer_Long_Colinet)
```


```{r}
anova(DW_M_lmer_Long_Colinet)
```

#### Hoedjes Lab 

```{r}
anova(DW_F_lmer_Long_Hoedjes)
```

```{r}
anova(DW_M_lmer_Long_Hoedjes)
```

#### Onder Lab

```{r}
anova(DW_F_lmer_Long_Onder)
```


```{r}
anova(DW_M_lmer_Long_Onder)
```

### Thorax length

#### Kozeretska Lab

```{r}
anova(TL_F_lmer_Long_Kozeretska)
```


```{r}
anova(TL_M_lmer_Long_Kozeretska)
```

#### Posnien Lab 

```{r}
anova(TL_F_lmer_Long_Posnien)
```

```{r}
anova(TL_M_lmer_Long_Posnien)
```

#### Ritchie Lab

```{r}
anova(TL_F_lmer_Long_Ritchie)
```

```{r}
anova(TL_M_lmer_Long_Ritchie)
```

#### Schmidt Lab

```{r}
anova(TL_F_lmer_Long_Schmidt)
```

### Wing area

### _Left_

#### Onder Lab

```{r}
anova(WA_Long_F_L_Onder_lmer)
```


```{r}
anova(WA_Long_M_L_Onder_lmer)
```

#### Posnien Lab 


```{r}
anova(WA_Long_F_L_Posnien_lmer)
```


```{r}
anova(WA_Long_M_L_Posnien_lmer)
```

#### Ritchie Lab

```{r}
anova(WA_Long_F_L_Ritchie_lmer)
```


```{r}
anova(WA_Long_M_L_Ritchie_lmer)
```

#### Stamenkovic-Radak Lab

```{r}
anova(WA_Long_F_L_StamenkovicRadak_lmer)
```


```{r}
anova(WA_Long_M_L_StamenkovicRadak_lmer)
```

### _Right_


```{r}
anova(WA_Long_F_R_Onder_lmer)
```


```{r}
anova(WA_Long_M_R_Onder_lmer)
```

#### Posnien Lab 


```{r}
anova(WA_Long_F_R_Posnien_lmer)
```


```{r}
anova(WA_Long_M_R_Posnien_lmer)
```

#### Ritchie Lab


```{r}
anova(WA_Long_F_R_Ritchie_lmer)
```

```{r}
anova(WA_Long_M_R_Ritchie_lmer)
```

#### Stamenkovic-Radak Lab


```{r}
anova(WA_Long_F_R_StamenkovicRadak_lmer)
```


```{r}
anova(WA_Long_M_R_StamenkovicRadak_lmer)
```


### Fecundity

#### Billeter Lab

```{r}
anova(Fec_lmer_Long_Billeter)
```

#### Fricke Lab 

```{r}
anova(Fec_lmer_Long_Fricke)
```

### Lifespan

#### Flatt Lab


```{r}
anova(LS_Long_P_F_Flatt_lmer)
```

```{r}
anova(LS_Long_P_M_Flatt_lmer)
```

#### Parsch Lab 


```{r}
anova(LS_Long_L_F_Parsch_lmer)
```

```{r}
anova(LS_Long_L_M_Parsch_lmer)
```

#### Pasyukova Lab

```{r}
anova(LS_Long_L_F_Pasyukova_lmer)
```


```{r}
anova(LS_Long_L_M_Pasyukova_lmer)
```

### Cold-shock mortality

#### Gonzalez Lab

```{r}
anova(CSM_F_lmer_Long_Gonzalez)
```


```{r}
anova(CSM_M_lmer_Long_Gonzalez)
```

#### Kozeretska Lab 

```{r}
anova(CSM_F_lmer_Long_Kozeretska)
```

```{r}
anova(CSM_M_lmer_Long_Kozeretska)
```

#### Vieira Lab

```{r}
anova(CSM_F_lmer_Long_Vieira)
```


```{r}
anova(CSM_M_lmer_Long_Vieira)
```

### Chill-coma recovery time

#### Mensh Lab 

#### Vieira Lab

```{r}
anova(CCRT_F_lmer_Long_Vieira)
```


```{r}
anova(CCRT_M_lmer_Long_Vieira)
```


### Heat-shock mortality


#### Parsch Lab

```{r}
anova(HSM_F_lmer_Long_Parsch)
```

```{r}
anova(HSM_M_lmer_Long_Parsch)
```

#### Vieira Lab 

```{r}
anova(HSM_F_lmer_Long_Vieira)
```


```{r}
anova(HSM_M_lmer_Long_Vieira)
```

### Diapause

#### Bergland Lab

```{r eval=FALSE}
anova(Dia_lmer_Long_Bergland)
```

#### Flatt Lab 

```{r eval=FALSE}
anova(Dia_lmer_Long_Flatt)
```

#### Schlotterer Lab

```{r eval=FALSE}
anova(Dia_lmer_Long_Schlotterer)
```

### Circadian eclosion timing

### Locomotor activity

#### Tauber Lab

_ND_

```{r}
anova(LA_lmer_Long_ND_Tauber)
```

_CircPhase_

```{r}
anova(LA_lmer_Long_CircPhase_Tauber)
```

_Activity_

```{r}
anova(LA_lmer_Long_Activity_Tauber)
```

### Starvation resistance

#### Gonzalez Lab
```{r}
anova(SR_F_lmer_Long_Gonzalez)
```


```{r}
anova(SR_M_lmer_Long_Gonzalez)
```

#### Onder Lab 

```{r}
anova(SR_F_lmer_Long_Onder)
```


```{r}
anova(SR_M_lmer_Long_Onder)
```

#### Pasyukova Lab

```{r}
anova(SR_F_lmer_Long_Pasyukova)
```


```{r}
anova(SR_M_lmer_Long_Pasyukova)
```

### Pigmentation

#### Abbott Lab
_PercT4_

```{r}
anova(Pgm_lmer_Long_Abbott_PercT4)
```

_PercT5_

```{r}
anova(Pgm_lmer_Long_Abbott_PercT5)
```

_PercT6_

```{r}
anova(Pgm_lmer_Long_Abbott_PercT6)
```

_TotalPerc_

```{r}
anova(Pgm_lmer_Long_Abbott_TotalPerc)
```

#### Gibert Lab 

_PercT4_

```{r}
anova(Pgm_lmer_Long_Gibert_PercT4)
```

_PercT5_

```{r}
anova(Pgm_lmer_Long_Gibert_PercT5)
```

_PercT6_

```{r}
anova(Pgm_lmer_Long_Gibert_PercT6)
```

_TotalPerc_


```{r}
anova(Pgm_lmer_Long_Gibert_TotalPerc)
```

#### Schmidt Lab

_Tergite8_


```{r}
anova(Pgm2_lmer_Long_Schmidt_Tergite8)
```

_Tergite9_


```{r}
anova(Pgm2_lmer_Long_Schmidt_Tergite9)
```

_Tergite10_

```{r}
anova(Pgm2_lmer_Long_Schmidt_Tergite10)
```

_Total_

```{r}
anova(Pgm2_lmer_Long_Schmidt_Total)
```



## Geography - Altitude

### Viability

#### Gibert Lab

```{r}
anova(Via_lmer_Alt_Gibert)
```

#### Grath Lab

```{r}
anova(Via_lmer_Alt_Grath)
```

#### Hoedjes Lab

```{r}
anova(Via_lmer_Alt_Hoedjes)
```

#### Schmidt Lab

```{r}
anova(Via_lmer_Alt_Schmidt) 
```

#### Stamenkovic-Radak Lab

```{r}
anova(Via_lmer_Alt_StamenkovicRadak)
```

#### Zwaan Lab

```{r}
anova(Via_lmer_Alt_Zwaan)
```

### Developmental time

#### Egg-to-pupa developmental time

##### Schmidt Lab

```{r}
anova(DT_P_lmer_Alt)
```

#### Egg-to-adult developmental time

##### Gibert Lab

```{r}
anova(DT_A_F_lmer_Alt_Gibert)
```

```{r}
anova(DT_A_M_lmer_Alt_Gibert)
```

##### Grath Lab

```{r}
anova(DT_A_F_lmer_Alt_Grath)
```


```{r}
anova(DT_A_M_lmer_Alt_Grath)
```

##### Hoedjes Lab

```{r}
anova(DT_A_F_lmer_Alt_Hoedjes)
```

```{r}
anova(DT_A_M_lmer_Alt_Hoedjes)
```

##### Schmidt Lab

```{r}
anova(DT_A_F_lmer_Alt_Schmidt)
```

```{r}
anova(DT_A_M_lmer_Alt_Schmidt)
```

##### Stamenkovic-Radak Lab

```{r}
anova(DT_A_F_lmer_Alt_StamenkovicRadak)
```

```{r}
anova(DT_A_M_lmer_Alt_StamenkovicRadak)
```


##### Zwaan Lab

```{r}
anova(DT_A_F_lmer_Alt_Zwaan)
```

```{r}
anova(DT_A_M_lmer_Alt_Zwaan)
```

### Dry weight


#### Colinet Lab

```{r}
anova(DW_F_lmer_Alt_Colinet)
```


```{r}
anova(DW_M_lmer_Alt_Colinet)
```

#### Hoedjes Lab 

```{r}
anova(DW_F_lmer_Alt_Hoedjes)
```

```{r}
anova(DW_M_lmer_Alt_Hoedjes)
```

#### Onder Lab



```{r}
anova(DW_F_lmer_Alt_Onder)
```


```{r}
anova(DW_M_lmer_Alt_Onder)
```

### Thorax length



#### Kozeretska Lab


```{r}
anova(TL_F_lmer_Alt_Kozeretska)
```


```{r}
anova(TL_M_lmer_Alt_Kozeretska)
```

#### Posnien Lab 

```{r}
anova(TL_F_lmer_Alt_Posnien)
```

```{r}
anova(TL_M_lmer_Alt_Posnien)
```

#### Ritchie Lab


```{r}
anova(TL_F_lmer_Alt_Ritchie)
```


```{r}
anova(TL_M_lmer_Alt_Ritchie)
```

#### Schmidt Lab

```{r}
anova(TL_F_lmer_Alt_Schmidt)
```

### Wing area


### _Left_

#### Onder Lab

```{r}
anova(WA_Alt_F_L_Onder_lmer)
```


```{r}
anova(WA_Alt_M_L_Onder_lmer)
```

#### Posnien Lab 

```{r}
anova(WA_Alt_F_L_Posnien_lmer)
```


```{r}
anova(WA_Alt_M_L_Posnien_lmer)
```

#### Ritchie Lab

```{r}
anova(WA_Alt_F_L_Ritchie_lmer)
```

```{r}
anova(WA_Alt_M_L_Ritchie_lmer)
```

#### Stamenkovic-Radak Lab


```{r}
anova(WA_Alt_F_L_StamenkovicRadak_lmer)
```


```{r}
anova(WA_Alt_M_L_StamenkovicRadak_lmer)
```

### _Right_

```{r wing-area-right}
```


```{r}
anova(WA_Alt_F_R_Onder_lmer)
```

```{r}
anova(WA_Alt_M_R_Onder_lmer)
```

#### Posnien Lab 

```{r}
anova(WA_Alt_F_R_Posnien_lmer)
```

```{r}
anova(WA_Alt_M_R_Posnien_lmer)
```

#### Ritchie Lab


```{r}
anova(WA_Alt_F_R_Ritchie_lmer)
```

```{r}
anova(WA_Alt_M_R_Ritchie_lmer)
```

#### Stamenkovic-Radak Lab

```{r}
anova(WA_Alt_F_R_StamenkovicRadak_lmer)
```


```{r}
anova(WA_Alt_M_R_StamenkovicRadak_lmer)
```


### Fecundity

#### Billeter Lab

```{r}
anova(Fec_lmer_Alt_Billeter)
```

#### Fricke Lab 

```{r}
anova(Fec_lmer_Alt_Fricke)
```

### Lifespan

#### Flatt Lab

```{r}
anova(LS_Alt_P_F_Flatt_lmer)
```

```{r}
anova(LS_Alt_P_M_Flatt_lmer)
```

#### Parsch Lab 


```{r}
anova(LS_Alt_L_F_Parsch_lmer)
```


```{r}
anova(LS_Alt_L_M_Parsch_lmer)
```

#### Pasyukova Lab

```{r}
anova(LS_Alt_L_F_Pasyukova_lmer)
```

```{r}
anova(LS_Alt_L_M_Pasyukova_lmer)
```

### Cold-shock mortality


#### Gonzalez Lab

```{r}
anova(CSM_F_lmer_Alt_Gonzalez)
```



```{r}
anova(CSM_M_lmer_Alt_Gonzalez)
```

#### Kozeretska Lab 


```{r}
anova(CSM_F_lmer_Alt_Kozeretska)
```

```{r}
anova(CSM_M_lmer_Alt_Kozeretska)
```

#### Vieira Lab

```{r}
anova(CSM_F_lmer_Alt_Vieira)
```

```{r}
anova(CSM_M_lmer_Alt_Vieira)
```

### Chill-coma recovery time


#### Vieira Lab

```{r}
anova(CCRT_F_lmer_Alt_Vieira)
```


```{r}
anova(CCRT_M_lmer_Alt_Vieira)
```

#### Mensh Lab 


### Heat-shock mortality


#### Parsch Lab

```{r}
anova(HSM_F_lmer_Alt_Parsch)
```

```{r}
anova(HSM_M_lmer_Alt_Parsch)
```

#### Vieira Lab 


```{r}
anova(HSM_F_lmer_Alt_Vieira)
```


```{r}
anova(HSM_M_lmer_Alt_Vieira)
```

### Diapause


#### Bergland Lab

```{r eval=FALSE}
anova(Dia_lmer_Alt_Bergland)
```

#### Flatt Lab 

```{r eval=FALSE}
anova(Dia_lmer_Alt_Flatt)
```


#### Schlotterer Lab

```{r eval=FALSE}
anova(Dia_lmer_Alt_Schlotterer)
```

### Circadian eclosion timing

### Locomotor activity


#### Tauber Lab

_ND_

```{r}
anova(LA_lmer_Alt_ND_Tauber)
```

_CircPhase_

```{r}
anova(LA_lmer_Alt_CircPhase_Tauber)
```

_Activity_


```{r}
anova(LA_lmer_Alt_Activity_Tauber)
```

### Starvation resistance

#### Gonzalez Lab

```{r}
anova(SR_F_lmer_Alt_Gonzalez)
```


```{r}
anova(SR_M_lmer_Alt_Gonzalez)
```

#### Onder Lab 

```{r}
anova(SR_F_lmer_Alt_Onder)
```

```{r}
anova(SR_M_lmer_Alt_Onder)
```

#### Pasyukova Lab

```{r}
anova(SR_F_lmer_Alt_Pasyukova)
```

```{r}
anova(SR_M_lmer_Alt_Pasyukova)
```

### Pigmentation


#### Abbott Lab
_PercT4_

```{r}
anova(Pgm_lmer_Alt_Abbott_PercT4)
```

_PercT5_

```{r}
anova(Pgm_lmer_Alt_Abbott_PercT5)
```

_PercT6_

```{r}
anova(Pgm_lmer_Alt_Abbott_PercT6)
```

_TotalPerc_

```{r}
anova(Pgm_lmer_Alt_Abbott_TotalPerc)
```

#### Gibert Lab 

_PercT4_


```{r}
anova(Pgm_lmer_Alt_Gibert_PercT4)
```

_PercT5_

```{r}
anova(Pgm_lmer_Alt_Gibert_PercT5)
```

_PercT6_

```{r}
anova(Pgm_lmer_Alt_Gibert_PercT6)
```

_TotalPerc_


```{r}
anova(Pgm_lmer_Alt_Gibert_TotalPerc)
```

#### Schmidt Lab

_Tergite4_


```{r}
anova(Pgm2_lmer_Alt_Schmidt_Tergite8)
```

_Tergite5_


```{r}
anova(Pgm2_lmer_Alt_Schmidt_Tergite9)
```

_Tergite6_


```{r}
anova(Pgm2_lmer_Alt_Schmidt_Tergite10)
```

_Total_

```{r}
anova(Pgm2_lmer_Alt_Schmidt_Total)
```

# Results IV : Trait Correlations _tbd_

## Stress related traits
## Temperature related traits
## Size related traits

# Results V : Qst - Fst  _tbd_

# Results VI : Heritability  _tbd_
